{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 10,
   "metadata": {},
   "outputs": [],
   "source": [
    "const rawPaper = `Article\n",
    "Machine Learning Identiﬁes Stemness Features\n",
    "Associated with Oncogenic Dedifferentiation\n",
    "Graphical Abstract\n",
    "Highlights\n",
    "dEpigenetic and expression-based stemness indices\n",
    "measure oncogenic dedifferentiation\n",
    "dImmune microenvironment content and PD-L1 levelsassociate with stemness indices\n",
    "dStemness index is increased in metastatic tumors andreveals intratumor heterogeneity\n",
    "dApplying stemness indices reveals potential drug targets foranti-cancer therapiesAuthors\n",
    "Tathiane M. Malta, Artem Sokolov,Andrew J. Gentles, ..., Peter W. Laird,Houtan Noushmehr, Maciej Wiznerowicz\n",
    "Correspondence\n",
    "hnoushm1@hfhs.org (H.N.),\n",
    "maciej.wiznerowicz@iimo.pl (M.W.)\n",
    "In Brief\n",
    "Stemness features extracted from\n",
    "transcriptomic and epigenetic data fromTCGA tumors reveal novel biological andclinical insight, as well as potential drugtargets for anti-cancer therapies.\n",
    "Malta et al., 2018, Cell 173, 338–354\n",
    "April 5, 2018 ª2018 The Authors. Published by Elsevier Inc.\n",
    "https://doi.org/10.1016/j.cell.2018.03.034\n",
    "Article\n",
    "Machine Learning Identiﬁes Stemness Features\n",
    "Associated with Oncogenic Dedifferentiation\n",
    "Tathiane M. Malta,1,2,23 Artem Sokolov,3,23Andrew J. Gentles,4Tomasz Burzykowski,5Laila Poisson,1\n",
    "John N. Weinstein,6Bo_zena Kami /C19nska,7Joerg Huelsken,8Larsson Omberg,9Olivier Gevaert,4Antonio Colaprico,10,11\n",
    "Patrycja Czerwi /C19nska,12Sylwia Mazurek,12,13Lopa Mishra,14Holger Heyn,15Alex Krasnitz,16Andrew K. Godwin,17\n",
    "Alexander J. Lazar,6The Cancer Genome Atlas Research Network, Joshua M. Stuart,18Katherine A. Hoadley,19\n",
    "Peter W. Laird,20Houtan Noushmehr,1,2,23, *and Maciej Wiznerowicz12,21,22,23,24, *\n",
    "1Henry Ford Health System, Detroit, MI 48202, USA\n",
    "2University of Sa ˜o Paulo, Ribeira ˜o Preto-SP 14049, Brazil\n",
    "3Harvard Medical School, Boston, MA 02115, USA\n",
    "4Stanford University, Palo Alto, CA 94305, USA\n",
    "5Hasselt University, 3590 Diepenbeek, Belgium\n",
    "6The University of Texas MD Anderson Cancer Center, Houston, TX 77030, USA\n",
    "7Nencki Institute of Experimental Biology of PAS, 02093 Warsaw, Poland\n",
    "8Swiss Federal Institute of Technology Lausanne (EPFL), CH-1015 Lausanne; Switzerland\n",
    "9Sage Bionetworks, Seattle, WA 98109 USA\n",
    "10Universite ´Libre de Bruxelles, 1050 Bruxelles, Belgium\n",
    "11Interuniversity Institute of Bioinformatics in Brussels (IB)2, 1050 Bruxelles; Belgium\n",
    "12Pozna /C19n University of Medical Sciences, 61701 Pozna /C19n, Poland\n",
    "13Postgraduate School of Molecular Medicine, Medical University of Warsaw, 02109 Warsaw, Poland\n",
    "14George Washington University, Washington, D.C. 20052, USA\n",
    "15Centre for Genomic Regulation (CNAG-CRG), 08003 Barcelona, Spain\n",
    "16Cold Spring Harbor Laboratory, Cold Spring Harbor, NY 11724, USA\n",
    "17University of Kansas Medical Center, Kansas City, KS 66160, USA\n",
    "18University of California, Santa Cruz, Santa Cruz, CA 95064, USA\n",
    "19University of North Carolina, Chapel Hill, NC 27599, USA\n",
    "20Van Andel Research Institute, Grand Rapids, MI 49503, USA\n",
    "21Greater Poland Cancer Center, 61866 Pozna /C19n, Poland\n",
    "22International Institute for Molecular Oncology, 60203 Pozna /C19n, Poland\n",
    "23These authors contributed equally\n",
    "24Lead Contact\n",
    "*Correspondence: hnoushm1@hfhs.org (H.N.), maciej.wiznerowicz@iimo.pl (M.W.)\n",
    "https://doi.org/10.1016/j.cell.2018.03.034\n",
    "SUMMARY\n",
    "Cancer progression involves the gradual loss of a\n",
    "differentiated phenotype and acquisition of pro-genitor and stem-cell-like features. Here, we providenovel stemness indices for assessing the degreeof oncogenic dedifferentiation. We used an innova-\n",
    "tive one-class logistic regression (OCLR) machine-\n",
    "learning algorithm to extract transcriptomic andepigenetic feature sets derived from non-trans-formed pluripotent stem cells and their differentiatedprogeny. Using OCLR, we were able to identify previ-ously undiscovered biological mechanisms asso-ciated with the dedifferentiated oncogenic state.Analyses of the tumor microenvironment revealed\n",
    "unanticipated correlation of cancer stemness with\n",
    "immune checkpoint expression and inﬁltratingimmune cells. We found that the dedifferentiatedoncogenic phenotype was generally most prominentin metastatic tumors. Application of our stemnessindices to single-cell data revealed patterns of\n",
    "intra-tumor molecular heterogeneity. Finally, theindices allowed for the identiﬁcation of novel targetsand possible targeted therapies aimed at tumordifferentiation.\n",
    "INTRODUCTION\n",
    "Stemness, deﬁned as the potential for self-renewal and differen-\n",
    "tiation from the cell of origin, was originally attributed to normal\n",
    "stem cells that possess the ability to give rise to all cell types in\n",
    "the adult organism. Cancer progression involves gradual lossof a differentiated phenotype and acquisition of progenitor-like,stem-cell-like features. Undifferentiated primary tumors are\n",
    "more likely to result in cancer cell spread to distant organs,\n",
    "causing disease progression and poor prognosis, particularlybecause metastases are usually resistant to available therapies\n",
    "(Friedmann-Morvinski and Verma, 2014; Ge et al., 2017; Shibue\n",
    "and Weinberg, 2017; Visvader and Lindeman, 2012 ).\n",
    "An increasing number of genomic, epigenomic, transcrip-\n",
    "tomic, and proteomic signatures have been associated with\n",
    "338 Cell173, 338–354, April 5, 2018 ª2018 The Authors. Published by Elsevier Inc.\n",
    "This is an open access article under the CC BY-NC-ND license ( http://creativecommons.org/licenses/by-nc-nd/4.0/ ).A\n",
    "B\n",
    "C\n",
    "Figure 1. Development and Validation of the Stemness Indices\n",
    "(A) Overall methodology. Highlighted are data sources Progenitor Cell Biology Consortium (PCBC), Roadmap, and ENCODE databases; the OCLR machine-\n",
    "learning algorithm; and the resulting stemness indices mRNAsi, mDNAsi, and EREG-mRNAsi. The indices for each TCGA tumor sample were correlated withknown cancer biology, tumor pathology, clinical information, and drug sensitivity.(B) Stemness indices of the validation set derived using our stemness signature.\n",
    "(legend continued on next page)\n",
    "Cell173, 338–354, April 5, 2018 339cancer stemness. Those molecular features are causally con-\n",
    "nected to particular oncogenic signaling pathways that regulate\n",
    "transcriptional networks that sustain the growth and proliferation\n",
    "of cancer cells ( Ben-Porath et al., 2008; Eppert et al., 2011; Kim\n",
    "et al., 2010 ). Transcriptional and epigenetic dysregulation of can-\n",
    "cer cells frequently leads to oncogenic dedifferentiation and\n",
    "acquisition of stemness features by altering core signaling path-ways that regulate the phenotypes of normal stem cells ( Bradner\n",
    "et al., 2017; Young, 2011 ). Cell-extrinsic mechanisms can also\n",
    "affect maintenance of the undifferentiated state, largely through\n",
    "epigenetic mechanisms. Tumors comprise a complex, diverse,integrated ecosystem of relatively differentiated cancer cells,\n",
    "cancer stem cells, endothelial cells, tumor-associated ﬁbro-\n",
    "blasts, and inﬁltrating immune cells, among other cell types.The microenvironment of a tumor, considered as a pathologically\n",
    "formed ‘‘organ,’’ is frequently characterized by hypoxia, as well\n",
    "as by abnormal levels of various cytokines, growth factors, andmetabolites ( Lyssiotis and Kimmelman, 2017 ). It provides\n",
    "numerous opportunities for cell-cell signals to modulate the epi-\n",
    "genome and expression of stem-cell-like programs in cancer\n",
    "cells, frequently independent of their genetic backgrounds ( Gin-\n",
    "gold et al., 2016 ).\n",
    "Over the last decade, The Cancer Genome Atlas (TCGA) has\n",
    "illuminated the landscapes of primary tumors by generatingcomprehensive molecular proﬁles composed of genomic, epige-\n",
    "nomic, transcriptomic, and (post-translational) proteomic char-\n",
    "acteristics ( Hoadley et al., 2014; Tomczak et al., 2015 ), along\n",
    "with histopathological and clinical annotations. The resulting\n",
    "resource enabled us to analyze cancer stemness quite exten-\n",
    "sively in almost 12,000 samples of 33 tumor types.\n",
    "First, we deﬁned signatures to quantify stemness using pub-\n",
    "licly available molecular proﬁles from normal cell types that\n",
    "exhibit various degrees of stemness. By multi-platform analyses\n",
    "of their transcriptome, methylome, and transcription-factor bind-ing sites using an innovative one-class logistic regression\n",
    "(OCLR) machine-learning algorithm ( Sokolov et al., 2016 ), we ob-\n",
    "tained two independent stemness indices. One (mDNAsi) wasreﬂective of epigenetic features; the other (mRNAsi) was reﬂec-\n",
    "tive of gene expression. We then identiﬁed associations between\n",
    "the two stemness indices and novel oncogenic pathways,somatic alterations, and microRNA (miRNA) and transcriptionalregulatory networks. Those features correlated with, and\n",
    "perhaps govern, cancer stemness in particular molecular sub-\n",
    "types of TCGA tumors. Importantly, higher values for stemnessindices were associated with biological processes active in\n",
    "cancer stem cells and with greater tumor dedifferentiation, as\n",
    "reﬂected in histopathological grade. Metastatic tumor cellsappeared more dedifferentiated phenotypically, probably\n",
    "contributing to their aggressiveness. We also found tumor het-\n",
    "erogeneity at the single-cell level by measuring stemness in tran-scriptome proﬁles obtained from individual cancer cells. Using\n",
    "CIBERSORT to proﬁle immune cell types in TCGA tumors, weobtained insight into the interface of the immune system with\n",
    "stemness. Finally, we identiﬁed compounds speciﬁc to selected\n",
    "molecular targets and mechanisms that may eventually lead to\n",
    "novel treatments that trigger differentiation and exhaust thestemness potential of highly aggressive neoplasms.\n",
    "RESULTS\n",
    "DNA-Methylation- and mRNA-Expression-Based\n",
    "Stemness Classiﬁers\n",
    "We analyzed publicly available non-tumor and tumor datasetsfor which transcriptomic and epigenomic molecular proﬁles\n",
    "were available ( Figure 1 A). We derived stemness indices using\n",
    "an OCLR algorithm trained on stem cell (ESC, embryonic stemcell; iPSC, induced pluripotent stem cell) classes and their differ-\n",
    "entiated ecto-, meso-, and endoderm progenitors. We chose\n",
    "OCLR because it does not penalize misclassiﬁcation of stem-cell-derived progenitors at different stages of differentiationthat still carry some of the undifferentiated features in their\n",
    "molecular proﬁles (its output was also validated against random\n",
    "forests in Figure S1 A). OCLR-based transcriptomic and epige-\n",
    "netic signatures were applied to TCGA datasets to calculate\n",
    "the mRNAsi and mDNAsi. Each stemness index (si) ranges\n",
    "from low (zero) to high (one) stemness ( Table S1 ). The tumor\n",
    "samples stratiﬁed by the indices were used for the integrative\n",
    "analyses.\n",
    "mRNA Expression-Based Stemness Index\n",
    "We validated the mRNAsi by applying it to an external dataset\n",
    "composed of both stem cells and somatic differentiated cells(Nazor et al., 2012 )(Figure 1 B) and by scoring molecular sub-\n",
    "types of breast cancers and gliomas that are characterized by\n",
    "different degrees of oncogenic dedifferentiation associated\n",
    "with pathology and clinical outcome ( Figures S1 B and S1C). All\n",
    "stem cell samples attained higher stemness index values than\n",
    "samples from differentiated cells. TCGA tumors display various\n",
    "degrees of cancer stemness as revealed by mRNAsi ( Figure 1 C\n",
    "[left]) and mDNAsi ( Figure 1 C [right]). Germ-cell tumors, basal\n",
    "breast cancer, and Ly-Hem cancers displayed highly dedifferen-\n",
    "tiated phenotypes in comparison to other tumor types.\n",
    "Using gene set enrichment analysis (GSEA), we compared our\n",
    "signature to 16 gene sets that were associated with stemness\n",
    "in cancer and healthy cells in previous studies ( Ben-Porath\n",
    "et al., 2008; Ivanova et al., 2006; Kim and Orkin, 2011;Mathur et al., 2008; Palmer et al., 2012; Sato et al., 2003;\n",
    "Venezia et al., 2004; Yan et al., 2011 ). These sets spanned\n",
    "2,564 unique genes, with no 2 sets overlapping by more than134 genes. In all cases, the published stemness gene sets\n",
    "were signiﬁcantly enriched in mRNAsi ( Figure 2 A). We found\n",
    "that ‘‘cancer hallmark’’ gene sets were signiﬁcantly enriched,as were MYC targets, which signiﬁcantly contributed to the pos-\n",
    "itive side of the signature ( Hanahan and Weinberg, 2011 ). This is\n",
    "(C) TCGA tumor types sorted by the stemness indices obtained from transcriptomic (mRNAsi) and epigenetic (mDNAsi) features; indices were scaled fro m 0 (low)\n",
    "to 1 (high). The TCGA tumor types were grouped based on their histology and cell of origin into stem cell-like (SC), lympho-hematopoietic (Ly-Hem), ad eno-\n",
    "carcinomas, squamous cell carcinomas (Squamous), neuronal lineage (Neuronal), sarcomas (Sar), kidney tumors (Kidney), and not belonging to any of the above\n",
    "(Misc) ( Table S2 ).\n",
    "See also Figures S1 andS2andTables S1 andS2.\n",
    "340 Cell173, 338–354, April 5, 2018B A\n",
    "C\n",
    "D\n",
    "Figure 2. Biological Processes Associated with Cancer Stemness\n",
    "(A) Gene Set Enrichment Analysis showing RNA sequencing (RNA-seq)-based stemness signature evaluated in the context of gene sets representative fo r\n",
    "hallmarks of stemness and cancer.(B) Correlation between mRNAsi and mRNA expression for published hallmarks of stemness.(C) Correlation between mRNAsi and selected oncogenic processes.(D) Association between the epigenomic-based stemness signature (EREG-mDNAsi and EREG-mRNAsi) and enrichment in the transcription factor bindin g sites.\n",
    "See also Figure S2 andTable S2 .\n",
    "Cell173, 338–354, April 5, 2018 341consistent with MYC being one of the transcription factors that\n",
    "drive pluripotency in ESCs ( Young, 2011 ).\n",
    "Wingless-related integration site (Wnt)/ b-catenin and TGF- b\n",
    "signaling pathways were signiﬁcantly enriched on the negativeside of the stemness signature. This negative enrichment does\n",
    "not imply absence of speciﬁc signals in cancer stem cells, but\n",
    "rather that this signaling is lower relative to stem-cell-derived pro-genitors, as captured by the signature weights. This is againconsistent with other GSEA results, as both signaling pathways\n",
    "are known mediators of the epithelial-mesenchymal transition\n",
    "(EMT) mechanism ( Gonzalez and Medici, 2014 ). We also\n",
    "computed the correlation of mRNAsi against mRNA expression\n",
    "of published pan-cancer EMT markers ( Mak et al., 2016 ), which\n",
    "revealedsigniﬁcant correlations withformosttumors( FigureS2 C).\n",
    "This is consistent with the biology of ESCs, which grow as epithe-\n",
    "lioid, polygonal cells in vitro and epithelial cancer precursors hav-\n",
    "ing stem-like properties. Importantly, most TCGA samples are pri-mary tumors of an epithelial phenotype. Most skin melanomacases come from lymph nodes, and this tumor type shows higher\n",
    "expression of vimentin, a key marker of a mesenchymal pheno-\n",
    "type. mRNAsi is positively correlated with other core stem cell fac-tors: EZH2 ,OCT4 ,a n d SOX2 (Figure 2 Ba n d Table S2 ). Finally,\n",
    "Moonlight analysis of the oncogenic signatures from the Molecu-\n",
    "lar Signatures Database (MSigDB) further validated our gene-expression-based index and conﬁrmed engagement of MYC\n",
    "and EZH2, along with E2F3, MTOR, and SHH in driving oncogenic\n",
    "dedifferentiation ( Figure 2 C) (Colaprico et al., 2018 ).\n",
    "DNA Methylation-Based Stemness Index\n",
    "We deﬁned the mDNAsi using OCLR by combining (1) super-vised classiﬁcation between ESCs/iPSCs and their progenies,\n",
    "(2) stem cell signatures associated with pluripotency-speciﬁc\n",
    "genomic enhancer elements based on ChromHMM from Road-map, and (3) ELMER, which uses DNA methylation to identify\n",
    "enhancer elements and correlates their state with the expression\n",
    "of nearby genes. 219 CpG probes ( Figure S2 A) were selected in\n",
    "training OCLR using the Progenitor Cell Biology Consortium\n",
    "(PCBC) datasets. By selecting probes previously deﬁned to\n",
    "be active stemness-speciﬁc enhancers, we conﬁrmed the abilityof our approach to derive an mDNAsi. Since we focusedexclusively on hypomethylated, functionally important CpG\n",
    "probes associated with stem cells, we further explored cis-acti-\n",
    "vated genes.\n",
    "We scored each TCGA sample using the mDNAsi and used an\n",
    "external dataset to conﬁrm that stem cells had higher mDNAsi\n",
    "than differentiated samples ( Figure 1 B [left plot]). TCGA tumor\n",
    "types show different degrees of an inferred dedifferentiated\n",
    "phenotype ( Figure 1 C [right]). Within these, individual tumor sam-\n",
    "ples show variation for cancer stemness. As anticipated, TCGAsamples derived from the primary tumors show higher cancerstemness indices compared to non-tumor samples obtained\n",
    "from adjacent normal tissue of origin ( Figure S1 E [bottom]).\n",
    "Most of our selected probes fell within non-promoter ele-\n",
    "ments, yet the SOX2-OCT4 transcription factor binding motif is\n",
    "one of the most highly enriched signatures within these regions.\n",
    "The SOX2-OCT4 complex is a critical master regulator of plurip-otency and stemness and is highly enriched in tumor samples\n",
    "with high mDNAsi ( Figure 2 D).Correlations of mRNAsi and mDNAsi\n",
    "Since the inputs for mDNAsi and mRNAsi are not necessarily\n",
    "complementary, we explored stratiﬁcation of glioma samples\n",
    "by the epigenetically regulated mRNAsi (EREG-mRNAsi), asstemess index generated using a set of stemness-related epige-\n",
    "netically regulated genes. The EREG-mRNAsi, based on both\n",
    "RNA expression and epigenetics, elucidates the discrepancybetween mDNAsi and mRNAsi and shows a positive correlationwith both indices ( Figure S1 F). Both mRNAsi and mDNAsi show\n",
    "good correspondence for a majority of tumors ( Figures S1 F and\n",
    "S2B). We observed major discrepancies in the case of brain\n",
    "lower grade glioma (LGG), thyroid carcinoma (THCA), and thy-\n",
    "moma (THYM). For gliomas, mDNAsi is correlated positively\n",
    "with tumor pathology and clinical features, while mRNAsi showsa negative correlation. This result could arise from a high fre-\n",
    "quency of isocitrate dehydrogenase mutations ( IDH1/2 ) muta-\n",
    "tions and resulting DNA hypermethylation.\n",
    "Stemness Index Can Stratify Recognized\n",
    "Undifferentiated CancersWe examined breast invasive carcinoma (BRCA), acute myeloid\n",
    "leukemia (AML), and gliomas to study if the mRNAsi/mDNAsi\n",
    "predict stemness in poorly differentiated tumors. In BRCA, wefound a strong association between the stemness index and\n",
    "known clinical and molecular features ( Figure 3 A [left]). The\n",
    "mRNAsi was highest in the basal subtype, known to exhibitan aggressive phenotype associated with an undifferentiatedstate. BRCA samples with high mRNAsi were more likely to\n",
    "be estrogen receptor (ER)-negative and enriched for FAT3\n",
    "andTP53 mutations. We noted that high mRNAsi was associ-\n",
    "ated with higher protein expression of FOXM1, CYCLINB1,\n",
    "and MSH6, as well as higher miRNA-200 family expression\n",
    "(Figure 3 A [right]). Invasive lobular type of BRCA (ILC), charac-\n",
    "terized by better prognosis in comparison to invasive ductal\n",
    "carcinoma (IDC), has a lower mRNAsi ( Figure 3 A [right]). We\n",
    "also applied our indices to non-TCGA BRCA samples ( Rey-\n",
    "ngold et al., 2014 ) and found a similar correlation between\n",
    "mRNAsi and mDNAsi in those samples. Moreover, mRNAsi\n",
    "also stratiﬁed BRCA samples with distinct histology in this da-taset ( Figure S1 B). Using datasets with estimated tumor cell\n",
    "type composition provided by the epigenetic deconvolution\n",
    "method ( Onuchic et al., 2016 ), we found that both mRNAsi\n",
    "and mDNAsi were more highly correlated with malignantepithelial cells than with normal epithelial cells, suggesting\n",
    "that our indices identify distinct cancerous epithelial cell popu-\n",
    "lations characterized by different features or degrees of stem-ness ( Figure S1 D).\n",
    "We found an association between the mRNAsi, RNA expres-\n",
    "sion subtypes previously deﬁned by TCGA, and the French-American-British (FAB) classiﬁcation of AML ( Figure 3 B). The\n",
    "mRNAsi showed the strongest correlation with the stage of\n",
    "myeloid differentiation of the AML samples. FAB subtypes M0\n",
    "(undifferentiated), M1 (with minimal maturation), and M2 (withmaturation) were characterized by high mRNAsi. In contrast,\n",
    "the M3 well-matured promyelocytic subtype, which is associ-\n",
    "ated with benign chromosomal abnormalities and favorableclinical outcome, had low mRNAsi ( Figure 3 B [right upper]).\n",
    "High mRNAsi was associated with higher expression of\n",
    "342 Cell173, 338–354, April 5, 2018A\n",
    "B\n",
    "C\n",
    "Figure 3. Molecular and Clinical Features Associated with Stemness in Breast Cancer, Acute Myeloid Leukemia, and Gliomas\n",
    "(A) (Left) An overview of the association between known molecular and biological processes and stemness in BRCA. Columns represent samples sorted by\n",
    "mRNAsi from low to high (top row). Rows represent molecular and biological processes associated with mRNAsi. Rows named ‘‘EDec CEp 2 and 4’’ representestimated cell type proportions. (Top right) Boxplots of mRNAsi in individual samples, stratiﬁed by molecular subtype and histology. (Bottom right ) Correlation of\n",
    "mRNAsi and representative protein expression and microRNA.(B) Similar to (A), association of mRNAsi in AML. (Top right) mRNAsi by mRNA-based molecular subtype and by FAB classiﬁcation. (Bottom right) Correla tion\n",
    "scores of mRNAsi and representative microRNA.(C) As in (A) and (B), GBM and LGG sorted by mDNAsi. (Top right) mDNAsi by molecular subtype and grade. (Bottom right) Correlation scores of mDNAsi andrepresentative protein expression and microRNA.All molecular and clinical features shown are statistically signiﬁcant. See also Figures S1 ,S3,S4, and S5.\n",
    "Cell173, 338–354, April 5, 2018 343AB\n",
    "C D\n",
    "E\n",
    "Figure 4. Selected Molecular and Clinical Features Associated with the Stemness Indices in TCGA Tumors\n",
    "(A) Association of molecular and clinical features with stemness in LUAD. (Top) mDNAsi by integrative molecular subtypes, smoking history, and muta tions of\n",
    "TP53 and SETD2. (Bottom) Correlation scores of mDNAsi and representative protein expression.(B) Stemness in HNSC. (Top) mDNAsi stratiﬁed by molecular subtypes and mutation of NSD1. (Bottom) Correlation scores of mDNAsi and representative pr otein\n",
    "and microRNA expression.(C) Stemness in LIHC. (Top) mRNAsi stratiﬁed by grade and mutations of TP53, CTNNB1, and AXIN1. (Bottom) Correlation scores of mRNAsi and representat ive\n",
    "protein and microRNA expression.\n",
    "(legend continued on next page)\n",
    "344 Cell173, 338–354, April 5, 2018miR-181c-3p, miR-22-3p, and miR-30b-3p ( Figure 3 B [right\n",
    "bottom]).\n",
    "We found a strong association between high mDNAsi, high\n",
    "pathologic grade, and recently published molecular subtypesof glioma ( Figure 3 C). mDNAsi was low in less-aggressive\n",
    "gliomas that are characterized by codel and glioma CpG methyl-\n",
    "ator phenotype (G-CIMP)-high features and was highest in highlyaggressive glioblastoma multiformes (GBMs) characterized byIDH mutations (G-CIMP low) and poor clinical outcome. Also,\n",
    "high mDNAsi is strongly associated with more aggressive clas-\n",
    "sical and mesenchymal subtypes of GBM, suggesting that itcan stratify tumors with distinct clinical outcomes. We also found\n",
    "that high mDNAsi was associated with mutations in NF1 and\n",
    "EGFR and infrequent mutations in IDH1 ,TP53 ,CIC, and ATRX\n",
    "(Figure 3 C [left]), with higher expression of ANNEXIN-A1 protein\n",
    "and lower expression of ANNEXIN-A7 and with expression of the\n",
    "miR-200 family ( Figure 3 C [right bottom]).\n",
    "We obtained similar results on non-TCGA glioma samples for\n",
    "which both mRNA expression and DNA methylation data were\n",
    "available ( Turcan et al., 2012 )(Figure S1 C). The negative correla-\n",
    "tion between mDNAsi and mRNAsi was restricted to LGG sam-ples—speciﬁcally, the IDH mutant subtypes (G-CIMP high and\n",
    "codel). IDH1 mutations are known to reduce cell differentiation,\n",
    "and high values of the mRNAsi in a subset of IDH mutant gliomasmight capture this phenomenon ( Lu et al., 2012 ).\n",
    "Pan-cancer Stemness Landscape\n",
    "Next, we tested the ability of our indices to identify previously un-\n",
    "explored features of cancer stemness across all TCGA tumors .\n",
    "First, we performed an enrichment analysis by sorting all TCGAsamples by stemness index for each tumor type and lookingfor associations with mutations and molecular and clinical fea-\n",
    "tures. The most salient associations of mRNAsi and mDNAsi\n",
    "are presented in Figure 4 , while the following results of the\n",
    "comprehensive analyses are shown in the supplementary mate-\n",
    "rial: associations with mutations ( Figure S3 ), associations with\n",
    "miRNA expression and protein abundance ( Figure S4 ), associa-\n",
    "tions with the tumor grading, and clinical outcome ( Figure S5 ).\n",
    "Correlations of mRNAsi and mDNAsi with Mutations in\n",
    "Genes, miRNA, and Expression of Proteins\n",
    "We found a strong association of mDNAsi with known molecular\n",
    "subtypes, with somatic mutations in SETD2 and TP53 genes,\n",
    "and with tobacco smoking status in lung adenocarcinoma(LUAD) ( Figures 4 A and S3). Current smokers and recently\n",
    "reformed smokers have higher mDNAsi than non-smokers or\n",
    "long-term reformed smokers. This suggests that the stemnessof LUAD tumors might be activated in response to environmental\n",
    "stimuli such as smoking and might inﬂuence the aggressiveness\n",
    "of the tumor. We also found an association between mDNAsi andhigher protein expression of CYCLINB1 and FOXM1, which is a\n",
    "pro-stemness transcription factor upstream of CYCLINB1 ( Fig-ure 4 A [lower plots]). FOXM1 has been associated with dediffer-\n",
    "entiation in pancreatic cancer cells ( Bao et al., 2011 ), as well as\n",
    "tumor proliferation in the kidney ( Xue et al., 2012 ) and ovarian\n",
    "(Wen et al., 2014 ) cancers. Our result suggests that it could be\n",
    "a driver of dedifferentiation and proliferation in breast and lung\n",
    "cancers. Stemness of LUAD tumors is also associated with lower\n",
    "expression of ANNEXIN-A1 ( Figure 4 A). ANNEXIN-A1 has been\n",
    "indicated as a differentiation marker in pancreatic ( Bai et al.,\n",
    "2004\n",
    ") and urothelial ( Kang et al., 2012 ) cancers; therefore, we\n",
    "suspect that the relationship between ANNEXIN and FOXM1\n",
    "expression and tumor differentiation may extend to other tumortypes ( Figure S4 C).\n",
    "Analyses of head and neck squamous cell carcinoma (HNSC)\n",
    "samples revealed that high indices are correlated with NSD1 mu-\n",
    "tation, E-cadherin protein expression, miR-200-3p, and previ-\n",
    "ously identiﬁed classical molecular subtypes ( Figure 4 B).NSD1\n",
    "mutation was recently linked in HNSC tumors to blockade ofcellular differentiation and promotion of oncogenesis ( Papillon-\n",
    "Cavanagh et al., 2017 ). Interestingly, miR-200 family members\n",
    "have been implicated in cancer initiation and metastasis, as\n",
    "well as self-renewal of healthy stem cells ( Gregory et al., 2008;\n",
    "Tellez et al., 2011 ). HNSC tumors with high mDNAsi have\n",
    "reduced programmed death ligand 1 (PD-L1) protein level\n",
    "(Figure 4 B).\n",
    "In liver hepatocellular carcinoma (LIHC) samples, we found an\n",
    "association between mRNAsi and high pathological grade ( Fig-\n",
    "ure 4 C). Negative associations between mRNAsi and the proba-\n",
    "bility of overall survival (OS) or progression-free survival (PFS)\n",
    "were detected ( Figures 4 E and S5C). In contrast to the majority\n",
    "of tumor types, LIHC samples with high mRNAsi have lowexpression of miR-200 family members ( Figure 4 C). The miR-\n",
    "200 family is known to be associated with progression of hepa-\n",
    "tocellular carcinoma ( Tsai et al., 2017; Wong et al., 2015 ), and the\n",
    "miR-200b-ZEB1 circuit has been suggested as a master regu-lator of stemness in these cancers ( Tsai et al., 2017 ). We found\n",
    "associations of mRNAsi with higher CYCLINB1 and ACC1 and\n",
    "with lower PD-L1 and ANNEXIN-A1 protein expression in LIHC(Figure 4 C). ACC1 was associated with pathomorphological\n",
    "markers of LIHC aggressiveness (vascular invasion and poor\n",
    "differentiation), and its upregulation was correlated with poorOS and disease recurrence in hepatocellular carcinoma pa-tients ( Wang et al., 2016 ). LIHC samples with high mRNAsi\n",
    "were associated with speciﬁc genomic alterations (e.g., TP53 ,\n",
    "CTNNB1 ,AXIN1 ).\n",
    "Detailed analyses of adrenocortical carcinoma (ACC) sam-\n",
    "ples revealed an association between high mRNAsi and\n",
    "deﬁned molecular subtypes ( Zheng et al., 2016 ), clinical stage,\n",
    "and mutations in PRKAR1A and TP53 genes ( Figure 4 D). We\n",
    "found a positive correlation between mRNAsi and adrenal\n",
    "differentiation score that is based on expression of 25 genesthat are important for adrenal function ( Zheng et al., 2016 )\n",
    "(Figure 4 D).\n",
    "(D) Stemness in ACC. (Top) mRNAsi stratiﬁed by mRNA molecular subtypes, clinical stage, and mutations of PRKAR1A and TP53. (Bottom) Correlation scor es of\n",
    "mRNAsi and adrenal differentiation score.(E) Cox proportional hazards model analysis. (Left) Progression-free survival. (Right) Overall survival. Hazard ratio greater than one denotes a t rend toward higher\n",
    "stemness index with worse outcome.See also Figures S3 ,S4, and S5.\n",
    "Cell173, 338–354, April 5, 2018 345Stemness Indices Are Correlated with Tumor Pathology\n",
    "and Predictive of Clinical Outcome\n",
    "We observed a positive correlation between tumor histology and\n",
    "pathology grading and both stemness indices for the majority ofthe TCGA cases ( Figures 3 A, 3C, 4C, 4D, S1B,S5A, and S5B).\n",
    "For mRNAsi, the most signiﬁcant correlations were found for\n",
    "BRCA (IDC and ILC), cervical squamous cell carcinoma and en-docervical adenocarcinoma (CESC), LIHC, pancreatic adeno-\n",
    "carcinoma (PAAD), and uterine corpus endometrial carcinoma\n",
    "(UCEC) ( Figure S5 A). Interestingly, mRNAsi shows low values\n",
    "in GBM and stomach adenocarcinoma (STAD). On the otherhand, mDNAsi strongly stratiﬁes glioma by the pathology grade,\n",
    "culminating with the highest value for GBM ( Figure S5 B). The\n",
    "reversed values of mDNAsi and mRNAsi in case of gliomaswere also evident in the clinical data analyses. An adverse asso-\n",
    "ciation between the mRNAsi and survival was detected ( Fig-\n",
    "ure 4 E), which was signiﬁcant for OS and PFS after adjusting\n",
    "for clinical factors ( Figures S5 C). In contrast, the mDNAsi had\n",
    "no signiﬁcant association with OS and PFS after correcting for\n",
    "A\n",
    "BC\n",
    "D\n",
    "EFigure 5. Analysis of Cancer Stemness in\n",
    "the Context of Metastatic State and Intratu-mor Heterogeneity\n",
    "(A) mRNAsi is higher in cancer metastases in\n",
    "comparison to the TCGA primary tumors.(B) mDNAsi is higher in recurrent glioma samplescompared to the primary glioma occurrence fromthe same patient. G-CIMP, glioma CpG methylatorphenotype.(C and D) Application of mRNAsi to a single-celltranscriptome of gliomas and breast cancer revealintratumor heterogeneity and various degrees ofthe oncogenic dedifferentiation.(E) Correlation of mRNAsi and mRNA expression ofCDH1 (epithelial marker) and CDH2 (mesenchymalmarker) in the cancer metastases.\n",
    "clinical factors. We found a positive corre-\n",
    "lation between previously published gli-oma subtypes and mDNAsi, suggesting\n",
    "that mDNAsi might recapitulate prog-\n",
    "nostic molecular subtypes ( Figure 3 C).\n",
    "The discordance between the mRNAsi\n",
    "and the mDNAsi for gliomas may be ex-\n",
    "plained in part by the dominant genomicalteration associated with the LGG tumortype. Roughly 80% of LGG tumors carry\n",
    "anIDH1/2 mutation and, as demon-\n",
    "strated by our group and others, confera genome-wide hypermethylator pheno-\n",
    "type (G-CIMP) ( Noushmehr et al., 2010;\n",
    "Turcan et al., 2012 ). Given that the\n",
    "mDNAsi is driven primarily by low methyl-\n",
    "ation levels associated with the stemness\n",
    "phenotype, the LGG tumors mightresemble non-stem like phenotypes,\n",
    "which are predominantly hypermethy-\n",
    "lated. The subgroup of G-CIMP with thelowest overall DNA methylation levels\n",
    "(G-CIMP low) is associated with the worst outcomes. Compared\n",
    "to G-CIMP-high tumors, G-CIMP-low tumors are known to be\n",
    "more proliferative, express cell-cycle-related genes, and havevarious stem-cell-like genomic features ( Ceccarelli et al., 2016 ).\n",
    "Cancer Stemness Indices Are Higher in Tumor\n",
    "Metastases and Reveal Intratumor Heterogeneity\n",
    "The TCGA samples are derived mostly from primary tumors,\n",
    "except for skin melanoma, for which tissues are mostly metasta-tic lymph nodes. We used the mRNAsi to interrogate the MET500dataset comprising expression proﬁles from 500 metastatic\n",
    "samples obtained from 22 different organs ( Robinson et al.,\n",
    "2017 ). In most cases, mRNAsi was signiﬁcantly higher in meta-\n",
    "static samples compared to primary TCGA tumors ( Figure 5 A).\n",
    "Prostate and pancreatic adenocarcinoma metastases had the\n",
    "most dedifferentiated phenotypes and are also more aggressiveand resistant to therapies in contrast to primary tumors. Weaker\n",
    "association with the mRNAsi was due to a small number of avail-\n",
    "able samples (n < 20). Interestingly, testicular germ cell tumors\n",
    "346 Cell173, 338–354, April 5, 2018(TGCTs) present the less differentiated phenotypes in primary\n",
    "tumors when compared to distant metastases. Primary TGCT\n",
    "tumor cells have high mRNAsi and may differentiate whenmetastasizing to distant organs. A similar trend was observed\n",
    "for STAD.\n",
    "Using another dataset, we found that mDNAsi was signiﬁ-\n",
    "cantly higher in glioma samples obtained at ﬁrst recurrence in\n",
    "contrast to primary gliomas ( Figure 5 B). Our results reveal signif-\n",
    "icant dedifferentiation of glioma cancer cells that contribute toglioma recurrence, which is frequently associated with poorprognosis and resistance to treatment ( de Souza et al., 2018 ).\n",
    "By taking advantage of single-cell transcriptome datasets, we\n",
    "used mRNAsi to probe tumor heterogeneity for oncogenic dedif-ferentiation of individual cancer cells ( Chung et al., 2017; Tirosh\n",
    "et al., 2016 ). We revealed high variation of stemness in the glioma\n",
    "and breast primary tumors. Individual glioma cells showedhigher variation of oncogenic dedifferentiation in comparison to\n",
    "breast cancer cells ( Figure 5 C). Single cells from metastasesA\n",
    "BFigure 6. Association of Stemness Index\n",
    "with Immune Microenvironment\n",
    "(A) mDNAsi and mRNAsi in the context of im-\n",
    "mune microenvironment. Each panel shows theSpearman correlation between the stemness in-dex and PD-L1 protein expression plotted againstSpearman correlation between the same stem-ness index and total leukocyte fraction, as esti-mated from DNA methylation data.(B) Highlight of tumor types that exhibit strongcorrelation between stemness and PD-L1expression or total leukocyte fraction.See also Figure S6 .\n",
    "had higher stemness index in breast can-\n",
    "cer (Figure 5 D). Interestingly, the negative\n",
    "correlation of EMT signature and stem-\n",
    "ness that we observed in TCGA primary\n",
    "tumors was also found in metastatic sam-ples ( Figure 5 E).\n",
    "Stemness Index Evaluated in the\n",
    "Context of Immune Response\n",
    "We found that for many tumors, higher\n",
    "stemness indices are associated with areduced leukocyte fraction and lowerPD-L1 expression ( Figure 6 A). For\n",
    "mDNAsi, the most distinctive negative\n",
    "correlations were found in the PanCan-12squamous cluster (LUSC, lung squamous\n",
    "cell carcinoma; HNSC; BLCA, bladder ur-\n",
    "othelial carcinoma) ( Hoadley et al., 2014 )\n",
    "and in GBM ( Figures 6 A [left panel] and\n",
    "S6B). For the mRNAsi, the highest nega-\n",
    "tive correlation values were seen inGBM/LGG, prostate adenocarcinoma\n",
    "(PRAD), LIHC, and uterine carcinosar-\n",
    "coma (UCS) tumors ( Figures 6 A [right\n",
    "panel] and S6A). We expect that such tu-\n",
    "mors will be less susceptible to immune\n",
    "checkpoint blockade treatments due to insufﬁcient immune\n",
    "cell inﬁltration or preexisting downregulation of the PD-L1pathway, which makes further inhibition ineffective. Our ﬁndings\n",
    "are consistent with previous reports showing a strong correlation\n",
    "between PD-L1 protein expression and inﬁltration of CD8+ cyto-toxic lymphocytes ( Zaretsky et al., 2016 ).\n",
    "We further explored correlations between stemness and im-\n",
    "mune microenvironment variables in the context of molecularsubtypes of tumors. Figure 6 B highlights several tumor types\n",
    "with the strongest (positive or negative) correlations. Except\n",
    "for kidney renal clear cell carcinoma (KIRC), the association\n",
    "between stemness and PD-L1 expression and leukocyte fractionis readily apparent from the increasing and decreasing trends\n",
    "of individual variables across the molecular subtypes. For\n",
    "example, we found mesenchymal tumors to have the highestPD-L1 expression levels, the most signiﬁcant leukocyte\n",
    "fractions, and the lowest mDNAsi compared to other HNSC\n",
    "subtypes, suggesting potential susceptibility to checkpoint\n",
    "Cell173, 338–354, April 5, 2018 347blockade inhibitors. The use of immunotherapy for HNSC tumors\n",
    "is under active investigation ( Economopoulou et al., 2016; Fuer-\n",
    "eder, 2016 ) with the recent FDA approval of pembrolizumab;\n",
    "however, whether the effectiveness of therapy is limited to spe-ciﬁc HNSC molecular subtypes is not clear from those reports.\n",
    "To assess other relationships between stemness and tumor\n",
    "microenvironment, we computed correlations between stem-ness indices and individual types of immune cells. By applyingCIBERSORT, we scored 22 immune cell types for their relative\n",
    "abundance in TCGA tumor samples. These cell types included\n",
    "natural killer (NK) cells, monocytes, macrophages, dendriticand mast cells, eosinophils, and neutrophils. We also obtained\n",
    "absolute estimates by scaling their relative abundance by overall\n",
    "leukocyte inﬁltration in each tumor as determined by ESTIMATEapplied to DNA methylation data. For any given TCGA sample,\n",
    "we calculated the correlation between mDNAsi/mRNAsi and\n",
    "the estimated fraction of individual immune cell types. In additionto individual immune subpopulation fractions, we considered thefunctional activation of distinct cells by measuring the difference\n",
    "between activated and resting fractions of NK cells, CD4+\n",
    "T cells, and macrophages. This approach was motivated byrecent observations that activation of peripheral CD4+ T cells\n",
    "triggered by immunotherapy is responsible for the speciﬁc killing\n",
    "of tumor cells ( Spitzer et al., 2017 ).\n",
    "Although the squamous cluster tumors had a negative cor-\n",
    "relation between stemness and the fraction of CD4+ T cell pop-\n",
    "ulations, the activation state of the CD4+ T cells was higher indedifferentiated tumors. This ﬁnding is consistent with our\n",
    "observation that PD-L1 protein expression is lower in these tu-\n",
    "mors, suggesting again that immune checkpoint blockade mightbe ineffective, and an additional mechanism of immune evasionmay be operative. The opposite trend is present in thymomas,\n",
    "where PD-L1 protein expression and the fraction of the CD4+\n",
    "T cell population are positively correlated with tumor dedif-ferentiation. Likewise, the activation state of CD4+ T cells is\n",
    "lower in dedifferentiated tumors, suggesting that they might be\n",
    "more susceptible to immunotherapy treatments ( Figures S6 A\n",
    "and S6B).\n",
    "Connectivity Map Analysis Identiﬁes Potential\n",
    "Compounds/Inhibitors Capable of Targeting the\n",
    "Stemness Signature\n",
    "We employed the Connectivity Map (CMap), a data-driven, sys-tematic approach for discovering associations among genes,\n",
    "chemicals, and biological conditions, to search for candidate\n",
    "compounds that might target pathways associated with stem-ness. We found enrichment for compounds associated with\n",
    "stemness in at least three cancer types ( Figure 7 A). 5 com-\n",
    "pounds are signiﬁcantly enriched in more than 10 cancer typesand have been reported to inhibit stemness-related tumorige-nicity: the dopamine receptor antagonists thioridazine and pro-\n",
    "chlorperazine ( Cheng et al., 2015; Lu et al., 2015; Dolma et al.,\n",
    "2016 ), the Wnt signaling inhibitor pyrvinium ( Xu et al., 2016 ),\n",
    "the HSP90 inhibitor tanespimycin, and the protein synthesis in-\n",
    "hibitor puromycin. Further, the telomerase inhibitor gossypol\n",
    "induced apoptosis and growth inhibition of cancer stem cells(CSCs) ( Volate et al., 2010 ), and histone deacetylase inhibitors\n",
    "such as trichostatin A (SAHA) reduced glioblastoma stem cellgrowth ( Chiao et al., 2013 ). According to our analysis, pyrvinium\n",
    "and puromycin could inhibit stemness in LUAD. We found\n",
    "several candidates with recognized anti-CSC activity for\n",
    "HNSC, including the aforementioned compounds. For LIHC,thioridazine, a prospective inhibitor of lung cancer stem cells\n",
    "(Yue et al., 2016 ), pyrvinium, puromycin, prochlorperazine, and\n",
    "others are potential compounds targeting undifferentiated tu-mors ( Figure 7 ).\n",
    "CMap mode-of-action (MoA) analysis of the 74 compounds\n",
    "revealed 56 mechanisms of action shared by the above com-\n",
    "pounds ( Figure 7 B and Table S4 B). Five compounds (ﬂuspirilene,\n",
    "pimozide, prochlorperazine, thioridazine, and triﬂuoperazine)\n",
    "shared the MoA of dopamine receptor antagonist. We observed\n",
    "that entinostat, trichostatin-a, and vorinostat shared MoA asHDAC inhibitors, and LY-294002, zaprinast, zardaverine share\n",
    "MoA as phosphodiesterase inhibitors.\n",
    "CMap target analysis revealed 212 distinct drug-target genes\n",
    "shared by the mentioned compounds ( Figure S7 andTable S4 C).\n",
    "Eight genes are targets of ﬁve different compounds—namely,\n",
    "DRD2 (8 drugs), HTR2A (7 drugs), HRH1 (6 drugs), ADRA1A\n",
    "(5 drugs), CALM1 (5 drugs), CHRM3 (5 drugs), HTR1A (5 drugs),and HTR2C (5 drugs).\n",
    "Recent polypharmacology studies suggest the need to design\n",
    "compounds that act on multiple genes or molecular pathways. Inthis study, we observed similar mechanisms of action among\n",
    "different compounds, suggesting that selective therapies\n",
    "can target the undifferentiated phenotypes for selected can-cer types.\n",
    "DISCUSSIONThis study is based on integrated analysis of cancer stemness in\n",
    "almost 12,000 primary human tumors of 33 different cancertypes. We interrogated TCGA data for mutations, DNA methyl-\n",
    "ation, expression of mRNA and miRNA, expression and post-\n",
    "translational modiﬁcation of proteins, histopathological grade,and clinical outcome. Applying CIBERSORT, we gained insight\n",
    "into the tumor microenvironment and composition of immune\n",
    "cell inﬁltrates. By applying a machine-learning algorithm to mo-lecular datasets from normal stem cells and their progeny, wedeveloped two different molecular metrics of stemness and\n",
    "then used them to assess epigenomic and transcriptomic fea-\n",
    "tures of TCGA cancers according to their grade of oncogenicdedifferentiation. Ultimately, the analyses led us to potentially\n",
    "actionable targets (and their MoAs) as candidates for possible\n",
    "differentiation therapy of solid tumors and metastases. Ourapproach could be applied to longitudinal study of samples\n",
    "from primary, recurrent, and metastatic cancers, and gene\n",
    "expression signatures identiﬁed in the tumor samples can be\n",
    "used to interrogate CMap to suggest actionable targets and in-hibitors for further analysis.\n",
    "To the best of our knowledge, this is the ﬁrst study in which\n",
    "molecular PCBC datasets comprised of stem cells and deﬁnedpopulations of their differentiated progeny have been leveraged\n",
    "to develop a classiﬁcation tool and machine-learning algorithm\n",
    "for analysis of a spectrum of human malignancies. A number ofcancer stemness scores, based on genes that are differentially\n",
    "expressed between CSCs and non-CSCs, have been published\n",
    "348 Cell173, 338–354, April 5, 2018A\n",
    "B\n",
    "Figure 7. Correlation of Cancer Stemness With Drug Resistance: Connectivity Map Analysis\n",
    "(A) Heatmap showing enrichment score (positive in blue, negative in red) of each compound from the CMap for each cancer type. Compounds are sorted from\n",
    "right to left by descending number of cancer type signiﬁcantly enriched.(B) Heatmap showing each compound (perturbagen) from the CMap that shares mechanisms of action (rows) and sorted by descending number of compoundwith shared mechanisms of action.See also Figure S7 andTables S3 andS4.\n",
    "Cell173, 338–354, April 5, 2018 349and are relevant to clinical outcomes in AML ( Eppert et al., 2011;\n",
    "Gentles et al., 2010; Ng et al., 2016 ). In those studies, gene sets\n",
    "enriched in ESCs (e.g., targets of NANOG, OCT4, SOX2, and\n",
    "c-MYC) were frequently overexpressed in poorly differentiatedtumors compared with well-differentiated ones. In breast\n",
    "cancers, those gene sets were associated with high-grade es-\n",
    "trogen receptor-negative, basal-like tumors and poor clinicaloutcome ( Ben-Porath et al., 2008 ). Another web-based tool,\n",
    "StemChecker, uses a curated set of 49 published stemness sig-\n",
    "natures deﬁned by gene expression, RNAi screens, transcription\n",
    "factor binding sites, text mining of the literature, and othercomputational approaches. But it has been tested only for\n",
    "pancreatic ductal adenocarcinoma. In that case, high expression\n",
    "of stemness genes correlated with poor prognosis ( Pinto et al.,\n",
    "2015 ). All previous studies were transcriptome-based and\n",
    "limited to a narrow set of genes and a small number of tu-\n",
    "mor types.\n",
    "In the present study, we found oncogenic dedifferentiation to\n",
    "be associated with several characteristics: mutations in genes\n",
    "that encode oncogenes and epigenetic modiﬁers, perturbations\n",
    "in speciﬁc mRNA/miRNA transcriptional networks, and deregu-lation of signaling pathways. Cancer stemness also appeared to\n",
    "involve core expression of MYC ,OCT4 ,SOX2 , and other genes\n",
    "involved in the regulatory circuitry that underlies normal andmalignant self-renewal potential. Our indices derived from\n",
    "mRNA expression and DNA methylation signatures reliably\n",
    "stratiﬁed tumors of known stemness phenotype. High mRNAsiwas associated with basal breast carcinomas but also Her2\n",
    "and lumB subtypes that are more aggressive than the hor-\n",
    "mone-dependent lumA group. In contrast, high mDNAsi wasstrongly associated with high-grade glioblastomas, poor OS,and PFS. The association between stemness signatures\n",
    "and adverse outcome for some tumor types, including gli-\n",
    "omas, may reﬂect malignant cell origins or the impact of theirmicroenvironment.\n",
    "Dedifferentiated cells can arise from different sources: from\n",
    "long-lived stem or progenitor cells that accumulate mutationsin oncogenic pathways or via dedifferentiation from non-stem\n",
    "cancer cells that convert to CSCs through deregulation of devel-\n",
    "opmental and/or non-developmental pathways. It is important todistinguish between the inherent stemness of CSCs and dedif-ferentiation induced by the tumor microenvironment. However,\n",
    "addressing that issue would require further validation beyond\n",
    "the scope of this study using other genomic datasets and/or lab-oratory experiments.\n",
    "Both stemness indices were lowest in normal cells, increased\n",
    "in primary tumors, and highest in metastases, consistent with theidea that tumor progression generally involves oncogenic dedif-\n",
    "ferentiation. Interestingly, we observed negative associations\n",
    "between stemness and EMT gene signatures. The relationshipbetween EMT and stemness remains a hotly debated topic,\n",
    "with several studies showing that EMT is necessarily associated\n",
    "with stemness ( Fabregat et al., 2016 ). However, most TCGA data\n",
    "are obtained from primary tumors, which exist in a pre-EMTstate, since EMT is strongly associated with tumor progression\n",
    "and with metastasis for many tumor types. Cancer cells in\n",
    "many primary solid tumors are basically epithelial regardless oftheir degrees of dedifferentiation, but some cells in such con-texts could acquire mesenchymal characteristics either by\n",
    "accumulating additional mutations or by undergoing epigenetic\n",
    "changes shaped by the tumor microenvironment. Those mesen-\n",
    "chymal cells can traverse the underlying tissue, enter the blood-stream, and seed distant organs, where they reacquire an\n",
    "epithelial phenotype to form metastatic tumors.\n",
    "We observed epithelial phenotypes and increased stemess\n",
    "index in molecular proﬁles of tumor-type-matched metastaticsamples in the MET500 cohort. This portends an association\n",
    "between dedifferentiation and spread of tumor cells to distant\n",
    "organs. The observation is further supported by high mDNAsiin samples from recurrent gliomas. It appears that tumor growth\n",
    "de novo , or at recurrence/metastasis, is associated with an\n",
    "increased stemness phenotype. Decreased mRNAsi levelsseen in TGCT suggest its possible differentiation as a germ cell\n",
    "tumor type induced by the microenvironment of liver or lung\n",
    "parenchyma, the organs it most often colonizes. Clinically, ingeneral, tumor progression is associated with greater aggres-siveness and resistance to therapy of almost all types.\n",
    "The mRNAsi was high for individual primary glioma and breast\n",
    "cancer cells. Interestingly, when applied to transcriptomic pro-ﬁles obtained from analysis of single cancer cells in bulk tumors,\n",
    "stemness indices revealed a high degree of intratumor heteroge-\n",
    "neity with respect to dedifferentiation phenotype. The heteroge-neity was greater in gliomas than in breast cancer, suggesting\n",
    "that intratumor environment, including stromal cells, hypoxia,\n",
    "and inﬁltration of immune cells, may play a role in shaping CSCniches and affect cancer cell developmental plasticity. Further\n",
    "molecular analyses of cancer cells stratiﬁed by the stemness\n",
    "phenotype would provide novel insights into the biology of pri-mary tumors.\n",
    "We found that for a number of tumor types (GBM, LUSC,\n",
    "HNSC, and BLCA), higher mDNAsi was associated with reduced\n",
    "leukocyte fraction and/or lower PD-L1 expression. Such tumorsare expected to be less susceptible to immune checkpoint\n",
    "blockade, due either to insufﬁcient immune cell inﬁltration of tu-\n",
    "mors or to inherent downregulation of the PD-L1 pathway. Bothfactors can render immune checkpoint immunotherapy ineffec-\n",
    "tive. The interaction between PD-L1 on cancer cells and PD1 re-\n",
    "ceptor on T cells helps cancer cells elude the immune system bypreventing activation of cytotoxic T cells in lymph nodes andsubsequent recruitment of other immune cell types to the tumor\n",
    "site ( Chen and Mellman, 2013 ). The presence of tumor-inﬁltrating\n",
    "lymphocytes and/or PD-L1 expression correlates with aggres-siveness in gastrointestinal stromal tumors ( Bertucci et al.,\n",
    "2015 ) and breast carcinomas ( Polo´nia et al., 2017 ). Common fea-\n",
    "tures shared between cancer cells and stem cells in the contextof the immune response are being highlighted by a growing num-\n",
    "ber of studies showing that vaccination with ESCs or iPSCs can\n",
    "raise speciﬁc immune response against cancer cells ( Kooreman\n",
    "et al., 2018 ). That ﬁnding may indicate that both cell populations\n",
    "use protein networks that, in tumors, result in uncontrolled self-\n",
    "renewal and dedifferentiated phenotypes histopathologicallydeﬁned by loss of architecture speciﬁc to the tissue of origin.We speculate that the indices described here may help\n",
    "predict the efﬁcacy of stem-cell-based immunotherapies and\n",
    "contribute to the identiﬁcation of patients who will respond tosuch therapies.\n",
    "350 Cell173, 338–354, April 5, 2018We interrogated CMap using the gene expression signatures\n",
    "from tumor samples with the highest and lowest mRNAsi levels.\n",
    "Surprisingly, perhaps, the CMap analysis, which is based on\n",
    "only a limited number of treated cell lines, very precisely selecteddrugs that have been shown to affect cancer stem cells with spec-\n",
    "iﬁcity. These translational analyses may ultimately pave the way\n",
    "for implementation of differentiation therapies for solid tumors.\n",
    "Here, we have also shown that cancer hallmarks can be ex-\n",
    "tracted from datasets on cells with deﬁned phenotypes and\n",
    "used to train machine-learning methods applicable to index mo-\n",
    "lecular proﬁles of cancer. Our mRNAsi and mDNAsi can be trans-lated into stemness scores (e.g., STEM50) that stratify tumors\n",
    "based on their dedifferentiation features, thus providing bio-\n",
    "markers for prediction of patient outcomes and response to dif-ferentiation therapies.\n",
    "By deﬁning new metrics of cancer stemness and using them to\n",
    "interrogate TCGA datasets, our results provide a comprehensivecharacterization of dedifferentiation as new and signiﬁcant hall-marks of cancer. The strengths of the approach are that it lever-\n",
    "ages features of dedifferentiated cells across a spectrum of\n",
    "tumor types that reﬂect tumor pathology and, in some cases,clinical outcome. This study also provides strategies for inte-\n",
    "grated analysis of cancer genomics based on machine-learning\n",
    "methods trained on molecular proﬁles obtained from cells withdeﬁned phenotypes. The ﬁndings based on those methods\n",
    "may advance the development of objective diagnostics tools\n",
    "for quantitating cancer stemness in clinical tumors, perhaps lead-ing eventually to new biomarkers that predict tumor recurrence,\n",
    "guide treatment selection, or improve responses to therapy.\n",
    "STAR+METHODS\n",
    "Detailed methods are provided in the online version of this paper\n",
    "and include the following:\n",
    "dKEY RESOURCES TABLE\n",
    "dCONTACT FOR REAGENT AND RESOURCE SHARING\n",
    "dEXPERIMENTAL MODEL AND SUBJECT DETAILS\n",
    "dMETHOD DETAILS\n",
    "BDNA Methylation Data\n",
    "BRNA Expression Data\n",
    "BStemness Index Derived Using OCLR\n",
    "BDNA Methylation Stemness Signatures\n",
    "BStemness versus Molecular and Clinical Features\n",
    "BStemness versus Clinical Predictors\n",
    "BCompounds Targeting with Cancer Stemness\n",
    "dQUANTIFICATION AND STATISTICAL ANALYSIS\n",
    "dDATA AND SOFTWARE AVAILABILITY\n",
    "SUPPLEMENTAL INFORMATION\n",
    "Supplemental Information includes seven ﬁgures and four tables and can be\n",
    "found with this article online at https://doi.org/10.1016/j.cell.2018.03.034 .\n",
    "ACKNOWLEDGMENTS\n",
    "We thank Marcin Cie /C19slik from Michigan Center for Translational Pathology at\n",
    "University of Michigan for providing the MTE500 dataset. This work was sup-ported by the following grants: NIH grants U54 HG003273, U54HG003067, U54 HG003079, U24 CA143799, U24 CA143835, U24\n",
    "CA143840, U24 CA143843, U24 CA143845, U24 CA143848, U24CA143858, U24 CA143866, U24 CA143867, U24 CA143882, U24CA143883, U24 CA144025, and P30 CA016672; NCI grants 5R01CA180778,3U24CA143858, 1U24CA210990, 5U54HG006097, 1U24CA210949, and1U24CA210950; NIGMS grant 5R01GM109031; the Henry Ford Cancer Insti-tute’s Early Career Investigator Award grant A20054 to T.M.M; Sao PauloResearch Foundation (FAPESP) grants 2014/02245-3 and 2016/01975-3 toT.M.M. and H.N.; FAPESP grants 2014/08321-3, 2015/07925-5, 2016/01389-7, 2016/10436-9, 2016/06488-3, 2016/12329-5, and 2016/15485-8,\n",
    "and Henry Ford Hospital grant A30935 to H.N.; Spanish Institute of Health Car-\n",
    "los III grant CP14/00229; Mary K. Chapman Foundation gift ‘‘Chapman Foun-dation Fund for Bioinformatics,’’ CPRIT grant RP13039, and the Michael & Su-san Dell Foundation grant ‘‘The Lorraine Dell Program in Bioinformatics’’ toJ.N.W.; and the Polish Science Foundation Welcome grant 2010/3-3 to M.W.\n",
    "AUTHOR CONTRIBUTIONS\n",
    "The Cancer Genome Atlas Research Network contributed collectively to this\n",
    "study. The contributions of other authors are as follows: epigenetic-derivedstemness index, T.M.M., P.W.L., and H.N.; mRNA-expression-derived stem-\n",
    "ness index, A.S.; methodology, T.M.M., A.S., and H.N.; integrative analysis,\n",
    "T.M.M., A.S., and H.N.; clinical analysis, T.B. and L.P; drug analysis, A.C.;data interpretation, T.M.M., A.S., J.H., B.K., H.H., A.J.G., A.C., L.M., A.J.L.,A.K., A.K.G., J.M.S., K.A.H., P.C., O.G., S.M., P.W.L., H.N., and M.W.; data cu-ration, L.O.; writing, T.M.M., A.S., J.W., B.K., J.H., A.C., J.N.W., H.N., andM.W.; visualization, T.M.M., A.S., H.N., and M.W.; overall concept and coordi-nation, T.M.M., A.S., P.W.L., H.N., and M.W.\n",
    "DECLARATION OF INTERESTS\n",
    "Michael Seiler, Peter G. Smith, Ping Zhu, Silvia Buonamici, and Lihua Yu are\n",
    "employees of H3 Biomedicine, Inc. Parts of this work are the subject of a\n",
    "patent application: WO2017040526 titled ‘‘Splice variants associated with neo-morphic sf3b1 mutants.’’ Shouyoung Peng, Anant A. Agrawal, James Palacino,and Teng Teng are employees of H3 Biomedicine, Inc. Andrew D. Cherniack,Ashton C. Berger, and Galen F. Gao receive research support from Bayer Phar-maceuticals. Gordon B. Mills serves on the External Scientiﬁc Review Board ofAstrazeneca. Anil Sood is on the Scientiﬁc Advisory Board for Kiyatec and is ashareholder in BioPath. Jonathan S. Serody receives funding from Merck, Inc.Kyle R. Covington is an employee of Castle Biosciences, Inc. Preethi H. Gunar-atne is founder, CSO, and shareholder of NextmiRNA Therapeutics. ChristinaYau is a part-time employee/consultant at NantOmics. Franz X. Schaub is anemployee and shareholder of SEngine Precision Medicine, Inc. Carla Grandoriis an employee, founder, and shareholder of SEngine Precision Medicine, Inc.Robert N. Eisenman is a member of the Scientiﬁc Advisory Boards and share-holder of Shenogen Pharma and Kronos Bio. Daniel J. Weisenberger is aconsultant for Zymo Research Corporation. Joshua M. Stuart is the founderof Five3 Genomics and shareholder of NantOmics. Marc T. Goodman receivesresearch support from Merck, Inc. Andrew J. Gentles is a consultant for Ci-\n",
    "bermed. Charles M. Perou is an equity stock holder, consultant, and Board\n",
    "of Directors member of BioClassiﬁer and GeneCentric Diagnostics and isalso listed as an inventor on patent applications on the Breast PAM50 andLung Cancer Subtyping assays. Matthew Meyerson receives research supportfrom Bayer Pharmaceuticals; is an equity holder in, consultant for, and Scien-tiﬁc Advisory Board chair for OrigiMed; and is an inventor of a patent for EGFRmutation diagnosis in lung cancer, licensed to LabCorp. Eduard Porta-Pardo isan inventor of a patent for domainXplorer. Han Liang is a shareholder and sci-entiﬁc advisor of Precision Scientiﬁc and Eagle Nebula. Da Yang is an inventoron a pending patent application describing the use of antisense oligonucleo-tides against speciﬁc lncRNA sequence as diagnostic and therapeutic tools.Yonghong Xiao was an employee and shareholder of TESARO, Inc. Bin Fengis an employee and shareholder of TESARO, Inc. Carter Van Waes receivedresearch funding for the study of IAP inhibitor ASTX660 through a CooperativeAgreement between NIDCD, NIH, and Astex Pharmaceuticals. Raunaq Malho-tra is an employee and shareholder of Seven Bridges, Inc. Peter W. Laird serveson the Scientiﬁc Advisory Board for AnchorDx. Joel Tepper is a consultant at\n",
    "Cell173, 338–354, April 5, 2018 351EMD Serono. Kenneth Wang serves on the Advisory Board for Boston Scienti-\n",
    "ﬁc, Microtech, and Olympus. Andrea Califano is a founder, shareholder, andadvisory board member of DarwinHealth, Inc. and a shareholder and advisoryboard member of Tempus, Inc. Toni K. Choueiri serves as needed on advisoryboards for Bristol-Myers Squibb, Merck, and Roche. Lawrence Kwong re-ceives research support from Array BioPharma. Sharon E. Plon is a memberof the Scientiﬁc Advisory Board for Baylor Genetics Laboratory. Beth Y. Karlanserves on the Advisory Board of Invitae.\n",
    "Received: September 5, 2017\n",
    "Revised: January 30, 2018Accepted: March 14, 2018Published: April 5, 2018\n",
    "REFERENCES\n",
    "Agarwal, S., Graepel, T., Herbrich, R., Har-Peled, S., and Roth, D. (2005).\n",
    "Generalization Bounds for the Area Under the ROC Curve. J. Mach. Learn.Res. 6, 393–425 .\n",
    "Bai, X.-F., Ni, X.-G., Zhao, P., Liu, S.-M., Wang, H.-X., Guo, B., Zhou, L.-P., Liu,\n",
    "F., Zhang, J.-S., Wang, K., et al. (2004). Overexpression of annexin 1 in pancre-atic cancer and its clinical signiﬁcance. World J. Gastroenterol. 10, 1466–1470 .\n",
    "Bao, B., Wang, Z., Ali, S., Kong, D., Banerjee, S., Ahmad, A., Li, Y., Azmi, A.S.,\n",
    "Miele, L., and Sarkar, F.H. (2011). Over-expression of FoxM1 leads to epithe-lial-mesenchymal transition and cancer stem cell phenotype in pancreaticcancer cells. J. Cell. Biochem. 112, 2296–2306 .\n",
    "Ben-Porath, I., Thomson, M.W., Carey, V.J., Ge, R., Bell, G.W., Regev, A., and\n",
    "Weinberg, R.A. (2008). An embryonic stem cell-like gene expression signaturein poorly differentiated aggressive human tumors. Nat. Genet. 40, 499–507 .\n",
    "Benjamini, Y., and Hochberg, Y. (1995). Controlling the False Discovery Rate:\n",
    "A Practical and Powerful Approach to Multiple Testing. J. R. Stat. Soc. Ser. B57, 289–300 .\n",
    "Bertucci, F., Finetti, P., Mamessier, E., Pantaleo, M.A., Astolﬁ, A., Ostrowski,\n",
    "J., and Birnbaum, D. (2015). PDL1 expression is an independent prognosticfactor in localized GIST. OncoImmunology 4, e1002729 .\n",
    "Bradner, J.E., Hnisz, D., and Young, R.A. (2017). Transcriptional addiction in\n",
    "cancer. Cell 168, 629–643 .\n",
    "Ceccarelli, M., Barthel, F.P., Malta, T.M., Sabedot, T.S., Salama, S.R., Murray,\n",
    "B.A., Morozova, O., Newton, Y., Radenbaugh, A., Pagnotta, S.M., et al.; TCGAResearch Network (2016). Molecular proﬁling reveals biologically discrete sub-sets and pathways of progression in diffuse glioma. Cell 164, 550–563 .\n",
    "Chen, D.S., and Mellman, I. (2013). Oncology meets immunology: the cancer-\n",
    "immunity cycle. Immunity 39, 1–10 .\n",
    "Cheng, H.W., Liang, Y.H., Kuo, Y.L., Chuu, C.P., Lin, C.Y., Lee, M.H., Wu, A.T.,\n",
    "Yeh, C.T., Chen, E.I., Whang-Peng, J., et al. (2015). Identiﬁcation of thiorida-zine, an antipsychotic drug, as an antiglioblastoma and anticancer stem cellagent using public gene expression data. Cell Death Dis 6, e1753 .\n",
    "Chiao, M.T., Cheng, W.Y., Yang, Y.C., Shen, C.C., and Ko, J.L. (2013). Suber-\n",
    "oylanilide hydroxamic acid (SAHA) causes tumor growth slowdown and trig-\n",
    "gers autophagy in glioblastoma stem cells. Autophagy 9, 1509–1526 .\n",
    "Chung, W., Eum, H.H., Lee, H.-O., Lee, K.-M., Lee, H.-B., Kim, K.-T., Ryu, H.S.,\n",
    "Kim, S., Lee, J.E., Park, Y.H., et al. (2017). Single-cell RNA-seq enablescomprehensive tumour and immune cell proﬁling in primary breast cancer.Nat. Commun. 8, 15081 .\n",
    "Colaprico, A., Silva, T.C., Olsen, C., Garofano, L., Cava, C., Garolini, D., Sabe-\n",
    "dot, T.S., Malta, T.M., Pagnotta, S.M., Castiglioni, I., et al. (2016). TCGAbio-links: an R/Bioconductor package for integrative analysis of TCGA data.Nucleic Acids Res. 44, e71 .\n",
    "Colaprico, A., Olsen, C., Cava, C., Terkelsen, T., Silva, T.C., Olsen, A., Cantini,\n",
    "L., Bertoli, G., Zinovyev, A., Barillot, E., et al. (2018). Moonlight: a tool forbiological interpretation and driver genes discovery. bioRxiv, doi 10.1101/265322.\n",
    "Daily, K., Ho Sui, S.J., Schriml, L.M., Dexheimer, P.J., Salomonis, N., Schroll,\n",
    "R., Bush, S., Keddache, M., Mayhew, C., Lotia, S., et al. (2017). Molecular,phenotypic, and sample-associated data to describe pluripotent stem cell\n",
    "lines and derivatives. Sci. Data 4, 170030 .\n",
    "Davis, S., Bilke, S., Triche Jr, T., and Bootwalla, M. (2015). methylumi: Handle\n",
    "Illumina methylation data. R Package Version 2.18.0.\n",
    "de Souza, C.F., Sabedot, T.S., Malta, T.M., Stetson, L., Morozova, O., Soko-\n",
    "lov, A., Laird, P.W., Wiznerowicz, M., Iavarone, A., Snyder, J., et al. (2018).Distinct DNA Methylation Shift in a Subset of Glioma CpG Island MethylatorPhenotype (G-CIMP) during Tumor Recurrence. Cell Rep. Published online\n",
    "April 10, 2018. https://doi.org/10.1016/j.celrep.2018.03.107 .\n",
    "Dolma, S., Selvadurai, H.J., Lan, X., Lee, L., Kushida, M., Voisin, V., Whet-\n",
    "stone, H., So, M., Aviv, T., Park, N., et al. (2016). Inhibition of dopamine recep-tor D4 impedes autophagic ﬂux, proliferation, and survival of glioblastomastem cells. Cancer Cell 29, 859–873 .\n",
    "Economopoulou, P., Perisanidis, C., Giotakis, E.I., and Psyrri, A. (2016). The\n",
    "emerging role of immunotherapy in head and neck squamous cell carcinoma(HNSCC): anti-tumor immunity and clinical applications. Ann. Transl. Med.4, 173 .\n",
    "Eppert, K., Takenaka, K., Lechman, E.R., Waldron, L., Nilsson, B., van Galen,\n",
    "P., Metzeler, K.H., Poeppl, A., Ling, V., Beyene, J., et al. (2011). Stem cell geneexpression programs inﬂuence clinical outcome in human leukemia. Nat. Med.17, 1086–1093 .\n",
    "Fabregat, I., Malfettone, A., and Soukupova, J. (2016). New Insights into the\n",
    "Crossroads between EMT and Stemness in the Context of Cancer. J. Clin.\n",
    "Med. 5.\n",
    "Friedmann-Morvinski, D., and Verma, I.M. (2014). Dedifferentiation and re-\n",
    "programming: origins of cancer stem cells. EMBO Rep. 15, 244–253 .\n",
    "Fuereder, T. (2016). Immunotherapy for head and neck squamous cell carci-\n",
    "noma. Memo 9, 66–69 .\n",
    "Ge, Y., Gomez, N.C., Adam, R.C., Nikolova, M., Yang, H., Verma, A., Lu,\n",
    "C.P.-J., Polak, L., Yuan, S., Elemento, O., and Fuchs, E. (2017). Stem cell line-age inﬁdelity drives wound repair and cancer. Cell 169, 636–650 .\n",
    "Gentles, A.J., Plevritis, S.K., Majeti, R., and Alizadeh, A.A. (2010). Association\n",
    "of a leukemic stem cell gene expression signature with clinical outcomes inacute myeloid leukemia. JAMA 304, 2706–2715 .\n",
    "Gentles, A.J., Newman, A.M., Liu, C.L., Bratman, S.V., Feng, W., Kim, D., Nair,\n",
    "V.S., Xu, Y., Khuong, A., Hoang, C.D., et al. (2015). The prognostic landscape\n",
    "of genes and inﬁltrating immune cells across human cancers. Nat. Med. 21,\n",
    "938–945 .\n",
    "Gevaert, O., Villalobos, V., Sikic, B.I., and Plevritis, S.K. (2013). Identiﬁcation of\n",
    "ovarian cancer driver genes by using module network integration of multi-omics data. Interface Focus 3, 20130013 .\n",
    "Gingold, J., Zhou, R., Lemischka, I.R., and Lee, D.-F. (2016). Modeling Cancer\n",
    "with Pluripotent Stem Cells. Trends Cancer 2, 485–494 .\n",
    "Gonzalez, D.M., and Medici, D. (2014). Signaling mechanisms of the epithelial-\n",
    "mesenchymal transition. Sci. Signal. 7, re8 .\n",
    "Gregory, P.A., Bert, A.G., Paterson, E.L., Barry, S.C., Tsykin, A., Farshid, G.,\n",
    "Vadas, M.A., Khew-Goodall, Y., and Goodall, G.J. (2008). The miR-200 familyand miR-205 regulate epithelial to mesenchymal transition by targeting ZEB1and SIP1. Nat. Cell Biol. 10, 593–601 .\n",
    "Hanahan, D., and Weinberg, R.A. (2011). Hallmarks of cancer: the next gener-\n",
    "ation. Cell 144, 646–674 .\n",
    "Hoadley, K.A., Yau, C., Wolf, D.M., Cherniack, A.D., Tamborero, D., Ng, S.,\n",
    "Leiserson, M.D.M., Niu, B., McLellan, M.D., Uzunangelov, V., et al.; CancerGenome Atlas Research Network (2014). Multiplatform analysis of 12 cancertypes reveals molecular classiﬁcation within and across tissues of origin.Cell158, 929–944 .\n",
    "Ivanova, N., Dobrin, R., Lu, R., Kotenko, I., Levorse, J., DeCoste, C., Schafer,\n",
    "X., Lun, Y., and Lemischka, I.R. (2006). Dissecting self-renewal in stem cellswith RNA interference. Nature 442, 533–538 .\n",
    "Kang, W.-Y., Chen, W.-T., Huang, Y.-C., Su, Y.-C., and Chai, C.-Y. (2012).\n",
    "Overexpression of annexin 1 in the development and differentiation of urothe-lial carcinoma. Kaohsiung J. Med. Sci. 28, 145–150 .\n",
    "352 Cell173, 338–354, April 5, 2018Kim, J., and Orkin, S.H. (2011). Embryonic stem cell-speciﬁc signatures in can-\n",
    "cer: insights into genomic regulatory networks and implications for medicine.Genome Med. 3,7 5.\n",
    "Kim, J., Woo, A.J., Chu, J., Snow, J.W., Fujiwara, Y., Kim, C.G., Cantor, A.B.,\n",
    "and Orkin, S.H. (2010). A Myc network accounts for similarities between em-bryonic stem and cancer cell transcription programs. Cell 143, 313–324 .\n",
    "Kooreman, N.G., Kim, Y., de Almeida, P.E., Termglinchan, V., Diecke, S.,\n",
    "Shao, N.-Y., Wei, T.-T., Yi, H., Dey, D., Nelakanti, R., et al. (2018). AutologousiPSC-Based Vaccines Elicit Anti-tumor Responses In Vivo. Cell Stem Cell.Published online February 8, 2018. https://doi.org/10.1016/j.stem.2018.\n",
    "01.016 .\n",
    "Kundaje, A., Meuleman, W., Ernst, J., Bilenky, M., Yen, A., Heravi-Moussavi,\n",
    "A., Kheradpour, P., Zhang, Z., Wang, J., Ziller, M.J., et al.; Roadmap Epige-\n",
    "nomics Consortium (2015). Integrative analysis of 111 reference human epige-nomes. Nature 518, 317–330 .\n",
    "Lamb, J., Crawford, E.D., Peck, D., Modell, J.W., Blat, I.C., Wrobel, M.J.,\n",
    "Lerner, J., Brunet, J.-P., Subramanian, A., Ross, K.N., et al. (2006). The Con-nectivity Map: using gene-expression signatures to connect small molecules,genes, and disease. Science 313, 1929–1935 .\n",
    "Lu, C., Ward, P.S., Kapoor, G.S., Rohle, D., Turcan, S., Abdel-Wahab, O., Ed-\n",
    "wards, C.R., Khanin, R., Figueroa, M.E., Melnick, A., et al. (2012). IDH mutationimpairs histone demethylation and results in a block to cell differentiation.Nature 483, 474–478 .\n",
    "Lu, M., Li, J., Luo, Z., Zhang, S., Xue, S., Wang, K., Shi, Y., Zhang, C., Chen, H.,\n",
    "and Li, Z. (2015). Roles of dopamine receptors and their antagonist thioridazinein hepatoma metastasis. Onco. Targets Ther 8, 1543–1552 .\n",
    "Lyssiotis, C.A., and Kimmelman, A.C. (2017). Metabolic interactions in the tu-\n",
    "mor microenvironment. Trends Cell Biol. 27, 863–875 .\n",
    "Mak, M.P., Tong, P., Diao, L., Cardnell, R.J., Gibbons, D.L., William, W.N.,\n",
    "Skoulidis, F., Parra, E.R., Rodriguez-Canales, J., Wistuba, I.I., et al. (2016). APatient-Derived, Pan-Cancer EMT Signature Identiﬁes Global Molecular Alter-ations and Immune Target Enrichment Following Epithelial-to-MesenchymalTransition. Clin. Cancer Res. 22, 609–620 .\n",
    "Mathur, D., Danford, T.W., Boyer, L.A., Young, R.A., Gifford, D.K., and Jae-\n",
    "nisch, R. (2008). Analysis of the mouse embryonic stem cell regulatory net-\n",
    "works obtained by ChIP-chip and ChIP-PET. Genome Biol. 9, R126 .\n",
    "Nazor, K.L., Altun, G., Lynch, C., Tran, H., Harness, J.V., Slavin, I., Garitaonan-\n",
    "dia, I., Mu ¨ller, F.-J., Wang, Y.-C., Boscolo, F.S., et al. (2012). Recurrent\n",
    "variations in DNA methylation in human pluripotent stem cells and their differ-entiated derivatives. Cell Stem Cell 10, 620–634 .\n",
    "Ng, S.W.K., Mitchell, A., Kennedy, J.A., Chen, W.C., McLeod, J., Ibrahimova,\n",
    "N., Arruda, A., Popescu, A., Gupta, V., Schimmer, A.D., et al. (2016). A 17-genestemness score for rapid determination of risk in acute leukaemia. Nature 540,\n",
    "433–437 .\n",
    "Noushmehr, H., Weisenberger, D.J., Diefes, K., Phillips, H.S., Pujara, K., Ber-\n",
    "man, B.P., Pan, F., Pelloski, C.E., Sulman, E.P., Bhat, K.P., et al.; CancerGenome Atlas Research Network (2010). Identiﬁcation of a CpG island meth-ylator phenotype that deﬁnes a distinct subgroup of glioma. Cancer Cell 17,\n",
    "510–522 .\n",
    "Onuchic, V., Hartmaier, R.J., Boone, D.N., Samuels, M.L., Patel, R.Y., White,\n",
    "W.M., Garovic, V.D., Oesterreich, S., Roth, M.E., Lee, A.V., and Milosavljevic,A. (2016). Epigenomic Deconvolution of Breast Tumors Reveals MetabolicCoupling between Constituent Cell Types. Cell Rep. 17, 2075–2086 .\n",
    "Palmer, N.P., Schmid, P.R., Berger, B., and Kohane, I.S. (2012). A gene\n",
    "expression proﬁle of stem cell pluripotentiality and differentiation is conservedacross diverse solid and hematopoietic cancers. Genome Biol. 13, R71 .\n",
    "Papillon-Cavanagh, S., Lu, C., Gayden, T., Mikael, L.G., Bechet, D., Karam-\n",
    "boulas, C., Ailles, L., Karamchandani, J., Marchione, D.M., Garcia, B.A.,et al. (2017). Impaired H3K36 methylation deﬁnes a subset of head and necksquamous cell carcinomas. Nat. Genet. 49, 180–185 .\n",
    "Pinto, J.P., Kalathur, R.K., Oliveira, D.V., Barata, T., Machado, R.S.R.,\n",
    "Machado, S., Pacheco-Leyva, I., Duarte, I., and Futschik, M.E. (2015).StemChecker: a web-based tool to discover and explore stemness signatures\n",
    "in gene sets. Nucleic Acids Res. 43(W1), W72–W77 .\n",
    "Polo´nia, A., Pinto, R., Cameselle-Teijeiro, J.F., Schmitt, F.C., and Paredes, J.\n",
    "(2017). Prognostic value of stromal tumour inﬁltrating lymphocytes and pro-grammed cell death-ligand 1 expression in breast cancer. J. Clin. Pathol. 70,\n",
    "860–867\n",
    ".\n",
    "R Development Core Team (2017). R: A Language and Environment for Statis-\n",
    "tical Computing (R Foundation for Statistical Computing) .\n",
    "Reyngold, M., Turcan, S., Giri, D., Kannan, K., Walsh, L.A., Viale, A., Drobnjak,\n",
    "M., Vahdat, L.T., Lee, W., and Chan, T.A. (2014). Remodeling of the methyl-ation landscape in breast cancer metastasis. PLoS ONE 9, e103896 .\n",
    "Robinson, D.R., Wu, Y.-M., Lonigro, R.J., Vats, P., Cobain, E., Everett, J., Cao,\n",
    "X., Rabban, E., Kumar-Sinha, C., Raymond, V., et al. (2017). Integrative clinicalgenomics of metastatic cancer. Nature 548, 297–303 .\n",
    "Royston, P., and Altman, D.G. (1994). Regression Using Fractional Polyno-\n",
    "mials of Continuous Covariates: Parsimonious Parametric Modelling. J. R.Stat. Soc. Ser. C. Appl. Stat 43, 429 .\n",
    "Salomonis, N., Dexheimer, P.J., Omberg, L., Schroll, R., Bush, S., Huo, J.,\n",
    "Schriml, L., Ho Sui, S., Keddache, M., Mayhew, C., et al. (2016). IntegratedGenomic Analysis of Diverse Induced Pluripotent Stem Cells from the Progen-itor Cell Biology Consortium. Stem Cell Reports 7, 110–125 .\n",
    "Sato, N., Sanjuan, I.M., Heke, M., Uchida, M., Naef, F., and Brivanlou, A.H.\n",
    "(2003). Molecular signature of human embryonic stem cells and its comparisonwith the mouse. Dev. Biol. 260, 404–413 .\n",
    "Sergushichev, A. (2016). An algorithm for fast preranked gene set enrichment\n",
    "analysis using cumulative statistic calculation. bioRxiv, doi 10.1101/060012.\n",
    "Shibue, T., and Weinberg, R.A. (2017). EMT, CSCs, and drug resistance: the\n",
    "mechanistic link and clinical implications. Nat. Rev. Clin. Oncol. 14, 611–629 .\n",
    "Silva, T.C., Colaprico, A., Olsen, C., Bontempi, G., Ceccarelli, M., Berman,\n",
    "B.P., and Noushmehr, H. (2017). TCGAbiolinksGUI: A graphical user interfaceto analyze cancer molecular and clinical data. bioRxiv, doi 10.1101/147496.\n",
    "Sokolov, A., Paull, E.O., and Stuart, J.M. (2016). One-class detection of cell\n",
    "states in tumor subtypes. Pac. Symp. Biocomput. 21, 405–416 .\n",
    "Spitzer, M.H., Carmi, Y., Reticker-Flynn, N.E., Kwek, S.S., Madhireddy, D.,\n",
    "Martins, M.M., Gherardini, P.F., Prestwood, T.R., Chabon, J., Bendall, S.C.,\n",
    "et al. (2017). Systemic immunity is required for effective cancer immuno-therapy. Cell 168, 487–502 .\n",
    "StataCorp (2013). Stata Statistical Software: Release 13 (College Station, TX:\n",
    "StataCorp LP) .\n",
    "Sturm, D., Witt, H., Hovestadt, V., Khuong-Quang, D.-A., Jones, D.T.W., Ko-\n",
    "nermann, C., Pfaff, E., To ¨njes, M., Sill, M., Bender, S., et al. (2012). Hotspot\n",
    "mutations in H3F3A and IDH1 deﬁne distinct epigenetic and biological sub-groups of glioblastoma. Cancer Cell 22, 425–437 .\n",
    "Subramanian, A., Tamayo, P., Mootha, V.K., Mukherjee, S., Ebert, B.L., Gil-\n",
    "lette, M.A., Paulovich, A., Pomeroy, S.L., Golub, T.R., Lander, E.S., and Me-sirov, J.P. (2005). Gene set enrichment analysis: a knowledge-based approachfor interpreting genome-wide expression proﬁles. Proc. Natl. Acad. Sci. USA102, 15545–15550 .\n",
    "Subramanian, A., Narayan, R., Corsello, S.M., Peck, D.D., Natoli, T.E., Lu, X.,\n",
    "Gould, J., Davis, J.F., Tubelli, A.A., Asiedu, J.K., et al. (2017). A next generationconnectivity map: L1000 platform and the ﬁrst 1,000,000 proﬁles. Cell 171,\n",
    "1437–1452 .\n",
    "Tellez, C.S., Juri, D.E., Do, K., Bernauer, A.M., Thomas, C.L., Damiani, L.A.,\n",
    "Tessema, M., Leng, S., and Belinsky, S.A. (2011). EMT and stem cell-like prop-erties associated with miR-205 and miR-200 epigenetic silencing are earlymanifestations during carcinogen-induced transformation of human lungepithelial cells. Cancer Res. 71, 3087–3097 .\n",
    "Therneau, T.M., and Grambsch, P.M. (2000). Estimating the survival and haz-\n",
    "ard functions. In Modeling Survival Data: Extending the Cox Model (New York,NY: Springer New York), pp. 7–37 .\n",
    "Tirosh, I., Venteicher, A.S., Hebert, C., Escalante, L.E., Patel, A.P., Yizhak, K.,\n",
    "Fisher, J.M., Rodman, C., Mount, C., Filbin, M.G., et al. (2016). Single-cell\n",
    "Cell173, 338–354, April 5, 2018 353RNA-seq supports a developmental hierarchy in human oligodendroglioma.\n",
    "Nature 539, 309–313 .\n",
    "Tomczak, K., Czerwi /C19nska, P., and Wiznerowicz, M. (2015). The Cancer\n",
    "Genome Atlas (TCGA): an immeasurable source of knowledge. Contemp. On-col. (Pozn.) 19(1A), A68–A77 .\n",
    "Tsai, S.-C., Lin, C.-C., Shih, T.-C., Tseng, R.-J., Yu, M.-C., Lin, Y.-J., and\n",
    "Hsieh, S.-Y. (2017). The miR-200b-ZEB1 circuit regulates diverse stemnessof human hepatocellular carcinoma. Mol. Carcinog. 56, 2035–2047 .\n",
    "Turcan, S., Rohle, D., Goenka, A., Walsh, L.A., Fang, F., Yilmaz, E., Campos,\n",
    "C., Fabius, A.W.M., Lu, C., Ward, P.S., et al. (2012). IDH1 mutation is sufﬁcientto establish the glioma hypermethylator phenotype. Nature 483, 479–483 .\n",
    "Venezia, T.A., Merchant, A.A., Ramos, C.A., Whitehouse, N.L., Young, A.S.,\n",
    "Shaw, C.A., and Goodell, M.A. (2004). Molecular signatures of proliferationand quiescence in hematopoietic stem cells. PLoS Biol. 2, e301 .\n",
    "Visvader, J.E., and Lindeman, G.J. (2012). Cancer stem cells: current status\n",
    "and evolving complexities. Cell Stem Cell 10, 717–728 .\n",
    "Volate, S.R., Kawsaki, B.T., Hurt, E.M., Milner, J.A., Kim, Y.S., White, J., and\n",
    "Farrar, W.L. (2010). Gossypol induces apoptosis by activating p53 in prostatecancer cells and prostate tumor-initiating cells. Mol. Cancer Ther 9, 461–470 .\n",
    "Wang, M.-D., Wu, H., Fu, G.-B., Zhang, H.-L., Zhou, X., Tang, L., Dong, L.-W.,\n",
    "Qin, C.-J., Huang, S., Zhao, L.-H., et al. (2016). Acetyl-coenzyme A carbox-ylase alpha promotion of glucose-mediated fatty acid synthesis enhancessurvival of hepatocellular carcinoma in mice and patients. Hepatology 63,\n",
    "1272–1286 .\n",
    "Wen, N., Wang, Y., Wen, L., Zhao, S.-H., Ai, Z.-H., Wang, Y., Wu, B., Lu, H.-X.,\n",
    "Yang, H., Liu, W.-C., and Li, Y. (2014). Overexpression of FOXM1 predicts poorprognosis and promotes cancer cell proliferation, migration and invasion inepithelial ovarian cancer. J. Transl. Med. 12, 134 .\n",
    "Wickham, H. (2009). ggplot2 (Springer Nature) .\n",
    "Wong, C.-M., Wei, L., Au, S.L.-K., Fan, D.N.-Y., Zhou, Y., Tsang, F.H.-C., Law,\n",
    "C.-T., Lee, J.M.-F., He, X., Shi, J., et al. (2015). MiR-200b/200c/429 subfamilynegatively regulates Rho/ROCK signaling pathway to suppress hepatocellular\n",
    "carcinoma metastasis. Oncotarget 6, 13658–13670 .\n",
    "Xu, L., Zhang, L., Hu, C., Liang, S., Fei, X., Yan, N., Zhang, Y., and Zhang, F.\n",
    "(2016). WNT pathway inhibitor pyrvinium pamoate inhibits the self-renewal\n",
    "and metastasis of breast cancer stem cells. Int. J. Oncol. 48, 1175–1186 .\n",
    "Xue, Y.J., Xiao, R.H., Long, D.Z., Wang, X.N., Zhang, G.N., Yuan, Y.H., Wu,\n",
    "G.Q., Yang, J., Wu, Y.T., Xu, H., et al. (2012). Overexpression of FoxM1 is asso-ciated with tumor progression in patients with clear cell renal cell carcinoma.J. Transl. Med 10, 200 .\n",
    "Yan, X., Ma, L., Yi, D., Yoon, J.G., Diercks, A., Foltz, G., Price, N.D., Hood, L.E.,\n",
    "and Tian, Q. (2011). A CD133-related gene expression signature identiﬁes an\n",
    "aggressive glioblastoma subtype with excessive mutations. Proc. Natl.Acad. Sci. USA 108, 1591–1596 .\n",
    "Yao, L., Shen, H., Laird, P.W., Farnham, P.J., and Berman, B.P. (2015). Infer-\n",
    "ring regulatory element landscapes and transcription factor networks fromcancer methylomes. Genome Biol. 16, 105 .\n",
    "Young, R.A. (2011). Control of the embryonic stem cell state. Cell 144,\n",
    "940–954 .\n",
    "Yue, H., Huang, D., Qin, L., Zheng, Z., Hua, L., Wang, G., Huang, J., and\n",
    "Huang, H. (2016). Targeting Lung Cancer Stem Cells with AntipsychologicalDrug Thioridazine. BioMed Res. Int. 2016 , 6709828 .\n",
    "Zaretsky, J.M., Garcia-Diaz, A., Shin, D.S., Escuin-Ordinas, H., Hugo, W.,\n",
    "Hu-Lieskovan, S., Torrejon, D.Y., Abril-Rodriguez, G., Sandoval, S., Barthly,\n",
    "L., et al. (2016). Mutations Associated with Acquired Resistance to PD-1Blockade in Melanoma. N. Engl. J. Med. 375, 819–829 .\n",
    "Zheng, S., Cherniack, A.D., Dewal, N., Mofﬁtt, R.A., Danilova, L., Murray, B.A.,\n",
    "Lerario, A.M., Else, T., Knijnenburg, T.A., Ciriello, G., et al.; Cancer GenomeAtlas Research Network (2016). Comprehensive Pan-Genomic Characteriza-tion of Adrenocortical Carcinoma. Cancer Cell 29, 723–736 .\n",
    "354 Cell173, 338–354, April 5, 2018STAR+METHODS\n",
    "KEY RESOURCES TABLE\n",
    "CONTACT FOR REAGENT AND RESOURCE SHARING\n",
    "Further information and requests for resources should be directed to and will be fulﬁlled by the Lead Contact, Maciej Wiznerowicz\n",
    "(maciej.wiznerowicz@iimo.pl ).\n",
    "EXPERIMENTAL MODEL AND SUBJECT DETAILS\n",
    "Clinical and molecular data were collected from the NIH Genomic Data Commons (GDC) of 11,392 participants from The Cancer\n",
    "Genome Atlas PanCancer Atlas cohort ( https://gdc.cancer.gov/about-data/publications/PanCanStemness-2018 ).REAGENT or RESOURCE SOURCE IDENTIFIER\n",
    "Software and Algorithms\n",
    "Workﬂow to reproduce the stemness index This paper https://bioinformaticsfmrp.github.io/PanCanStem_Web/\n",
    "CIBERSORT Gentles et al., 2015 https://precog.stanford.edu/\n",
    "R 3.3.1 R Development Core\n",
    "Team, 2017https://www.R-project.org\n",
    "ggplot2 (v2.2.1) Wickham, 2009 https://cran.r-project.org/web/packages/ggplot2/index.html\n",
    "gelnet (v1.2.1) Sokolov et al., 2016 https://cran.r-project.org/web/packages/gelnet/index.html\n",
    "GSEA Subramanian et al., 2005 https://software.broadinstitute.org/gsea/index.jsp\n",
    "TCGAbiolinks (v2.4.3) Colaprico et al., 2016 http://bioconductor.org/packages/release/bioc/html/\n",
    "TCGAbiolinks.html\n",
    "ELMER (v1.4.1) Yao et al., 2015 http://bioconductor.org/packages/ELMER/\n",
    "fgsea (v1.2.1) Sergushichev, 2016 http://bioconductor.org/packages/release/bioc/html/fgsea.html\n",
    "Methylumi (v2.20.0) Davis et al., 2015 http://bioconductor.org/packages/release/bioc/html/\n",
    "methylumi.html\n",
    "MoonlightR (v1.2.0) Colaprico et al., 2018 http://bioconductor.org/packages/release/bioc/html/\n",
    "MoonlightR.html\n",
    "Amaretto Gevaert et al., 2013 http://med.stanford.edu/gevaertlab/software.html\n",
    "STATA (v13) StataCorp, 2013 https://www.stata.com/\n",
    "Deposited Data\n",
    "TCGA data NIH Genomic Data\n",
    "Commons (GDC)https://gdc.cancer.gov/about-data/publications/\n",
    "PanCanStemness-2018\n",
    "Progenitor Cell Biology Consortium (PCBC) Daily et al., 2017;\n",
    "Salomonis et al., 2016https://www.synapse.org/#!Synapse:syn1773109/wiki/54962\n",
    "Chromatin State (ChromHMM) Kundaje et al., 2015 http://www.roadmapepigenomics.org\n",
    "Stem Cells Validation set Nazor et al., 2012 mRNA expression (GEO: GSE30652) and DNA methylation\n",
    "(GEO: GSE30654)\n",
    "Glioma validation set Sturm et al., 2012 mRNA expression (GEO: GSE36245) and DNA methylation\n",
    "(GEO: GSE36278)\n",
    "Glioma validation set Turcan et al., 2012 GEO: GSE30339\n",
    "BRCA validation set Reyngold et al., 2014 GEO: GSE59000\n",
    "Deconvolution of breast cancer (BRCA) Onuchic et al., 2016 http://genboree.org/theCommons/projects/edec\n",
    "MET500 - Metastatic solid tumors Robinson et al., 2017 Database of Genotypes and Phenotypes (dbGaP) accession\n",
    "number phs000673.v2.p1\n",
    "Gliomas Single Cell RNA expression Tirosh et al., 2016 GEO: GSE70630\n",
    "BRCA Single Cell RNA expression Chung et al., 2017 GEO: GSE75688\n",
    "Cell173, 338–354.e1–e5, April 5, 2018 e1METHOD DETAILS\n",
    "DNA Methylation Data\n",
    "A total of 9,627 PanCancer TCGA samples across 33 different tumor types were available for DNA methylation using the robust Illu-mina HumanMethylation 450 (HM450) platform. TCGA samples included primary (8,471), recurrent (41), and metastatic tumor (394)\n",
    "tissues and a set of 721 non-tumor tissues.\n",
    "Level 3 data were downloaded from TCGA Data Portal using TCGAbiolinks functions GDCquery, GDCdownload and GDCprepare\n",
    "importing into R ( http://www.r-project.org ) for further analysis ( Colaprico et al., 2016 ).\n",
    "DNA methylation level 3 data are b-values that were calculated from pre-processed raw data using the methylumi Bioconductor\n",
    "package ( Davis et al., 2015 ). Pre-processing steps included background correction, dye-bias normalization, and calculation of\n",
    "b-values and detection p values. b-values range from zero to one, with zero indicating no DNA methylation and one indicating com-\n",
    "plete DNA methylation. A detection p value compares the signal intensity difference between the analytical probes and a set of nega-\n",
    "tive control probes on the array. Any data point with a corresponding p value greater than 0.01 is deemed not statistically signiﬁcantly\n",
    "different from background and is thus masked as ‘‘NA’’ in TCGA level 3 data. The data levels and the ﬁles contained in each data levelpackage are on the NIH Genomic Data Commons (GDC).\n",
    "In addition to TCGA data, we used a dataset of 99 human stem/progenitor cells from the Progenitor Cell Biology Consortium\n",
    "(PCBC) ( https://www.synapse.org/#!Synapse:syn1773109 ) to deﬁne stem cell signatures ( Daily et al., 2017; Salomonis et al.,\n",
    "2016 ). PCBC samples were proﬁled using the Illumina HumanMethylation 450 (HM450) platform and consisted of 4 embryonic\n",
    "stem cells (ESC), 40 induced pluripotent stem cells (iPSC), 22 stem cell (SC)-derived embryoid bodies (EB), 11 SC-derived mesoderm\n",
    "day 5 (MESO, mesothelioma), 11 SC-derived ectoderm (ECTO), and 11 SC-derived deﬁnitive endoderm (DE). We downloaded raw\n",
    "IDAT ﬁles from PCBC Genomic Data Commons and processed the data according to the TCGA standard level 3 protocoldescribed above.\n",
    "RNA Expression Data\n",
    "PanCancer TCGA RNA sequence level 3 normalized data were downloaded from the GDC Data Portal using TCGAbiolinks functions\n",
    "GDCquery, GDCdownload and GDCprepare importing into R ( http://www.r-project.org ) for further analysis ( Colaprico et al., 2016 ).\n",
    "A total of 10,852 samples across 33 tumor types were available, including primary (9,702), recurrent (45) and metastatic tumor (395)tissues and a set of 710 non-tumor tissues.\n",
    "We also downloaded PCBC RNA sequence data from the PCBC Synapse Portal ( https://www.synapse.org/#!\n",
    "Synapse:syn1773109 ), consisting 16 ESC, 77 iPSC, 66 SC-derived EB, 29 SC-derived MESO, 29 SC-derived ECTO, and 36\n",
    "SC-derived DE ( Daily et al., 2017; Salomonis et al., 2016 ).\n",
    "Stemness Index Derived Using OCLR\n",
    "To calculate a stemness index (si) based on mRNA expression or DNA methylation, we built a predictive model using one-classlogistic regression (OCLR) ( Sokolov et al., 2016 ) on the pluripotent stem cell samples (ESC and iPSC) from the PCBC dataset\n",
    "(Daily et al., 2017; Salomonis et al., 2016 ).\n",
    "For mRNA expression-based signatures, to ensure compatibility with the TCGA PanCancer cohort, we ﬁrst mapped the gene\n",
    "names from Ensembl IDs to Human Genome Organization (HUGO), dropping any genes that had no such mapping. The resulting\n",
    "training matrix contained 12,945 mRNA expression values measured across all available PCBC samples. For DNA methylation-\n",
    "based signatures, we used each of the signatures (probe set) described below.\n",
    "We mean-centered the data, then applied OCLR to just the samples labeled SC (which included both ESC and iPSC). We chose to\n",
    "use the one-class framework because of its robustness in the absence of the a ‘‘negative’’ class. The PCBC data does not have data\n",
    "for fully differentiated cells, and progenitor cell types might exhibit some of the stemness signals.\n",
    "Once the signature is obtained, it can be applied to score new samples. For RNA expression data, we computed Spearman cor-\n",
    "relations between the model’s weight vector and the new sample’s expression proﬁle. We advocate for the use of Spearman corre-\n",
    "lation over the more traditional dot product operation because it is more robust with respect to potential cross-dataset batch effects\n",
    "that may arise. For DNA methylation data, which follow the beta distribution, the samples were scored using the standard applicationof a linear model: f(x) = w\n",
    "^Tx+b .\n",
    "We validated our approach using leave-one-out cross-validation by withholding each SC sample in turn. A separate signature was\n",
    "then trained on all other SC samples and used to score the withheld sample as well as all the non-SC samples. The performance wasmeasured using the area under the curve (AUC) metric, which can be interpreted as the probability that the model correctly ranks a\n",
    "positive sample above a negative ( Agarwal et al., 2005 ). In our cross-validation experiment, every withheld SC sample was scored\n",
    "higher than all the non-SC samples, yielding an overall AUC of 1.0.\n",
    "We performed additional validation of the stemness signature by applying it to an external dataset composed of pluripotent stem\n",
    "cells (ESC and iPSC), somatic cells (17 distinct tissue types and several primary cell lines of diverse origin), and hydatidiform mole\n",
    "samples ( Nazor et al., 2012 ). The mRNA expression data for the study were downloaded from GEO (GEO: GSE30652) as were DNA\n",
    "methylation data (GEO: GSE30654). We observed that all of the SC samples were correctly scored above all of the somatic samples\n",
    "e2Cell173, 338–354.e1–e5, April 5, 2018by both platforms ( Figure 1 B). This is particularly striking for mRNA expression, because mRNA expression in study by the Nazor et al.\n",
    "was measured using microarrays, whereas the signature was trained using RNA-seq data.\n",
    "Having validated the signature by using cross-validation and external SC data, we then applied it to score the TCGA PanCancer\n",
    "cohort using the same Spearman correlation (RNA expression) or linear model (DNA methylation) operators. The indices were sub-sequently mapped to the [0,1] range by using a linear transformation that subtracted the minimum and divided by the maximum. The\n",
    "mapping was done to assist with interpretation as well as integration with the stemness indices derived from other data platforms\n",
    "(i.e., DNA methylation and mRNA expression).\n",
    "Additionally, we downloaded independent, non-TCGA datasets of gliomas [( Sturm et al., 2012 ) (GEO: GSE36245, GEO:\n",
    "GSE36278) and ( Turcan et al., 2012 ) (GEO: GSE30339)] and BRCA samples ( Reyngold et al., 2014 ) (GSE59000) and applied our met-\n",
    "rics to measure the stemness in the validation data. For mRNA expression, the preprocessing consisted of mapping the Illumina\n",
    "probe IDs (Illumina HumanHT-12 V3.0 platform) to HUGO symbols, and then reducing the signature and the external dataset to acommon set of genes. We then computed the Spearman correlation between the signature and the external samples. For DNA\n",
    "methylation, we applied the linear model.\n",
    "DNA Methylation Stemness Signatures\n",
    "Due to the magnitude of the available DNA methylation platform Inﬁnium HumanMethylation450 (HM450), we deﬁned DNA methyl-\n",
    "ation-based stemness signatures as a reduced input to the OCLR machine learning algorithm. For the DNA methylation-based stem-ness indices, three signatures were utilized, each deﬁning a distinct, biologically relevant, molecular phenotype of stemness. First, weperformed a supervised analysis between human pluripotent stem cells (ESC and iPSC) and stem cell-derived progenitors (embryoid\n",
    "bodies [EB], mesoderm [MESO], ectoderm [ECTO], and deﬁnitive endoderm [DE]) ( bvalue mean difference < /C00.4 and false discov-\n",
    "ery rate [FDR] < 10e-22; bvalue mean difference > 0.3 and false discovery rate [FDR] < 10e-17).All ‘rs’ and ‘ch’ probes were removed\n",
    "prior to analyses. To eliminate somatic tissue-speciﬁc probes, we removed probes that were consistently methylated (standard de-\n",
    "viation bvalue > 0.05) in non-tumor adult tissues available through TCGA. This resulted in a set of 62 pluripotent cell-speciﬁc and\n",
    "differentially methylated regions, which was then used as input for the OCLR to determine the stemness index for each TCGA tumorsample, named ‘‘differentially methylated probes-based stemness index’’ (DMPsi). Interestingly, most of these probes (85%) were\n",
    "positioned within intergenic regions known as open seas ( Figure S2 A).\n",
    "Second, we deﬁned a stem cell signature associated with genomic enhancer elements. Enhancers have been shown to be a crit-\n",
    "ically relevant functional element for deﬁning gene target expression and chromatin organization. For this, we downloaded Chromatin\n",
    "State data (ChromHMM) from the NIH Roadmap Epigenomics Consortium ( http://www.roadmapepigenomics.org ), which deﬁned 18\n",
    "chromatin states (based on 6 different histone marks: H3K4me3, H3K27ac, H3K4me1, H3K36me3, H3K9me3, and H3K27me3)across 98 different cell types ( Kundaje et al., 2015 ). Brieﬂy, by using ChromHMM data we mapped the HM450 probes to the chro-\n",
    "matin states in each individual cell type; then we identiﬁed genomic regions corresponding to active enhancers that are speciﬁc to\n",
    "pluripotent stem cell states (ESC and iPSC), meaning that each region was deﬁned as active enhancers (according to their states:\n",
    "9-EnhA1 and 10-EnhA2 ( Kundaje et al., 2015 )) in all pluripotent stem cells (n = 9) whereas not enhancer (enhancer in less than\n",
    "25% of non-pluripotent stem cells (n = 89)) in non-pluripotent stem cells. We identiﬁed 82 DNA methylation probes of the HM450\n",
    "platform that mapped to enhancer elements and considered them to be a DNA methylation-based pluripotent stem cell enhancer\n",
    "signature, which was then used as input for the OCLR to evaluate stemness signatures for TCGA samples, named ‘‘enhancer-basedstemness index’’ (ENHsi) ( Figure S2 A).\n",
    "Third, we applied ELMER (Enhancer Linking by Methylation/Expression Relationships), an R/Bioconductor package ( Yao et al.,\n",
    "2015 ) that uses DNA methylation to identify enhancer elements and correlates enhancer state with expression of nearby genes to\n",
    "identify putative transcriptional targets. Using ELMER, we compared pluripotent stem cells (ESC and iPSC) to stem cell-derived pro-genitors (EB, MESO, ECTO, DE) from PCBC and identiﬁed 87 CpGs that were hypomethylated in the pluripotent state (ESC and iPSC)\n",
    "compared to stem cell-derived progenitors and that potentially regulate 103 genes. We conﬁrmed the importance of these probe-\n",
    "gene pair targets by identifying that the SOX2-OCT4 transcription factor binding motif was among the most highly enriched signa-tures within these elements (+/ /C0250 bp from the center). The SOX2-OCT4 complex is an important master regulator of pluripotency\n",
    "and stemness. We then derived a new set of signatures using the OCLR and deﬁned TCGA samples’ stemness as ‘‘epigenetically\n",
    "regulated stemness indices’’ for each molecular feature (RNA expression-based Epigenetically regulated-mRNAsi [EREG-mRNAsi]and DNA methylation-based [EREG-mDNAsi]).\n",
    "Because there was high concordance among the three DNA methylation-based indices (DMPsi, ENHsi, and EREG-mDNAsi)\n",
    "(not shown) and each contributes important and complementary biological relevance to stemness, we combined the three stemnesssignatures (total of 219 probes) and derived a comprehensive DNA methylation index, named mDNAsi ( Figure S2 A). The lists of\n",
    "probes and genes used to derive the stemenss indices are provided on the publication portal accompanying this publication\n",
    "(https://gdc.cancer.gov/about-data/publications/PanCanStemness-2018 ).\n",
    "Stemness versus Molecular and Clinical Features\n",
    "To evaluate the performance of our stemness indices across the entire TCGA cohort, we performed an enrichment analysis by sorting\n",
    "TCGA samples by stemness index for each tumor type and looked for associations with all available genomic features (by usingcomprehensive mutation data [MC3]), molecular features (previously published TCGA molecular subtypes available at TCGAbiolinks\n",
    "Cell173, 338–354.e1–e5, April 5, 2018 e3package ( http://bioconductor.org/packages/release/bioc/html/TCGAbiolinks.html )(Colaprico et al., 2016; Silva et al., 2017 ), through\n",
    "the function ‘‘PanCancerAtlas_subtypes(),’’ which provides full access to the curated matrix used for this study), and clinical features\n",
    "(more than 10,000 features). We used the fgsea R/Bioconductor package to compute the enrichment scores ( Sergushichev, 2016 ).\n",
    "Brieﬂy, for each tumor type we ranked the TCGA samples according to their stemness index (from -low to -high stemness index) andtested if any particular genomic/molecular/clinical feature was associated with either -low or -high stemness index in a non-random\n",
    "behavior. We performed 10,000 permutations for each parameter analyzed to calculated our enrichment score. We then normalized\n",
    "the enrichment scores to mean enrichment of random samples of the same size (NES - normalized enrichment score). Tables con-taining all the results can be accessed at https://gdc.cancer.gov/about-data/publications/PanCanStemness-2018 . In addition, an\n",
    "interactive portal with the results across all tumor samples/types versus mDNAsi and mRNAsi can be accessed at https://\n",
    "bioinformaticsfmrp.github.io/PanCanStem_Web/ where the user can search for any gene or molecular/clinical feature of interest.\n",
    "Stemness versus Clinical Predictors\n",
    "The associations between the three stemness indices and overall survival (OS) and progression free survival (PFS) in different tumors\n",
    "were evaluated in two stages. First, the proportional hazard (PH) model with the index as a single continuous covariate was used totest whether there was a statistically signiﬁcant effect on OS or PFS. Given that, for each outcome, the effects of the three indices\n",
    "were tested for 33 cancer types. The signiﬁcance level of the tests was adjusted for multiple testing to control the overall type I error\n",
    "probability at 5%. In the next stage, the cancer types for which at least one index showed a statistically signiﬁcant association witheither OS or PFS were analyzed in more detail by using a multivariable PH model that included relevant clinical factors. Moreover, themodel included a functional form of the index obtained by using degree-2 fractional polynomials ( Royston and Altman, 1994 ). The\n",
    "plausibility of the PH assumption was checked by using the test based on the scaled Schoenfeld residuals ( Therneau and Grambsch,\n",
    "2000 ). The analyses were conducted using STATA v13 software.\n",
    "To select the clinical factors for inclusion in the PH model used in the second stage of the OS/PFS analysis for selected cancer\n",
    "types, a detailed analysis of the association between the stemness indices and demographic and clinical features (such as sex,\n",
    "age, race, stage, grade, etc.) was carried out by using linear models. mRNAsi and EREG-mRNAsi were analyzed on the original scale,while mDNAsi was transformed logarithmically to make its distribution more symmetric. The ﬁt of the constructed models was as-\n",
    "sessed by using residual plots. The analyses were conducted using STATA v13 software.\n",
    "The screening of the association between the stemness indices and OS ( Figure 4 E) by using univariable proportional hazard (PH)\n",
    "models indicated a statistically signiﬁcant (using p values adjusted for multiple testing) effect of mRNAsi on OS for LGG (p < 0.0001)\n",
    "and STAD (p = 0.005) and on PFS for GBM (p = 0.04), LGG (p < 0.0001), LIHC (p = 0.05), STAD (p = 0.04), and UCEC (p = 0.03). For\n",
    "mDNAsi, an effect on OS was found for LGG (p < 0.0001) and on PFS for kidney renal papillary cell carcinoma (KIRP) (p = 0.04) andLGG (p < 0.0001). Finally, for EREG-mRNAsi, a statistically signiﬁcant effect on OS was found for ACC (p = 0.005), KIRC (p = 0.008),and LGG (p = 0.03), and on PFS for ACC (p = 0.03), LGG (p = 0.03), and UCEC (p = 0.04). In these selected cases, multivariable an-\n",
    "alyses were conducted (using STATA v13 software), which took into account the effect of clinical factors. The analyses conﬁrmed (by\n",
    "using unadjusted p values) the effect of mRNAsi on OS for STAD (p = 0.0001) and for GBM/LGG (p = 0.002) and the effect on PFS forGBM/LGG (p = 0.008) and LIHC (p = 0.002). For mDNAsi, the effect on PFS in KIRP was conﬁrmed (p = 0.0001), while for EREG-\n",
    "mRNAsi, the effect on PFS in UCEC was conﬁrmed (p = 0.05). These conﬁrmed results indicate that the indices have a potential\n",
    "role as novel, independent prognostic factors for the indicated tumor types.\n",
    "Compounds Targeting with Cancer Stemness\n",
    "To determine which target drugs might be useful against cancer stem cells, we used the Broad Institute’s Connectivity Map build 02(CM) ( Lamb et al., 2006 ), a public online tool ( https://portals.broadinstitute.org/cmap/ ) (with registration) that allows users to predict\n",
    "compounds that can activate or inhibit based on a gene expression signature.\n",
    "To further investigate about mechanism of actions (MoA) and drug-target we performed speciﬁc analysis within Connectivity Map\n",
    "tools ( https://clue.io/ )(Subramanian et al., 2017 ).\n",
    "Using Connectivity Map (Query) in May 2017 having data available from a collection of cell lines (MCF, PC3, HL60 and SKMEL5)\n",
    "and 164 compounds as small molecules perturbagens. We obtained 33 mRNA expression signatures (one for each cancer type) by\n",
    "applying a differential expression analysis to samples with high mRNAsi and low mRNAsi, using the function TCGAanalyze_DEA fromthe the R/Bioconductor package TCGAbiolinks version 2.5.9 ( Colaprico et al., 2016 ), carrying edgeR pipeline. The table with differ-\n",
    "entially expressed genes is reported as Table S3 . Due to a limitation of the Connectivity Map tool that matches gene symbol and\n",
    "HG-U133A probe set (eg 200800_s_at) GPL96 platform ID, we had to remove duplicate IDs after sorting by decreasing jlogFC j.\n",
    "We selected the top 1000 genes (500 upregulated and 500 downregulated) where the number of differentially expressed genes\n",
    "was enough or considering the aggregation of upregulated or downregulated genes.\n",
    "Connectivity MAP is a method similar to GSEA analysis and follows a 4 step approach: (i) looking for similarity between a query\n",
    "signature (diff.expr. genes) and expression proﬁles present in the dataset using pattern-matching strategy based on Kolmogorov-Smirnov test (ii) rank-ordering the list of genes according their diff.expr. relative to the control from the above expression proﬁles\n",
    "with signiﬁcantly similarity (iii) comparison of each rank-ordered list with a query signature to specify when upregulated query genes\n",
    "appear in the proximity of the top of the list or near the bottom (‘‘positive connectivity’’) or vice versa (‘‘negative connectivity’’) pro-ducing an Enrichment Score (ES) from /C01 to 1. (iv) All instances in the database are then ranked according to their connectivity\n",
    "e4Cell173, 338–354.e1–e5, April 5, 2018scores; those at the top are most strongly correlated to the query signature, and those at the bottom are most strongly anticorrelated.\n",
    "For each cancer type we obtained two tables that applied the Connectivity Map’s ﬁndings to stemness mRNA expression signa-\n",
    "tures, namely, ‘‘detailed results’’ and ‘‘permuted results.’’ We used the permuted results and ﬁlter (with p < 0.05), to identify an\n",
    "average of 74 compounds per tumor type that are predicted to repress or activate the stemness signature ( Table S4 A).\n",
    "Connectivity Map (CMap) was recently updated (September 2017) ( Subramanian et al., 2017 ), providing the end-users new func-\n",
    "tionalities and new graphical interface as web-server, previous registration ( https://clue.io/ ) allowing easily the extraction of drug-\n",
    "interaction knowledge using as input a signature of genes or compounds.\n",
    "The new interface ( https://clue.io/ ), provided 7 different analysis (query, touchstone, proteomics query, command, data library,\n",
    "repurposing, morpheus).\n",
    "In particular CMap Query it is a tool for perturbagens that give rise to similar (or opposing) expression signatures, for a technical\n",
    "limit, the CMap Query 2017 allows only to upload 150 genes max for upregulated genes and 150 genes for downregulated genes. Forthis reason we considered the results analyzed in May 2017 using 500 genes for up-downregulated genes.\n",
    "QUANTIFICATION AND STATISTICAL ANALYSISR version 3.3.1 was used for all statistical analyses, unless speciﬁed otherwise. The statistical details of all experiments are reported\n",
    "in the ﬁgure legends and ﬁgures, including statistical analysis performed, statistical signiﬁcance and exact n values.\n",
    "To identify differentially methylated DNA methylation probes, we used the Wilcoxon test followed by multiple testing using the Ben-\n",
    "jamini-Hochberg (BH) method to estimate false discovery rate ( Benjamini and Hochberg, 1995 ).\n",
    "To identify proteins and microRNAs differentially expressed between tumors with low versus high stemness index, we used a t test\n",
    "followed by multiple testing using BH.\n",
    "P values for the association between stemness index and continuous clinical data were also computed using a t test followed by\n",
    "multiple testing using BH.\n",
    "DATA AND SOFTWARE AVAILABILITY\n",
    "All data are available on the NIH Genomic Data Commons (GDC), https://gdc.cancer.gov/about-data/publications/\n",
    "PanCanStemness-2018 .\n",
    "The workﬂow to reproduce the stemness index, including downloading PCBC and TCGA PanCan33 datasets, training a stemness\n",
    "signature, and applying it to score TCGA samples can be accessed at https://bioinformaticsfmrp.github.io/PanCanStem_Web/ .\n",
    "An interactive portal with the results for enrichment of molecular and clinical features and Stemness Indices across all tumor sam-\n",
    "ples/types can be accessed at https://bioinformaticsfmrp.github.io/PanCanStem_Web/ where the user can search for any gene or\n",
    "molecular/clinical feature of interest.\n",
    "Cell173, 338–354.e1–e5, April 5, 2018 e5Supplemental Figures\n",
    "Figure S1. Validation of mRNAsi and mDNAsi, Related to Figures 1 and3\n",
    "(A) Comparison of mRNAsi and mDNAsi values, as scored by signatures learned from PCBC data by OCLR (x axis) and Random Forest (y axis).\n",
    "(B) Validation of mRNAsi in non-TCGA breast cancer (BRCA) samples ( Reyngold et al., 2014 ) to deﬁne stemness status. Stratiﬁcation of mRNAsi according to\n",
    "histology of TCGA samples (left) and of the validation cohort (right). IDC - invasive ductal carcinoma; ILC - invasive lobular carcinoma.(C) Validation of stemness indices (mDNAsi and mRNAsi) in non-TCGA glioma samples ( Sturm et al., 2012; Turcan et al., 2012 ) using our metric to deﬁne stemness\n",
    "status. The validation sets were previously classiﬁed into molecular subtypes of TCGA gliomas. Stratiﬁcation of stemness indices according to mole cular\n",
    "subtypes in the TCGA (left) and in the validation cohort (right).(D) Correlation of stemness indices and estimated cell composition derived from deconvolution of tumor epigenomic features in TCGA breast cancer sa mples\n",
    "(Onuchic et al., 2016 ).\n",
    "(legend continued on next page)(E) Stemness indices in tumor versus non-tumor samples across tissue type. Top plots show mRNAsi and bottom plots show mDNAsi. See list of Abbreviatio ns of\n",
    "the TCGA Tumor Types.(F) Correlation of stemness indices across TCGA tumor types. Tumor types on the x axis are sorted by low-to-high correlation between mDNAsi and mRNAsi . See\n",
    "list of Abbreviations of the TCGA Tumor Types. mRNAsi, RNA-based stemness index. mDNAsi, DNA methylation-based stemness index. EREG-mRNAsi,epigenetically regulated-mRNAsi.Figure S2. Validation of mRNAsi and mDNAsi and Correlation of mRNAsi with EMT, Related to Figures 1 and2\n",
    "(A) Heatmaps of DNA methylation levels of probe signature used to derive the mDNAsi in: Progenitor Cell Biology Consortium (PCBC) dataset (left), TCG A normal\n",
    "adjacent tissue (middle), and TCGA tumors (right). Columns represent samples. TCGA tumor samples are sorted from lowest to highest mDNAsi. Rows repr esent\n",
    "the DNA methylation probe signature (219 probes), which is a combination of three DNA methylation-based signatures: differentially methylated pro bes stemness\n",
    "index (DMPsi), enhancer stemness index (ENHsi), and epigenetically regulated-mDNAsi (EREG-mDNAsi) (see STAR Methods for detail). Annotation of probes are\n",
    "provided in the left of the heatmaps. First column (mDNAsi) indicates to which signature the probe belong. Second column (genomic location) indicate s the\n",
    "genomic location annotation of each probe from the UCSC genome browser. Chromatin State columns indicates the chromatin state of pluripotent stem ce lls\n",
    "(iPSC and ESC) from Roadmap (n = 9) ( Kundaje et al., 2015 ).\n",
    "(legend continued on next page)(B) Scatterplots showing correlation of mDNAsi (x axis) and mRNAsi (y axis) by tumor type. Samples are colored according to TCGA classiﬁcation in: pri mary,\n",
    "recurrent, metastatic, and non-tumors.(C) Correlation between mRNAsi and mRNA expression of epithelial-to-mesenchymal transition (EMT) markers. The markers are grouped according to wh ether\n",
    "they are associated with epithelial (pink) or mesenchymal (teal blue) phenotype.Figure S3. Mutation Enrichment Analysis in Correlation with the mRNAsi and mDNAsi, Related to Figures 3 and4\n",
    "(A) (Left Panel) Comparison of normalized enrichment scores (NES) in mDNAsi (x axis) and mRNAsi (y axis) computed by tumor type in the mutation enrichm ent\n",
    "analysis. Positive and negative NES entail enrichment and depletion of a given mutation being associated with a high stemness index, respectively. T he quadrants\n",
    "and colors represent genes with agreement (I) or disagreement (II) between the two distinct indices and being associated only with mDNAsi (III) or ass ociated only\n",
    "with mRNAsi (IV) between the two distinct indices (V, not signiﬁcant). (Right Panel) Stemness indices stratiﬁed by representative mutation status a nd tumor types.\n",
    "(B) Relationship of mutations in epigenetic modiﬁer genes and stemness indices mRNAsi (top) and mDNAsi (bottom). NES, normalized enrichment scores .Figure S4. Correlation of miRNA and Protein Expression against mRNAsi and mDNAsi, Related to Figures 3 and4.\n",
    "(A) (Left Panel) Comparison of miRNA expression associated with mDNAsi and mRNAsi. Log10-normalized FDR-adjusted P value is plotted for mDNAsi (x ax is)\n",
    "and mRNAsi (y axis) for each miRNA probe for each tumor type. If a particular miRNA is upregulated in samples with a high stemness index, the values aremultiplied by /C01. The quadrants and colors represent proteins with agreement or disagreement between the two distinct indices. (Right Panel) Correlation of\n",
    "stemness indices and representative miRNAs for selected tumor types.(B) Comparison of protein expression associated with mDNAsi and mRNAsi (left panel). Log10 (FDR-adjusted P value) is plotted for mDNAsi (x axis) and m RNAsi\n",
    "(y axis) for each protein for each tumor type. If the protein is upregulated in high stemness index, /C01 is multiplied to log10 providing positive values. The quadrants\n",
    "and colors represent proteins with agreement or disagreement between the two distinct indices. Right, correlation scores of stemness indices and re presentative\n",
    "proteins and tumor types.\n",
    "(C) Correlation of ANNEXIN1 and FOXM1 with mDNAsi (top) and mRNAsi (bottom) across tumor types.Figure S5. Tumor Pathology and Clinical Outcome in Correlation with the mRNAsi and mDNAsi, Related to Figures 3 and4.\n",
    "(A and B) Tumor pathology grade associations with mRNAsi (A) and mDNAsi (B) by TCGA tumors types.\n",
    "(C) OS and PFS curves for four (quartile-based) categories of the relevant indices. The ﬁgure also includes the estimated (solid line) functional for ms of the\n",
    "dependence of the logarithm of hazard ratio (log-HR) of OS or PFS on the indices. The forms in general correspond to the ordering of the survival curves. For\n",
    "selected indices and outcomes the estimated functional form suggests a non-monotonic relationship between log-HR and the index.Figure S6. Evaluation of CIBERSORT Estimates of Immune Cell Subpopulation Fractions in the Context of mRNAsi and mDNAsi, Related to\n",
    "Figure 6\n",
    "(A) Plots show correlations between the mRNAsi (top) and mDNAsi (bottom) and the estimated fraction of cells for a selected panel of immune cell types.\n",
    "Additional correlations of stemness with estimated immune cell activity, computed as the difference between activated and resting population frac tions, are\n",
    "included for NK cells, CD4+ T cells and macrophages. Correlation with PD-L1 protein expression is included for reference and the tumor types are sorte d ac-\n",
    "cording to it.(B) Moonlight dynamic recognition analysis to identify associations with the mRNAsi across tumor types.See also Table S3 .(legend on next page)Figure S7. Connectivity Map, Related to Figure 7\n",
    "Heatmap showing each compound (perturbagen) from the Connectivity Map that share gene targets in (rows). Sorted by descending number of targets.\n",
    "See also Table S3 .`"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "metadata": {},
   "outputs": [],
   "source": [
    "const reorgPrompt = (rawPaper: string) => `I need help reorganizing this text that was extracted from a pdf without taking into account reading order. It's interleaved different parts so that it doesn't make sense reading it in the way it currently is. It also has some unnecessary information like a long list of references that should be removed. But everything that is related to the main content of the paper should be kept no matter what. Return a version of the text in a linear form that is readable inside of a single codeblock that contains the ENTIRE paper within a single codeblock of \\`\\`\\` so that it can extracted with regex. Do not hallucinate anything. Do not make up any information.` + `\\n\\n` + \n",
    "`\n",
    "\\`\\`\\`\n",
    "${rawPaper}\n",
    "\\`\\`\\`\n",
    "`"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "I need help reorganizing this text that was extracted from a pdf without taking into account reading order. It's interleaved different parts so that it doesn't make sense reading it in the way it currently is. It also has some unnecessary information like a long list of references that should be removed. But everything that is related to the main content of the paper should be kept no matter what. Return a version of the text in a linear form that is readable inside of a single codeblock that contains the ENTIRE paper within a single codeblock of ``` so that it can extracted with regex. Do not hallucinate anything. Do not make up any information.\n",
      "\n",
      "\n",
      "```\n",
      "Article\n",
      "Machine Learning Identiﬁes Stemness Features\n",
      "Associated with Oncogenic Dedifferentiation\n",
      "Graphical Abstract\n",
      "Highlights\n",
      "dEpigenetic and expression-based stemness indices\n",
      "measure oncogenic dedifferentiation\n",
      "dImmune microenvironment content and PD-L1 levelsassociate with stemness indices\n",
      "dStemness index is increased in metastatic tumors andreveals intratumor heterogeneity\n",
      "dApplying stemness indices reveals potential drug targets foranti-cancer therapiesAuthors\n",
      "Tathiane M. Malta, Artem Sokolov,Andrew J. Gentles, ..., Peter W. Laird,Houtan Noushmehr, Maciej Wiznerowicz\n",
      "Correspondence\n",
      "hnoushm1@hfhs.org (H.N.),\n",
      "maciej.wiznerowicz@iimo.pl (M.W.)\n",
      "In Brief\n",
      "Stemness features extracted from\n",
      "transcriptomic and epigenetic data fromTCGA tumors reveal novel biological andclinical insight, as well as potential drugtargets for anti-cancer therapies.\n",
      "Malta et al., 2018, Cell 173, 338–354\n",
      "April 5, 2018 ª2018 The Authors. Published by Elsevier Inc.\n",
      "https://doi.org/10.1016/j.cell.2018.03.034\n",
      "Article\n",
      "Machine Learning Identiﬁes Stemness Features\n",
      "Associated with Oncogenic Dedifferentiation\n",
      "Tathiane M. Malta,1,2,23 Artem Sokolov,3,23Andrew J. Gentles,4Tomasz Burzykowski,5Laila Poisson,1\n",
      "John N. Weinstein,6Bo_zena Kami /C19nska,7Joerg Huelsken,8Larsson Omberg,9Olivier Gevaert,4Antonio Colaprico,10,11\n",
      "Patrycja Czerwi /C19nska,12Sylwia Mazurek,12,13Lopa Mishra,14Holger Heyn,15Alex Krasnitz,16Andrew K. Godwin,17\n",
      "Alexander J. Lazar,6The Cancer Genome Atlas Research Network, Joshua M. Stuart,18Katherine A. Hoadley,19\n",
      "Peter W. Laird,20Houtan Noushmehr,1,2,23, *and Maciej Wiznerowicz12,21,22,23,24, *\n",
      "1Henry Ford Health System, Detroit, MI 48202, USA\n",
      "2University of Sa ˜o Paulo, Ribeira ˜o Preto-SP 14049, Brazil\n",
      "3Harvard Medical School, Boston, MA 02115, USA\n",
      "4Stanford University, Palo Alto, CA 94305, USA\n",
      "5Hasselt University, 3590 Diepenbeek, Belgium\n",
      "6The University of Texas MD Anderson Cancer Center, Houston, TX 77030, USA\n",
      "7Nencki Institute of Experimental Biology of PAS, 02093 Warsaw, Poland\n",
      "8Swiss Federal Institute of Technology Lausanne (EPFL), CH-1015 Lausanne; Switzerland\n",
      "9Sage Bionetworks, Seattle, WA 98109 USA\n",
      "10Universite ´Libre de Bruxelles, 1050 Bruxelles, Belgium\n",
      "11Interuniversity Institute of Bioinformatics in Brussels (IB)2, 1050 Bruxelles; Belgium\n",
      "12Pozna /C19n University of Medical Sciences, 61701 Pozna /C19n, Poland\n",
      "13Postgraduate School of Molecular Medicine, Medical University of Warsaw, 02109 Warsaw, Poland\n",
      "14George Washington University, Washington, D.C. 20052, USA\n",
      "15Centre for Genomic Regulation (CNAG-CRG), 08003 Barcelona, Spain\n",
      "16Cold Spring Harbor Laboratory, Cold Spring Harbor, NY 11724, USA\n",
      "17University of Kansas Medical Center, Kansas City, KS 66160, USA\n",
      "18University of California, Santa Cruz, Santa Cruz, CA 95064, USA\n",
      "19University of North Carolina, Chapel Hill, NC 27599, USA\n",
      "20Van Andel Research Institute, Grand Rapids, MI 49503, USA\n",
      "21Greater Poland Cancer Center, 61866 Pozna /C19n, Poland\n",
      "22International Institute for Molecular Oncology, 60203 Pozna /C19n, Poland\n",
      "23These authors contributed equally\n",
      "24Lead Contact\n",
      "*Correspondence: hnoushm1@hfhs.org (H.N.), maciej.wiznerowicz@iimo.pl (M.W.)\n",
      "https://doi.org/10.1016/j.cell.2018.03.034\n",
      "SUMMARY\n",
      "Cancer progression involves the gradual loss of a\n",
      "differentiated phenotype and acquisition of pro-genitor and stem-cell-like features. Here, we providenovel stemness indices for assessing the degreeof oncogenic dedifferentiation. We used an innova-\n",
      "tive one-class logistic regression (OCLR) machine-\n",
      "learning algorithm to extract transcriptomic andepigenetic feature sets derived from non-trans-formed pluripotent stem cells and their differentiatedprogeny. Using OCLR, we were able to identify previ-ously undiscovered biological mechanisms asso-ciated with the dedifferentiated oncogenic state.Analyses of the tumor microenvironment revealed\n",
      "unanticipated correlation of cancer stemness with\n",
      "immune checkpoint expression and inﬁltratingimmune cells. We found that the dedifferentiatedoncogenic phenotype was generally most prominentin metastatic tumors. Application of our stemnessindices to single-cell data revealed patterns of\n",
      "intra-tumor molecular heterogeneity. Finally, theindices allowed for the identiﬁcation of novel targetsand possible targeted therapies aimed at tumordifferentiation.\n",
      "INTRODUCTION\n",
      "Stemness, deﬁned as the potential for self-renewal and differen-\n",
      "tiation from the cell of origin, was originally attributed to normal\n",
      "stem cells that possess the ability to give rise to all cell types in\n",
      "the adult organism. Cancer progression involves gradual lossof a differentiated phenotype and acquisition of progenitor-like,stem-cell-like features. Undifferentiated primary tumors are\n",
      "more likely to result in cancer cell spread to distant organs,\n",
      "causing disease progression and poor prognosis, particularlybecause metastases are usually resistant to available therapies\n",
      "(Friedmann-Morvinski and Verma, 2014; Ge et al., 2017; Shibue\n",
      "and Weinberg, 2017; Visvader and Lindeman, 2012 ).\n",
      "An increasing number of genomic, epigenomic, transcrip-\n",
      "tomic, and proteomic signatures have been associated with\n",
      "338 Cell173, 338–354, April 5, 2018 ª2018 The Authors. Published by Elsevier Inc.\n",
      "This is an open access article under the CC BY-NC-ND license ( http://creativecommons.org/licenses/by-nc-nd/4.0/ ).A\n",
      "B\n",
      "C\n",
      "Figure 1. Development and Validation of the Stemness Indices\n",
      "(A) Overall methodology. Highlighted are data sources Progenitor Cell Biology Consortium (PCBC), Roadmap, and ENCODE databases; the OCLR machine-\n",
      "learning algorithm; and the resulting stemness indices mRNAsi, mDNAsi, and EREG-mRNAsi. The indices for each TCGA tumor sample were correlated withknown cancer biology, tumor pathology, clinical information, and drug sensitivity.(B) Stemness indices of the validation set derived using our stemness signature.\n",
      "(legend continued on next page)\n",
      "Cell173, 338–354, April 5, 2018 339cancer stemness. Those molecular features are causally con-\n",
      "nected to particular oncogenic signaling pathways that regulate\n",
      "transcriptional networks that sustain the growth and proliferation\n",
      "of cancer cells ( Ben-Porath et al., 2008; Eppert et al., 2011; Kim\n",
      "et al., 2010 ). Transcriptional and epigenetic dysregulation of can-\n",
      "cer cells frequently leads to oncogenic dedifferentiation and\n",
      "acquisition of stemness features by altering core signaling path-ways that regulate the phenotypes of normal stem cells ( Bradner\n",
      "et al., 2017; Young, 2011 ). Cell-extrinsic mechanisms can also\n",
      "affect maintenance of the undifferentiated state, largely through\n",
      "epigenetic mechanisms. Tumors comprise a complex, diverse,integrated ecosystem of relatively differentiated cancer cells,\n",
      "cancer stem cells, endothelial cells, tumor-associated ﬁbro-\n",
      "blasts, and inﬁltrating immune cells, among other cell types.The microenvironment of a tumor, considered as a pathologically\n",
      "formed ‘‘organ,’’ is frequently characterized by hypoxia, as well\n",
      "as by abnormal levels of various cytokines, growth factors, andmetabolites ( Lyssiotis and Kimmelman, 2017 ). It provides\n",
      "numerous opportunities for cell-cell signals to modulate the epi-\n",
      "genome and expression of stem-cell-like programs in cancer\n",
      "cells, frequently independent of their genetic backgrounds ( Gin-\n",
      "gold et al., 2016 ).\n",
      "Over the last decade, The Cancer Genome Atlas (TCGA) has\n",
      "illuminated the landscapes of primary tumors by generatingcomprehensive molecular proﬁles composed of genomic, epige-\n",
      "nomic, transcriptomic, and (post-translational) proteomic char-\n",
      "acteristics ( Hoadley et al., 2014; Tomczak et al., 2015 ), along\n",
      "with histopathological and clinical annotations. The resulting\n",
      "resource enabled us to analyze cancer stemness quite exten-\n",
      "sively in almost 12,000 samples of 33 tumor types.\n",
      "First, we deﬁned signatures to quantify stemness using pub-\n",
      "licly available molecular proﬁles from normal cell types that\n",
      "exhibit various degrees of stemness. By multi-platform analyses\n",
      "of their transcriptome, methylome, and transcription-factor bind-ing sites using an innovative one-class logistic regression\n",
      "(OCLR) machine-learning algorithm ( Sokolov et al., 2016 ), we ob-\n",
      "tained two independent stemness indices. One (mDNAsi) wasreﬂective of epigenetic features; the other (mRNAsi) was reﬂec-\n",
      "tive of gene expression. We then identiﬁed associations between\n",
      "the two stemness indices and novel oncogenic pathways,somatic alterations, and microRNA (miRNA) and transcriptionalregulatory networks. Those features correlated with, and\n",
      "perhaps govern, cancer stemness in particular molecular sub-\n",
      "types of TCGA tumors. Importantly, higher values for stemnessindices were associated with biological processes active in\n",
      "cancer stem cells and with greater tumor dedifferentiation, as\n",
      "reﬂected in histopathological grade. Metastatic tumor cellsappeared more dedifferentiated phenotypically, probably\n",
      "contributing to their aggressiveness. We also found tumor het-\n",
      "erogeneity at the single-cell level by measuring stemness in tran-scriptome proﬁles obtained from individual cancer cells. Using\n",
      "CIBERSORT to proﬁle immune cell types in TCGA tumors, weobtained insight into the interface of the immune system with\n",
      "stemness. Finally, we identiﬁed compounds speciﬁc to selected\n",
      "molecular targets and mechanisms that may eventually lead to\n",
      "novel treatments that trigger differentiation and exhaust thestemness potential of highly aggressive neoplasms.\n",
      "RESULTS\n",
      "DNA-Methylation- and mRNA-Expression-Based\n",
      "Stemness Classiﬁers\n",
      "We analyzed publicly available non-tumor and tumor datasetsfor which transcriptomic and epigenomic molecular proﬁles\n",
      "were available ( Figure 1 A). We derived stemness indices using\n",
      "an OCLR algorithm trained on stem cell (ESC, embryonic stemcell; iPSC, induced pluripotent stem cell) classes and their differ-\n",
      "entiated ecto-, meso-, and endoderm progenitors. We chose\n",
      "OCLR because it does not penalize misclassiﬁcation of stem-cell-derived progenitors at different stages of differentiationthat still carry some of the undifferentiated features in their\n",
      "molecular proﬁles (its output was also validated against random\n",
      "forests in Figure S1 A). OCLR-based transcriptomic and epige-\n",
      "netic signatures were applied to TCGA datasets to calculate\n",
      "the mRNAsi and mDNAsi. Each stemness index (si) ranges\n",
      "from low (zero) to high (one) stemness ( Table S1 ). The tumor\n",
      "samples stratiﬁed by the indices were used for the integrative\n",
      "analyses.\n",
      "mRNA Expression-Based Stemness Index\n",
      "We validated the mRNAsi by applying it to an external dataset\n",
      "composed of both stem cells and somatic differentiated cells(Nazor et al., 2012 )(Figure 1 B) and by scoring molecular sub-\n",
      "types of breast cancers and gliomas that are characterized by\n",
      "different degrees of oncogenic dedifferentiation associated\n",
      "with pathology and clinical outcome ( Figures S1 B and S1C). All\n",
      "stem cell samples attained higher stemness index values than\n",
      "samples from differentiated cells. TCGA tumors display various\n",
      "degrees of cancer stemness as revealed by mRNAsi ( Figure 1 C\n",
      "[left]) and mDNAsi ( Figure 1 C [right]). Germ-cell tumors, basal\n",
      "breast cancer, and Ly-Hem cancers displayed highly dedifferen-\n",
      "tiated phenotypes in comparison to other tumor types.\n",
      "Using gene set enrichment analysis (GSEA), we compared our\n",
      "signature to 16 gene sets that were associated with stemness\n",
      "in cancer and healthy cells in previous studies ( Ben-Porath\n",
      "et al., 2008; Ivanova et al., 2006; Kim and Orkin, 2011;Mathur et al., 2008; Palmer et al., 2012; Sato et al., 2003;\n",
      "Venezia et al., 2004; Yan et al., 2011 ). These sets spanned\n",
      "2,564 unique genes, with no 2 sets overlapping by more than134 genes. In all cases, the published stemness gene sets\n",
      "were signiﬁcantly enriched in mRNAsi ( Figure 2 A). We found\n",
      "that ‘‘cancer hallmark’’ gene sets were signiﬁcantly enriched,as were MYC targets, which signiﬁcantly contributed to the pos-\n",
      "itive side of the signature ( Hanahan and Weinberg, 2011 ). This is\n",
      "(C) TCGA tumor types sorted by the stemness indices obtained from transcriptomic (mRNAsi) and epigenetic (mDNAsi) features; indices were scaled fro m 0 (low)\n",
      "to 1 (high). The TCGA tumor types were grouped based on their histology and cell of origin into stem cell-like (SC), lympho-hematopoietic (Ly-Hem), ad eno-\n",
      "carcinomas, squamous cell carcinomas (Squamous), neuronal lineage (Neuronal), sarcomas (Sar), kidney tumors (Kidney), and not belonging to any of the above\n",
      "(Misc) ( Table S2 ).\n",
      "See also Figures S1 andS2andTables S1 andS2.\n",
      "340 Cell173, 338–354, April 5, 2018B A\n",
      "C\n",
      "D\n",
      "Figure 2. Biological Processes Associated with Cancer Stemness\n",
      "(A) Gene Set Enrichment Analysis showing RNA sequencing (RNA-seq)-based stemness signature evaluated in the context of gene sets representative fo r\n",
      "hallmarks of stemness and cancer.(B) Correlation between mRNAsi and mRNA expression for published hallmarks of stemness.(C) Correlation between mRNAsi and selected oncogenic processes.(D) Association between the epigenomic-based stemness signature (EREG-mDNAsi and EREG-mRNAsi) and enrichment in the transcription factor bindin g sites.\n",
      "See also Figure S2 andTable S2 .\n",
      "Cell173, 338–354, April 5, 2018 341consistent with MYC being one of the transcription factors that\n",
      "drive pluripotency in ESCs ( Young, 2011 ).\n",
      "Wingless-related integration site (Wnt)/ b-catenin and TGF- b\n",
      "signaling pathways were signiﬁcantly enriched on the negativeside of the stemness signature. This negative enrichment does\n",
      "not imply absence of speciﬁc signals in cancer stem cells, but\n",
      "rather that this signaling is lower relative to stem-cell-derived pro-genitors, as captured by the signature weights. This is againconsistent with other GSEA results, as both signaling pathways\n",
      "are known mediators of the epithelial-mesenchymal transition\n",
      "(EMT) mechanism ( Gonzalez and Medici, 2014 ). We also\n",
      "computed the correlation of mRNAsi against mRNA expression\n",
      "of published pan-cancer EMT markers ( Mak et al., 2016 ), which\n",
      "revealedsigniﬁcant correlations withformosttumors( FigureS2 C).\n",
      "This is consistent with the biology of ESCs, which grow as epithe-\n",
      "lioid, polygonal cells in vitro and epithelial cancer precursors hav-\n",
      "ing stem-like properties. Importantly, most TCGA samples are pri-mary tumors of an epithelial phenotype. Most skin melanomacases come from lymph nodes, and this tumor type shows higher\n",
      "expression of vimentin, a key marker of a mesenchymal pheno-\n",
      "type. mRNAsi is positively correlated with other core stem cell fac-tors: EZH2 ,OCT4 ,a n d SOX2 (Figure 2 Ba n d Table S2 ). Finally,\n",
      "Moonlight analysis of the oncogenic signatures from the Molecu-\n",
      "lar Signatures Database (MSigDB) further validated our gene-expression-based index and conﬁrmed engagement of MYC\n",
      "and EZH2, along with E2F3, MTOR, and SHH in driving oncogenic\n",
      "dedifferentiation ( Figure 2 C) (Colaprico et al., 2018 ).\n",
      "DNA Methylation-Based Stemness Index\n",
      "We deﬁned the mDNAsi using OCLR by combining (1) super-vised classiﬁcation between ESCs/iPSCs and their progenies,\n",
      "(2) stem cell signatures associated with pluripotency-speciﬁc\n",
      "genomic enhancer elements based on ChromHMM from Road-map, and (3) ELMER, which uses DNA methylation to identify\n",
      "enhancer elements and correlates their state with the expression\n",
      "of nearby genes. 219 CpG probes ( Figure S2 A) were selected in\n",
      "training OCLR using the Progenitor Cell Biology Consortium\n",
      "(PCBC) datasets. By selecting probes previously deﬁned to\n",
      "be active stemness-speciﬁc enhancers, we conﬁrmed the abilityof our approach to derive an mDNAsi. Since we focusedexclusively on hypomethylated, functionally important CpG\n",
      "probes associated with stem cells, we further explored cis-acti-\n",
      "vated genes.\n",
      "We scored each TCGA sample using the mDNAsi and used an\n",
      "external dataset to conﬁrm that stem cells had higher mDNAsi\n",
      "than differentiated samples ( Figure 1 B [left plot]). TCGA tumor\n",
      "types show different degrees of an inferred dedifferentiated\n",
      "phenotype ( Figure 1 C [right]). Within these, individual tumor sam-\n",
      "ples show variation for cancer stemness. As anticipated, TCGAsamples derived from the primary tumors show higher cancerstemness indices compared to non-tumor samples obtained\n",
      "from adjacent normal tissue of origin ( Figure S1 E [bottom]).\n",
      "Most of our selected probes fell within non-promoter ele-\n",
      "ments, yet the SOX2-OCT4 transcription factor binding motif is\n",
      "one of the most highly enriched signatures within these regions.\n",
      "The SOX2-OCT4 complex is a critical master regulator of plurip-otency and stemness and is highly enriched in tumor samples\n",
      "with high mDNAsi ( Figure 2 D).Correlations of mRNAsi and mDNAsi\n",
      "Since the inputs for mDNAsi and mRNAsi are not necessarily\n",
      "complementary, we explored stratiﬁcation of glioma samples\n",
      "by the epigenetically regulated mRNAsi (EREG-mRNAsi), asstemess index generated using a set of stemness-related epige-\n",
      "netically regulated genes. The EREG-mRNAsi, based on both\n",
      "RNA expression and epigenetics, elucidates the discrepancybetween mDNAsi and mRNAsi and shows a positive correlationwith both indices ( Figure S1 F). Both mRNAsi and mDNAsi show\n",
      "good correspondence for a majority of tumors ( Figures S1 F and\n",
      "S2B). We observed major discrepancies in the case of brain\n",
      "lower grade glioma (LGG), thyroid carcinoma (THCA), and thy-\n",
      "moma (THYM). For gliomas, mDNAsi is correlated positively\n",
      "with tumor pathology and clinical features, while mRNAsi showsa negative correlation. This result could arise from a high fre-\n",
      "quency of isocitrate dehydrogenase mutations ( IDH1/2 ) muta-\n",
      "tions and resulting DNA hypermethylation.\n",
      "Stemness Index Can Stratify Recognized\n",
      "Undifferentiated CancersWe examined breast invasive carcinoma (BRCA), acute myeloid\n",
      "leukemia (AML), and gliomas to study if the mRNAsi/mDNAsi\n",
      "predict stemness in poorly differentiated tumors. In BRCA, wefound a strong association between the stemness index and\n",
      "known clinical and molecular features ( Figure 3 A [left]). The\n",
      "mRNAsi was highest in the basal subtype, known to exhibitan aggressive phenotype associated with an undifferentiatedstate. BRCA samples with high mRNAsi were more likely to\n",
      "be estrogen receptor (ER)-negative and enriched for FAT3\n",
      "andTP53 mutations. We noted that high mRNAsi was associ-\n",
      "ated with higher protein expression of FOXM1, CYCLINB1,\n",
      "and MSH6, as well as higher miRNA-200 family expression\n",
      "(Figure 3 A [right]). Invasive lobular type of BRCA (ILC), charac-\n",
      "terized by better prognosis in comparison to invasive ductal\n",
      "carcinoma (IDC), has a lower mRNAsi ( Figure 3 A [right]). We\n",
      "also applied our indices to non-TCGA BRCA samples ( Rey-\n",
      "ngold et al., 2014 ) and found a similar correlation between\n",
      "mRNAsi and mDNAsi in those samples. Moreover, mRNAsi\n",
      "also stratiﬁed BRCA samples with distinct histology in this da-taset ( Figure S1 B). Using datasets with estimated tumor cell\n",
      "type composition provided by the epigenetic deconvolution\n",
      "method ( Onuchic et al., 2016 ), we found that both mRNAsi\n",
      "and mDNAsi were more highly correlated with malignantepithelial cells than with normal epithelial cells, suggesting\n",
      "that our indices identify distinct cancerous epithelial cell popu-\n",
      "lations characterized by different features or degrees of stem-ness ( Figure S1 D).\n",
      "We found an association between the mRNAsi, RNA expres-\n",
      "sion subtypes previously deﬁned by TCGA, and the French-American-British (FAB) classiﬁcation of AML ( Figure 3 B). The\n",
      "mRNAsi showed the strongest correlation with the stage of\n",
      "myeloid differentiation of the AML samples. FAB subtypes M0\n",
      "(undifferentiated), M1 (with minimal maturation), and M2 (withmaturation) were characterized by high mRNAsi. In contrast,\n",
      "the M3 well-matured promyelocytic subtype, which is associ-\n",
      "ated with benign chromosomal abnormalities and favorableclinical outcome, had low mRNAsi ( Figure 3 B [right upper]).\n",
      "High mRNAsi was associated with higher expression of\n",
      "342 Cell173, 338–354, April 5, 2018A\n",
      "B\n",
      "C\n",
      "Figure 3. Molecular and Clinical Features Associated with Stemness in Breast Cancer, Acute Myeloid Leukemia, and Gliomas\n",
      "(A) (Left) An overview of the association between known molecular and biological processes and stemness in BRCA. Columns represent samples sorted by\n",
      "mRNAsi from low to high (top row). Rows represent molecular and biological processes associated with mRNAsi. Rows named ‘‘EDec CEp 2 and 4’’ representestimated cell type proportions. (Top right) Boxplots of mRNAsi in individual samples, stratiﬁed by molecular subtype and histology. (Bottom right ) Correlation of\n",
      "mRNAsi and representative protein expression and microRNA.(B) Similar to (A), association of mRNAsi in AML. (Top right) mRNAsi by mRNA-based molecular subtype and by FAB classiﬁcation. (Bottom right) Correla tion\n",
      "scores of mRNAsi and representative microRNA.(C) As in (A) and (B), GBM and LGG sorted by mDNAsi. (Top right) mDNAsi by molecular subtype and grade. (Bottom right) Correlation scores of mDNAsi andrepresentative protein expression and microRNA.All molecular and clinical features shown are statistically signiﬁcant. See also Figures S1 ,S3,S4, and S5.\n",
      "Cell173, 338–354, April 5, 2018 343AB\n",
      "C D\n",
      "E\n",
      "Figure 4. Selected Molecular and Clinical Features Associated with the Stemness Indices in TCGA Tumors\n",
      "(A) Association of molecular and clinical features with stemness in LUAD. (Top) mDNAsi by integrative molecular subtypes, smoking history, and muta tions of\n",
      "TP53 and SETD2. (Bottom) Correlation scores of mDNAsi and representative protein expression.(B) Stemness in HNSC. (Top) mDNAsi stratiﬁed by molecular subtypes and mutation of NSD1. (Bottom) Correlation scores of mDNAsi and representative pr otein\n",
      "and microRNA expression.(C) Stemness in LIHC. (Top) mRNAsi stratiﬁed by grade and mutations of TP53, CTNNB1, and AXIN1. (Bottom) Correlation scores of mRNAsi and representat ive\n",
      "protein and microRNA expression.\n",
      "(legend continued on next page)\n",
      "344 Cell173, 338–354, April 5, 2018miR-181c-3p, miR-22-3p, and miR-30b-3p ( Figure 3 B [right\n",
      "bottom]).\n",
      "We found a strong association between high mDNAsi, high\n",
      "pathologic grade, and recently published molecular subtypesof glioma ( Figure 3 C). mDNAsi was low in less-aggressive\n",
      "gliomas that are characterized by codel and glioma CpG methyl-\n",
      "ator phenotype (G-CIMP)-high features and was highest in highlyaggressive glioblastoma multiformes (GBMs) characterized byIDH mutations (G-CIMP low) and poor clinical outcome. Also,\n",
      "high mDNAsi is strongly associated with more aggressive clas-\n",
      "sical and mesenchymal subtypes of GBM, suggesting that itcan stratify tumors with distinct clinical outcomes. We also found\n",
      "that high mDNAsi was associated with mutations in NF1 and\n",
      "EGFR and infrequent mutations in IDH1 ,TP53 ,CIC, and ATRX\n",
      "(Figure 3 C [left]), with higher expression of ANNEXIN-A1 protein\n",
      "and lower expression of ANNEXIN-A7 and with expression of the\n",
      "miR-200 family ( Figure 3 C [right bottom]).\n",
      "We obtained similar results on non-TCGA glioma samples for\n",
      "which both mRNA expression and DNA methylation data were\n",
      "available ( Turcan et al., 2012 )(Figure S1 C). The negative correla-\n",
      "tion between mDNAsi and mRNAsi was restricted to LGG sam-ples—speciﬁcally, the IDH mutant subtypes (G-CIMP high and\n",
      "codel). IDH1 mutations are known to reduce cell differentiation,\n",
      "and high values of the mRNAsi in a subset of IDH mutant gliomasmight capture this phenomenon ( Lu et al., 2012 ).\n",
      "Pan-cancer Stemness Landscape\n",
      "Next, we tested the ability of our indices to identify previously un-\n",
      "explored features of cancer stemness across all TCGA tumors .\n",
      "First, we performed an enrichment analysis by sorting all TCGAsamples by stemness index for each tumor type and lookingfor associations with mutations and molecular and clinical fea-\n",
      "tures. The most salient associations of mRNAsi and mDNAsi\n",
      "are presented in Figure 4 , while the following results of the\n",
      "comprehensive analyses are shown in the supplementary mate-\n",
      "rial: associations with mutations ( Figure S3 ), associations with\n",
      "miRNA expression and protein abundance ( Figure S4 ), associa-\n",
      "tions with the tumor grading, and clinical outcome ( Figure S5 ).\n",
      "Correlations of mRNAsi and mDNAsi with Mutations in\n",
      "Genes, miRNA, and Expression of Proteins\n",
      "We found a strong association of mDNAsi with known molecular\n",
      "subtypes, with somatic mutations in SETD2 and TP53 genes,\n",
      "and with tobacco smoking status in lung adenocarcinoma(LUAD) ( Figures 4 A and S3). Current smokers and recently\n",
      "reformed smokers have higher mDNAsi than non-smokers or\n",
      "long-term reformed smokers. This suggests that the stemnessof LUAD tumors might be activated in response to environmental\n",
      "stimuli such as smoking and might inﬂuence the aggressiveness\n",
      "of the tumor. We also found an association between mDNAsi andhigher protein expression of CYCLINB1 and FOXM1, which is a\n",
      "pro-stemness transcription factor upstream of CYCLINB1 ( Fig-ure 4 A [lower plots]). FOXM1 has been associated with dediffer-\n",
      "entiation in pancreatic cancer cells ( Bao et al., 2011 ), as well as\n",
      "tumor proliferation in the kidney ( Xue et al., 2012 ) and ovarian\n",
      "(Wen et al., 2014 ) cancers. Our result suggests that it could be\n",
      "a driver of dedifferentiation and proliferation in breast and lung\n",
      "cancers. Stemness of LUAD tumors is also associated with lower\n",
      "expression of ANNEXIN-A1 ( Figure 4 A). ANNEXIN-A1 has been\n",
      "indicated as a differentiation marker in pancreatic ( Bai et al.,\n",
      "2004\n",
      ") and urothelial ( Kang et al., 2012 ) cancers; therefore, we\n",
      "suspect that the relationship between ANNEXIN and FOXM1\n",
      "expression and tumor differentiation may extend to other tumortypes ( Figure S4 C).\n",
      "Analyses of head and neck squamous cell carcinoma (HNSC)\n",
      "samples revealed that high indices are correlated with NSD1 mu-\n",
      "tation, E-cadherin protein expression, miR-200-3p, and previ-\n",
      "ously identiﬁed classical molecular subtypes ( Figure 4 B).NSD1\n",
      "mutation was recently linked in HNSC tumors to blockade ofcellular differentiation and promotion of oncogenesis ( Papillon-\n",
      "Cavanagh et al., 2017 ). Interestingly, miR-200 family members\n",
      "have been implicated in cancer initiation and metastasis, as\n",
      "well as self-renewal of healthy stem cells ( Gregory et al., 2008;\n",
      "Tellez et al., 2011 ). HNSC tumors with high mDNAsi have\n",
      "reduced programmed death ligand 1 (PD-L1) protein level\n",
      "(Figure 4 B).\n",
      "In liver hepatocellular carcinoma (LIHC) samples, we found an\n",
      "association between mRNAsi and high pathological grade ( Fig-\n",
      "ure 4 C). Negative associations between mRNAsi and the proba-\n",
      "bility of overall survival (OS) or progression-free survival (PFS)\n",
      "were detected ( Figures 4 E and S5C). In contrast to the majority\n",
      "of tumor types, LIHC samples with high mRNAsi have lowexpression of miR-200 family members ( Figure 4 C). The miR-\n",
      "200 family is known to be associated with progression of hepa-\n",
      "tocellular carcinoma ( Tsai et al., 2017; Wong et al., 2015 ), and the\n",
      "miR-200b-ZEB1 circuit has been suggested as a master regu-lator of stemness in these cancers ( Tsai et al., 2017 ). We found\n",
      "associations of mRNAsi with higher CYCLINB1 and ACC1 and\n",
      "with lower PD-L1 and ANNEXIN-A1 protein expression in LIHC(Figure 4 C). ACC1 was associated with pathomorphological\n",
      "markers of LIHC aggressiveness (vascular invasion and poor\n",
      "differentiation), and its upregulation was correlated with poorOS and disease recurrence in hepatocellular carcinoma pa-tients ( Wang et al., 2016 ). LIHC samples with high mRNAsi\n",
      "were associated with speciﬁc genomic alterations (e.g., TP53 ,\n",
      "CTNNB1 ,AXIN1 ).\n",
      "Detailed analyses of adrenocortical carcinoma (ACC) sam-\n",
      "ples revealed an association between high mRNAsi and\n",
      "deﬁned molecular subtypes ( Zheng et al., 2016 ), clinical stage,\n",
      "and mutations in PRKAR1A and TP53 genes ( Figure 4 D). We\n",
      "found a positive correlation between mRNAsi and adrenal\n",
      "differentiation score that is based on expression of 25 genesthat are important for adrenal function ( Zheng et al., 2016 )\n",
      "(Figure 4 D).\n",
      "(D) Stemness in ACC. (Top) mRNAsi stratiﬁed by mRNA molecular subtypes, clinical stage, and mutations of PRKAR1A and TP53. (Bottom) Correlation scor es of\n",
      "mRNAsi and adrenal differentiation score.(E) Cox proportional hazards model analysis. (Left) Progression-free survival. (Right) Overall survival. Hazard ratio greater than one denotes a t rend toward higher\n",
      "stemness index with worse outcome.See also Figures S3 ,S4, and S5.\n",
      "Cell173, 338–354, April 5, 2018 345Stemness Indices Are Correlated with Tumor Pathology\n",
      "and Predictive of Clinical Outcome\n",
      "We observed a positive correlation between tumor histology and\n",
      "pathology grading and both stemness indices for the majority ofthe TCGA cases ( Figures 3 A, 3C, 4C, 4D, S1B,S5A, and S5B).\n",
      "For mRNAsi, the most signiﬁcant correlations were found for\n",
      "BRCA (IDC and ILC), cervical squamous cell carcinoma and en-docervical adenocarcinoma (CESC), LIHC, pancreatic adeno-\n",
      "carcinoma (PAAD), and uterine corpus endometrial carcinoma\n",
      "(UCEC) ( Figure S5 A). Interestingly, mRNAsi shows low values\n",
      "in GBM and stomach adenocarcinoma (STAD). On the otherhand, mDNAsi strongly stratiﬁes glioma by the pathology grade,\n",
      "culminating with the highest value for GBM ( Figure S5 B). The\n",
      "reversed values of mDNAsi and mRNAsi in case of gliomaswere also evident in the clinical data analyses. An adverse asso-\n",
      "ciation between the mRNAsi and survival was detected ( Fig-\n",
      "ure 4 E), which was signiﬁcant for OS and PFS after adjusting\n",
      "for clinical factors ( Figures S5 C). In contrast, the mDNAsi had\n",
      "no signiﬁcant association with OS and PFS after correcting for\n",
      "A\n",
      "BC\n",
      "D\n",
      "EFigure 5. Analysis of Cancer Stemness in\n",
      "the Context of Metastatic State and Intratu-mor Heterogeneity\n",
      "(A) mRNAsi is higher in cancer metastases in\n",
      "comparison to the TCGA primary tumors.(B) mDNAsi is higher in recurrent glioma samplescompared to the primary glioma occurrence fromthe same patient. G-CIMP, glioma CpG methylatorphenotype.(C and D) Application of mRNAsi to a single-celltranscriptome of gliomas and breast cancer revealintratumor heterogeneity and various degrees ofthe oncogenic dedifferentiation.(E) Correlation of mRNAsi and mRNA expression ofCDH1 (epithelial marker) and CDH2 (mesenchymalmarker) in the cancer metastases.\n",
      "clinical factors. We found a positive corre-\n",
      "lation between previously published gli-oma subtypes and mDNAsi, suggesting\n",
      "that mDNAsi might recapitulate prog-\n",
      "nostic molecular subtypes ( Figure 3 C).\n",
      "The discordance between the mRNAsi\n",
      "and the mDNAsi for gliomas may be ex-\n",
      "plained in part by the dominant genomicalteration associated with the LGG tumortype. Roughly 80% of LGG tumors carry\n",
      "anIDH1/2 mutation and, as demon-\n",
      "strated by our group and others, confera genome-wide hypermethylator pheno-\n",
      "type (G-CIMP) ( Noushmehr et al., 2010;\n",
      "Turcan et al., 2012 ). Given that the\n",
      "mDNAsi is driven primarily by low methyl-\n",
      "ation levels associated with the stemness\n",
      "phenotype, the LGG tumors mightresemble non-stem like phenotypes,\n",
      "which are predominantly hypermethy-\n",
      "lated. The subgroup of G-CIMP with thelowest overall DNA methylation levels\n",
      "(G-CIMP low) is associated with the worst outcomes. Compared\n",
      "to G-CIMP-high tumors, G-CIMP-low tumors are known to be\n",
      "more proliferative, express cell-cycle-related genes, and havevarious stem-cell-like genomic features ( Ceccarelli et al., 2016 ).\n",
      "Cancer Stemness Indices Are Higher in Tumor\n",
      "Metastases and Reveal Intratumor Heterogeneity\n",
      "The TCGA samples are derived mostly from primary tumors,\n",
      "except for skin melanoma, for which tissues are mostly metasta-tic lymph nodes. We used the mRNAsi to interrogate the MET500dataset comprising expression proﬁles from 500 metastatic\n",
      "samples obtained from 22 different organs ( Robinson et al.,\n",
      "2017 ). In most cases, mRNAsi was signiﬁcantly higher in meta-\n",
      "static samples compared to primary TCGA tumors ( Figure 5 A).\n",
      "Prostate and pancreatic adenocarcinoma metastases had the\n",
      "most dedifferentiated phenotypes and are also more aggressiveand resistant to therapies in contrast to primary tumors. Weaker\n",
      "association with the mRNAsi was due to a small number of avail-\n",
      "able samples (n < 20). Interestingly, testicular germ cell tumors\n",
      "346 Cell173, 338–354, April 5, 2018(TGCTs) present the less differentiated phenotypes in primary\n",
      "tumors when compared to distant metastases. Primary TGCT\n",
      "tumor cells have high mRNAsi and may differentiate whenmetastasizing to distant organs. A similar trend was observed\n",
      "for STAD.\n",
      "Using another dataset, we found that mDNAsi was signiﬁ-\n",
      "cantly higher in glioma samples obtained at ﬁrst recurrence in\n",
      "contrast to primary gliomas ( Figure 5 B). Our results reveal signif-\n",
      "icant dedifferentiation of glioma cancer cells that contribute toglioma recurrence, which is frequently associated with poorprognosis and resistance to treatment ( de Souza et al., 2018 ).\n",
      "By taking advantage of single-cell transcriptome datasets, we\n",
      "used mRNAsi to probe tumor heterogeneity for oncogenic dedif-ferentiation of individual cancer cells ( Chung et al., 2017; Tirosh\n",
      "et al., 2016 ). We revealed high variation of stemness in the glioma\n",
      "and breast primary tumors. Individual glioma cells showedhigher variation of oncogenic dedifferentiation in comparison to\n",
      "breast cancer cells ( Figure 5 C). Single cells from metastasesA\n",
      "BFigure 6. Association of Stemness Index\n",
      "with Immune Microenvironment\n",
      "(A) mDNAsi and mRNAsi in the context of im-\n",
      "mune microenvironment. Each panel shows theSpearman correlation between the stemness in-dex and PD-L1 protein expression plotted againstSpearman correlation between the same stem-ness index and total leukocyte fraction, as esti-mated from DNA methylation data.(B) Highlight of tumor types that exhibit strongcorrelation between stemness and PD-L1expression or total leukocyte fraction.See also Figure S6 .\n",
      "had higher stemness index in breast can-\n",
      "cer (Figure 5 D). Interestingly, the negative\n",
      "correlation of EMT signature and stem-\n",
      "ness that we observed in TCGA primary\n",
      "tumors was also found in metastatic sam-ples ( Figure 5 E).\n",
      "Stemness Index Evaluated in the\n",
      "Context of Immune Response\n",
      "We found that for many tumors, higher\n",
      "stemness indices are associated with areduced leukocyte fraction and lowerPD-L1 expression ( Figure 6 A). For\n",
      "mDNAsi, the most distinctive negative\n",
      "correlations were found in the PanCan-12squamous cluster (LUSC, lung squamous\n",
      "cell carcinoma; HNSC; BLCA, bladder ur-\n",
      "othelial carcinoma) ( Hoadley et al., 2014 )\n",
      "and in GBM ( Figures 6 A [left panel] and\n",
      "S6B). For the mRNAsi, the highest nega-\n",
      "tive correlation values were seen inGBM/LGG, prostate adenocarcinoma\n",
      "(PRAD), LIHC, and uterine carcinosar-\n",
      "coma (UCS) tumors ( Figures 6 A [right\n",
      "panel] and S6A). We expect that such tu-\n",
      "mors will be less susceptible to immune\n",
      "checkpoint blockade treatments due to insufﬁcient immune\n",
      "cell inﬁltration or preexisting downregulation of the PD-L1pathway, which makes further inhibition ineffective. Our ﬁndings\n",
      "are consistent with previous reports showing a strong correlation\n",
      "between PD-L1 protein expression and inﬁltration of CD8+ cyto-toxic lymphocytes ( Zaretsky et al., 2016 ).\n",
      "We further explored correlations between stemness and im-\n",
      "mune microenvironment variables in the context of molecularsubtypes of tumors. Figure 6 B highlights several tumor types\n",
      "with the strongest (positive or negative) correlations. Except\n",
      "for kidney renal clear cell carcinoma (KIRC), the association\n",
      "between stemness and PD-L1 expression and leukocyte fractionis readily apparent from the increasing and decreasing trends\n",
      "of individual variables across the molecular subtypes. For\n",
      "example, we found mesenchymal tumors to have the highestPD-L1 expression levels, the most signiﬁcant leukocyte\n",
      "fractions, and the lowest mDNAsi compared to other HNSC\n",
      "subtypes, suggesting potential susceptibility to checkpoint\n",
      "Cell173, 338–354, April 5, 2018 347blockade inhibitors. The use of immunotherapy for HNSC tumors\n",
      "is under active investigation ( Economopoulou et al., 2016; Fuer-\n",
      "eder, 2016 ) with the recent FDA approval of pembrolizumab;\n",
      "however, whether the effectiveness of therapy is limited to spe-ciﬁc HNSC molecular subtypes is not clear from those reports.\n",
      "To assess other relationships between stemness and tumor\n",
      "microenvironment, we computed correlations between stem-ness indices and individual types of immune cells. By applyingCIBERSORT, we scored 22 immune cell types for their relative\n",
      "abundance in TCGA tumor samples. These cell types included\n",
      "natural killer (NK) cells, monocytes, macrophages, dendriticand mast cells, eosinophils, and neutrophils. We also obtained\n",
      "absolute estimates by scaling their relative abundance by overall\n",
      "leukocyte inﬁltration in each tumor as determined by ESTIMATEapplied to DNA methylation data. For any given TCGA sample,\n",
      "we calculated the correlation between mDNAsi/mRNAsi and\n",
      "the estimated fraction of individual immune cell types. In additionto individual immune subpopulation fractions, we considered thefunctional activation of distinct cells by measuring the difference\n",
      "between activated and resting fractions of NK cells, CD4+\n",
      "T cells, and macrophages. This approach was motivated byrecent observations that activation of peripheral CD4+ T cells\n",
      "triggered by immunotherapy is responsible for the speciﬁc killing\n",
      "of tumor cells ( Spitzer et al., 2017 ).\n",
      "Although the squamous cluster tumors had a negative cor-\n",
      "relation between stemness and the fraction of CD4+ T cell pop-\n",
      "ulations, the activation state of the CD4+ T cells was higher indedifferentiated tumors. This ﬁnding is consistent with our\n",
      "observation that PD-L1 protein expression is lower in these tu-\n",
      "mors, suggesting again that immune checkpoint blockade mightbe ineffective, and an additional mechanism of immune evasionmay be operative. The opposite trend is present in thymomas,\n",
      "where PD-L1 protein expression and the fraction of the CD4+\n",
      "T cell population are positively correlated with tumor dedif-ferentiation. Likewise, the activation state of CD4+ T cells is\n",
      "lower in dedifferentiated tumors, suggesting that they might be\n",
      "more susceptible to immunotherapy treatments ( Figures S6 A\n",
      "and S6B).\n",
      "Connectivity Map Analysis Identiﬁes Potential\n",
      "Compounds/Inhibitors Capable of Targeting the\n",
      "Stemness Signature\n",
      "We employed the Connectivity Map (CMap), a data-driven, sys-tematic approach for discovering associations among genes,\n",
      "chemicals, and biological conditions, to search for candidate\n",
      "compounds that might target pathways associated with stem-ness. We found enrichment for compounds associated with\n",
      "stemness in at least three cancer types ( Figure 7 A). 5 com-\n",
      "pounds are signiﬁcantly enriched in more than 10 cancer typesand have been reported to inhibit stemness-related tumorige-nicity: the dopamine receptor antagonists thioridazine and pro-\n",
      "chlorperazine ( Cheng et al., 2015; Lu et al., 2015; Dolma et al.,\n",
      "2016 ), the Wnt signaling inhibitor pyrvinium ( Xu et al., 2016 ),\n",
      "the HSP90 inhibitor tanespimycin, and the protein synthesis in-\n",
      "hibitor puromycin. Further, the telomerase inhibitor gossypol\n",
      "induced apoptosis and growth inhibition of cancer stem cells(CSCs) ( Volate et al., 2010 ), and histone deacetylase inhibitors\n",
      "such as trichostatin A (SAHA) reduced glioblastoma stem cellgrowth ( Chiao et al., 2013 ). According to our analysis, pyrvinium\n",
      "and puromycin could inhibit stemness in LUAD. We found\n",
      "several candidates with recognized anti-CSC activity for\n",
      "HNSC, including the aforementioned compounds. For LIHC,thioridazine, a prospective inhibitor of lung cancer stem cells\n",
      "(Yue et al., 2016 ), pyrvinium, puromycin, prochlorperazine, and\n",
      "others are potential compounds targeting undifferentiated tu-mors ( Figure 7 ).\n",
      "CMap mode-of-action (MoA) analysis of the 74 compounds\n",
      "revealed 56 mechanisms of action shared by the above com-\n",
      "pounds ( Figure 7 B and Table S4 B). Five compounds (ﬂuspirilene,\n",
      "pimozide, prochlorperazine, thioridazine, and triﬂuoperazine)\n",
      "shared the MoA of dopamine receptor antagonist. We observed\n",
      "that entinostat, trichostatin-a, and vorinostat shared MoA asHDAC inhibitors, and LY-294002, zaprinast, zardaverine share\n",
      "MoA as phosphodiesterase inhibitors.\n",
      "CMap target analysis revealed 212 distinct drug-target genes\n",
      "shared by the mentioned compounds ( Figure S7 andTable S4 C).\n",
      "Eight genes are targets of ﬁve different compounds—namely,\n",
      "DRD2 (8 drugs), HTR2A (7 drugs), HRH1 (6 drugs), ADRA1A\n",
      "(5 drugs), CALM1 (5 drugs), CHRM3 (5 drugs), HTR1A (5 drugs),and HTR2C (5 drugs).\n",
      "Recent polypharmacology studies suggest the need to design\n",
      "compounds that act on multiple genes or molecular pathways. Inthis study, we observed similar mechanisms of action among\n",
      "different compounds, suggesting that selective therapies\n",
      "can target the undifferentiated phenotypes for selected can-cer types.\n",
      "DISCUSSIONThis study is based on integrated analysis of cancer stemness in\n",
      "almost 12,000 primary human tumors of 33 different cancertypes. We interrogated TCGA data for mutations, DNA methyl-\n",
      "ation, expression of mRNA and miRNA, expression and post-\n",
      "translational modiﬁcation of proteins, histopathological grade,and clinical outcome. Applying CIBERSORT, we gained insight\n",
      "into the tumor microenvironment and composition of immune\n",
      "cell inﬁltrates. By applying a machine-learning algorithm to mo-lecular datasets from normal stem cells and their progeny, wedeveloped two different molecular metrics of stemness and\n",
      "then used them to assess epigenomic and transcriptomic fea-\n",
      "tures of TCGA cancers according to their grade of oncogenicdedifferentiation. Ultimately, the analyses led us to potentially\n",
      "actionable targets (and their MoAs) as candidates for possible\n",
      "differentiation therapy of solid tumors and metastases. Ourapproach could be applied to longitudinal study of samples\n",
      "from primary, recurrent, and metastatic cancers, and gene\n",
      "expression signatures identiﬁed in the tumor samples can be\n",
      "used to interrogate CMap to suggest actionable targets and in-hibitors for further analysis.\n",
      "To the best of our knowledge, this is the ﬁrst study in which\n",
      "molecular PCBC datasets comprised of stem cells and deﬁnedpopulations of their differentiated progeny have been leveraged\n",
      "to develop a classiﬁcation tool and machine-learning algorithm\n",
      "for analysis of a spectrum of human malignancies. A number ofcancer stemness scores, based on genes that are differentially\n",
      "expressed between CSCs and non-CSCs, have been published\n",
      "348 Cell173, 338–354, April 5, 2018A\n",
      "B\n",
      "Figure 7. Correlation of Cancer Stemness With Drug Resistance: Connectivity Map Analysis\n",
      "(A) Heatmap showing enrichment score (positive in blue, negative in red) of each compound from the CMap for each cancer type. Compounds are sorted from\n",
      "right to left by descending number of cancer type signiﬁcantly enriched.(B) Heatmap showing each compound (perturbagen) from the CMap that shares mechanisms of action (rows) and sorted by descending number of compoundwith shared mechanisms of action.See also Figure S7 andTables S3 andS4.\n",
      "Cell173, 338–354, April 5, 2018 349and are relevant to clinical outcomes in AML ( Eppert et al., 2011;\n",
      "Gentles et al., 2010; Ng et al., 2016 ). In those studies, gene sets\n",
      "enriched in ESCs (e.g., targets of NANOG, OCT4, SOX2, and\n",
      "c-MYC) were frequently overexpressed in poorly differentiatedtumors compared with well-differentiated ones. In breast\n",
      "cancers, those gene sets were associated with high-grade es-\n",
      "trogen receptor-negative, basal-like tumors and poor clinicaloutcome ( Ben-Porath et al., 2008 ). Another web-based tool,\n",
      "StemChecker, uses a curated set of 49 published stemness sig-\n",
      "natures deﬁned by gene expression, RNAi screens, transcription\n",
      "factor binding sites, text mining of the literature, and othercomputational approaches. But it has been tested only for\n",
      "pancreatic ductal adenocarcinoma. In that case, high expression\n",
      "of stemness genes correlated with poor prognosis ( Pinto et al.,\n",
      "2015 ). All previous studies were transcriptome-based and\n",
      "limited to a narrow set of genes and a small number of tu-\n",
      "mor types.\n",
      "In the present study, we found oncogenic dedifferentiation to\n",
      "be associated with several characteristics: mutations in genes\n",
      "that encode oncogenes and epigenetic modiﬁers, perturbations\n",
      "in speciﬁc mRNA/miRNA transcriptional networks, and deregu-lation of signaling pathways. Cancer stemness also appeared to\n",
      "involve core expression of MYC ,OCT4 ,SOX2 , and other genes\n",
      "involved in the regulatory circuitry that underlies normal andmalignant self-renewal potential. Our indices derived from\n",
      "mRNA expression and DNA methylation signatures reliably\n",
      "stratiﬁed tumors of known stemness phenotype. High mRNAsiwas associated with basal breast carcinomas but also Her2\n",
      "and lumB subtypes that are more aggressive than the hor-\n",
      "mone-dependent lumA group. In contrast, high mDNAsi wasstrongly associated with high-grade glioblastomas, poor OS,and PFS. The association between stemness signatures\n",
      "and adverse outcome for some tumor types, including gli-\n",
      "omas, may reﬂect malignant cell origins or the impact of theirmicroenvironment.\n",
      "Dedifferentiated cells can arise from different sources: from\n",
      "long-lived stem or progenitor cells that accumulate mutationsin oncogenic pathways or via dedifferentiation from non-stem\n",
      "cancer cells that convert to CSCs through deregulation of devel-\n",
      "opmental and/or non-developmental pathways. It is important todistinguish between the inherent stemness of CSCs and dedif-ferentiation induced by the tumor microenvironment. However,\n",
      "addressing that issue would require further validation beyond\n",
      "the scope of this study using other genomic datasets and/or lab-oratory experiments.\n",
      "Both stemness indices were lowest in normal cells, increased\n",
      "in primary tumors, and highest in metastases, consistent with theidea that tumor progression generally involves oncogenic dedif-\n",
      "ferentiation. Interestingly, we observed negative associations\n",
      "between stemness and EMT gene signatures. The relationshipbetween EMT and stemness remains a hotly debated topic,\n",
      "with several studies showing that EMT is necessarily associated\n",
      "with stemness ( Fabregat et al., 2016 ). However, most TCGA data\n",
      "are obtained from primary tumors, which exist in a pre-EMTstate, since EMT is strongly associated with tumor progression\n",
      "and with metastasis for many tumor types. Cancer cells in\n",
      "many primary solid tumors are basically epithelial regardless oftheir degrees of dedifferentiation, but some cells in such con-texts could acquire mesenchymal characteristics either by\n",
      "accumulating additional mutations or by undergoing epigenetic\n",
      "changes shaped by the tumor microenvironment. Those mesen-\n",
      "chymal cells can traverse the underlying tissue, enter the blood-stream, and seed distant organs, where they reacquire an\n",
      "epithelial phenotype to form metastatic tumors.\n",
      "We observed epithelial phenotypes and increased stemess\n",
      "index in molecular proﬁles of tumor-type-matched metastaticsamples in the MET500 cohort. This portends an association\n",
      "between dedifferentiation and spread of tumor cells to distant\n",
      "organs. The observation is further supported by high mDNAsiin samples from recurrent gliomas. It appears that tumor growth\n",
      "de novo , or at recurrence/metastasis, is associated with an\n",
      "increased stemness phenotype. Decreased mRNAsi levelsseen in TGCT suggest its possible differentiation as a germ cell\n",
      "tumor type induced by the microenvironment of liver or lung\n",
      "parenchyma, the organs it most often colonizes. Clinically, ingeneral, tumor progression is associated with greater aggres-siveness and resistance to therapy of almost all types.\n",
      "The mRNAsi was high for individual primary glioma and breast\n",
      "cancer cells. Interestingly, when applied to transcriptomic pro-ﬁles obtained from analysis of single cancer cells in bulk tumors,\n",
      "stemness indices revealed a high degree of intratumor heteroge-\n",
      "neity with respect to dedifferentiation phenotype. The heteroge-neity was greater in gliomas than in breast cancer, suggesting\n",
      "that intratumor environment, including stromal cells, hypoxia,\n",
      "and inﬁltration of immune cells, may play a role in shaping CSCniches and affect cancer cell developmental plasticity. Further\n",
      "molecular analyses of cancer cells stratiﬁed by the stemness\n",
      "phenotype would provide novel insights into the biology of pri-mary tumors.\n",
      "We found that for a number of tumor types (GBM, LUSC,\n",
      "HNSC, and BLCA), higher mDNAsi was associated with reduced\n",
      "leukocyte fraction and/or lower PD-L1 expression. Such tumorsare expected to be less susceptible to immune checkpoint\n",
      "blockade, due either to insufﬁcient immune cell inﬁltration of tu-\n",
      "mors or to inherent downregulation of the PD-L1 pathway. Bothfactors can render immune checkpoint immunotherapy ineffec-\n",
      "tive. The interaction between PD-L1 on cancer cells and PD1 re-\n",
      "ceptor on T cells helps cancer cells elude the immune system bypreventing activation of cytotoxic T cells in lymph nodes andsubsequent recruitment of other immune cell types to the tumor\n",
      "site ( Chen and Mellman, 2013 ). The presence of tumor-inﬁltrating\n",
      "lymphocytes and/or PD-L1 expression correlates with aggres-siveness in gastrointestinal stromal tumors ( Bertucci et al.,\n",
      "2015 ) and breast carcinomas ( Polo´nia et al., 2017 ). Common fea-\n",
      "tures shared between cancer cells and stem cells in the contextof the immune response are being highlighted by a growing num-\n",
      "ber of studies showing that vaccination with ESCs or iPSCs can\n",
      "raise speciﬁc immune response against cancer cells ( Kooreman\n",
      "et al., 2018 ). That ﬁnding may indicate that both cell populations\n",
      "use protein networks that, in tumors, result in uncontrolled self-\n",
      "renewal and dedifferentiated phenotypes histopathologicallydeﬁned by loss of architecture speciﬁc to the tissue of origin.We speculate that the indices described here may help\n",
      "predict the efﬁcacy of stem-cell-based immunotherapies and\n",
      "contribute to the identiﬁcation of patients who will respond tosuch therapies.\n",
      "350 Cell173, 338–354, April 5, 2018We interrogated CMap using the gene expression signatures\n",
      "from tumor samples with the highest and lowest mRNAsi levels.\n",
      "Surprisingly, perhaps, the CMap analysis, which is based on\n",
      "only a limited number of treated cell lines, very precisely selecteddrugs that have been shown to affect cancer stem cells with spec-\n",
      "iﬁcity. These translational analyses may ultimately pave the way\n",
      "for implementation of differentiation therapies for solid tumors.\n",
      "Here, we have also shown that cancer hallmarks can be ex-\n",
      "tracted from datasets on cells with deﬁned phenotypes and\n",
      "used to train machine-learning methods applicable to index mo-\n",
      "lecular proﬁles of cancer. Our mRNAsi and mDNAsi can be trans-lated into stemness scores (e.g., STEM50) that stratify tumors\n",
      "based on their dedifferentiation features, thus providing bio-\n",
      "markers for prediction of patient outcomes and response to dif-ferentiation therapies.\n",
      "By deﬁning new metrics of cancer stemness and using them to\n",
      "interrogate TCGA datasets, our results provide a comprehensivecharacterization of dedifferentiation as new and signiﬁcant hall-marks of cancer. The strengths of the approach are that it lever-\n",
      "ages features of dedifferentiated cells across a spectrum of\n",
      "tumor types that reﬂect tumor pathology and, in some cases,clinical outcome. This study also provides strategies for inte-\n",
      "grated analysis of cancer genomics based on machine-learning\n",
      "methods trained on molecular proﬁles obtained from cells withdeﬁned phenotypes. The ﬁndings based on those methods\n",
      "may advance the development of objective diagnostics tools\n",
      "for quantitating cancer stemness in clinical tumors, perhaps lead-ing eventually to new biomarkers that predict tumor recurrence,\n",
      "guide treatment selection, or improve responses to therapy.\n",
      "STAR+METHODS\n",
      "Detailed methods are provided in the online version of this paper\n",
      "and include the following:\n",
      "dKEY RESOURCES TABLE\n",
      "dCONTACT FOR REAGENT AND RESOURCE SHARING\n",
      "dEXPERIMENTAL MODEL AND SUBJECT DETAILS\n",
      "dMETHOD DETAILS\n",
      "BDNA Methylation Data\n",
      "BRNA Expression Data\n",
      "BStemness Index Derived Using OCLR\n",
      "BDNA Methylation Stemness Signatures\n",
      "BStemness versus Molecular and Clinical Features\n",
      "BStemness versus Clinical Predictors\n",
      "BCompounds Targeting with Cancer Stemness\n",
      "dQUANTIFICATION AND STATISTICAL ANALYSIS\n",
      "dDATA AND SOFTWARE AVAILABILITY\n",
      "SUPPLEMENTAL INFORMATION\n",
      "Supplemental Information includes seven ﬁgures and four tables and can be\n",
      "found with this article online at https://doi.org/10.1016/j.cell.2018.03.034 .\n",
      "ACKNOWLEDGMENTS\n",
      "We thank Marcin Cie /C19slik from Michigan Center for Translational Pathology at\n",
      "University of Michigan for providing the MTE500 dataset. This work was sup-ported by the following grants: NIH grants U54 HG003273, U54HG003067, U54 HG003079, U24 CA143799, U24 CA143835, U24\n",
      "CA143840, U24 CA143843, U24 CA143845, U24 CA143848, U24CA143858, U24 CA143866, U24 CA143867, U24 CA143882, U24CA143883, U24 CA144025, and P30 CA016672; NCI grants 5R01CA180778,3U24CA143858, 1U24CA210990, 5U54HG006097, 1U24CA210949, and1U24CA210950; NIGMS grant 5R01GM109031; the Henry Ford Cancer Insti-tute’s Early Career Investigator Award grant A20054 to T.M.M; Sao PauloResearch Foundation (FAPESP) grants 2014/02245-3 and 2016/01975-3 toT.M.M. and H.N.; FAPESP grants 2014/08321-3, 2015/07925-5, 2016/01389-7, 2016/10436-9, 2016/06488-3, 2016/12329-5, and 2016/15485-8,\n",
      "and Henry Ford Hospital grant A30935 to H.N.; Spanish Institute of Health Car-\n",
      "los III grant CP14/00229; Mary K. Chapman Foundation gift ‘‘Chapman Foun-dation Fund for Bioinformatics,’’ CPRIT grant RP13039, and the Michael & Su-san Dell Foundation grant ‘‘The Lorraine Dell Program in Bioinformatics’’ toJ.N.W.; and the Polish Science Foundation Welcome grant 2010/3-3 to M.W.\n",
      "AUTHOR CONTRIBUTIONS\n",
      "The Cancer Genome Atlas Research Network contributed collectively to this\n",
      "study. The contributions of other authors are as follows: epigenetic-derivedstemness index, T.M.M., P.W.L., and H.N.; mRNA-expression-derived stem-\n",
      "ness index, A.S.; methodology, T.M.M., A.S., and H.N.; integrative analysis,\n",
      "T.M.M., A.S., and H.N.; clinical analysis, T.B. and L.P; drug analysis, A.C.;data interpretation, T.M.M., A.S., J.H., B.K., H.H., A.J.G., A.C., L.M., A.J.L.,A.K., A.K.G., J.M.S., K.A.H., P.C., O.G., S.M., P.W.L., H.N., and M.W.; data cu-ration, L.O.; writing, T.M.M., A.S., J.W., B.K., J.H., A.C., J.N.W., H.N., andM.W.; visualization, T.M.M., A.S., H.N., and M.W.; overall concept and coordi-nation, T.M.M., A.S., P.W.L., H.N., and M.W.\n",
      "DECLARATION OF INTERESTS\n",
      "Michael Seiler, Peter G. Smith, Ping Zhu, Silvia Buonamici, and Lihua Yu are\n",
      "employees of H3 Biomedicine, Inc. Parts of this work are the subject of a\n",
      "patent application: WO2017040526 titled ‘‘Splice variants associated with neo-morphic sf3b1 mutants.’’ Shouyoung Peng, Anant A. Agrawal, James Palacino,and Teng Teng are employees of H3 Biomedicine, Inc. Andrew D. Cherniack,Ashton C. Berger, and Galen F. Gao receive research support from Bayer Phar-maceuticals. Gordon B. Mills serves on the External Scientiﬁc Review Board ofAstrazeneca. Anil Sood is on the Scientiﬁc Advisory Board for Kiyatec and is ashareholder in BioPath. Jonathan S. Serody receives funding from Merck, Inc.Kyle R. Covington is an employee of Castle Biosciences, Inc. Preethi H. Gunar-atne is founder, CSO, and shareholder of NextmiRNA Therapeutics. ChristinaYau is a part-time employee/consultant at NantOmics. Franz X. Schaub is anemployee and shareholder of SEngine Precision Medicine, Inc. Carla Grandoriis an employee, founder, and shareholder of SEngine Precision Medicine, Inc.Robert N. Eisenman is a member of the Scientiﬁc Advisory Boards and share-holder of Shenogen Pharma and Kronos Bio. Daniel J. Weisenberger is aconsultant for Zymo Research Corporation. Joshua M. Stuart is the founderof Five3 Genomics and shareholder of NantOmics. Marc T. Goodman receivesresearch support from Merck, Inc. Andrew J. Gentles is a consultant for Ci-\n",
      "bermed. Charles M. Perou is an equity stock holder, consultant, and Board\n",
      "of Directors member of BioClassiﬁer and GeneCentric Diagnostics and isalso listed as an inventor on patent applications on the Breast PAM50 andLung Cancer Subtyping assays. Matthew Meyerson receives research supportfrom Bayer Pharmaceuticals; is an equity holder in, consultant for, and Scien-tiﬁc Advisory Board chair for OrigiMed; and is an inventor of a patent for EGFRmutation diagnosis in lung cancer, licensed to LabCorp. Eduard Porta-Pardo isan inventor of a patent for domainXplorer. Han Liang is a shareholder and sci-entiﬁc advisor of Precision Scientiﬁc and Eagle Nebula. Da Yang is an inventoron a pending patent application describing the use of antisense oligonucleo-tides against speciﬁc lncRNA sequence as diagnostic and therapeutic tools.Yonghong Xiao was an employee and shareholder of TESARO, Inc. Bin Fengis an employee and shareholder of TESARO, Inc. Carter Van Waes receivedresearch funding for the study of IAP inhibitor ASTX660 through a CooperativeAgreement between NIDCD, NIH, and Astex Pharmaceuticals. Raunaq Malho-tra is an employee and shareholder of Seven Bridges, Inc. Peter W. Laird serveson the Scientiﬁc Advisory Board for AnchorDx. Joel Tepper is a consultant at\n",
      "Cell173, 338–354, April 5, 2018 351EMD Serono. Kenneth Wang serves on the Advisory Board for Boston Scienti-\n",
      "ﬁc, Microtech, and Olympus. Andrea Califano is a founder, shareholder, andadvisory board member of DarwinHealth, Inc. and a shareholder and advisoryboard member of Tempus, Inc. Toni K. Choueiri serves as needed on advisoryboards for Bristol-Myers Squibb, Merck, and Roche. Lawrence Kwong re-ceives research support from Array BioPharma. Sharon E. Plon is a memberof the Scientiﬁc Advisory Board for Baylor Genetics Laboratory. Beth Y. Karlanserves on the Advisory Board of Invitae.\n",
      "Received: September 5, 2017\n",
      "Revised: January 30, 2018Accepted: March 14, 2018Published: April 5, 2018\n",
      "REFERENCES\n",
      "Agarwal, S., Graepel, T., Herbrich, R., Har-Peled, S., and Roth, D. (2005).\n",
      "Generalization Bounds for the Area Under the ROC Curve. J. Mach. Learn.Res. 6, 393–425 .\n",
      "Bai, X.-F., Ni, X.-G., Zhao, P., Liu, S.-M., Wang, H.-X., Guo, B., Zhou, L.-P., Liu,\n",
      "F., Zhang, J.-S., Wang, K., et al. (2004). Overexpression of annexin 1 in pancre-atic cancer and its clinical signiﬁcance. World J. Gastroenterol. 10, 1466–1470 .\n",
      "Bao, B., Wang, Z., Ali, S., Kong, D., Banerjee, S., Ahmad, A., Li, Y., Azmi, A.S.,\n",
      "Miele, L., and Sarkar, F.H. (2011). Over-expression of FoxM1 leads to epithe-lial-mesenchymal transition and cancer stem cell phenotype in pancreaticcancer cells. J. Cell. Biochem. 112, 2296–2306 .\n",
      "Ben-Porath, I., Thomson, M.W., Carey, V.J., Ge, R., Bell, G.W., Regev, A., and\n",
      "Weinberg, R.A. (2008). An embryonic stem cell-like gene expression signaturein poorly differentiated aggressive human tumors. Nat. Genet. 40, 499–507 .\n",
      "Benjamini, Y., and Hochberg, Y. (1995). Controlling the False Discovery Rate:\n",
      "A Practical and Powerful Approach to Multiple Testing. J. R. Stat. Soc. Ser. B57, 289–300 .\n",
      "Bertucci, F., Finetti, P., Mamessier, E., Pantaleo, M.A., Astolﬁ, A., Ostrowski,\n",
      "J., and Birnbaum, D. (2015). PDL1 expression is an independent prognosticfactor in localized GIST. OncoImmunology 4, e1002729 .\n",
      "Bradner, J.E., Hnisz, D., and Young, R.A. (2017). Transcriptional addiction in\n",
      "cancer. Cell 168, 629–643 .\n",
      "Ceccarelli, M., Barthel, F.P., Malta, T.M., Sabedot, T.S., Salama, S.R., Murray,\n",
      "B.A., Morozova, O., Newton, Y., Radenbaugh, A., Pagnotta, S.M., et al.; TCGAResearch Network (2016). Molecular proﬁling reveals biologically discrete sub-sets and pathways of progression in diffuse glioma. Cell 164, 550–563 .\n",
      "Chen, D.S., and Mellman, I. (2013). Oncology meets immunology: the cancer-\n",
      "immunity cycle. Immunity 39, 1–10 .\n",
      "Cheng, H.W., Liang, Y.H., Kuo, Y.L., Chuu, C.P., Lin, C.Y., Lee, M.H., Wu, A.T.,\n",
      "Yeh, C.T., Chen, E.I., Whang-Peng, J., et al. (2015). Identiﬁcation of thiorida-zine, an antipsychotic drug, as an antiglioblastoma and anticancer stem cellagent using public gene expression data. Cell Death Dis 6, e1753 .\n",
      "Chiao, M.T., Cheng, W.Y., Yang, Y.C., Shen, C.C., and Ko, J.L. (2013). Suber-\n",
      "oylanilide hydroxamic acid (SAHA) causes tumor growth slowdown and trig-\n",
      "gers autophagy in glioblastoma stem cells. Autophagy 9, 1509–1526 .\n",
      "Chung, W., Eum, H.H., Lee, H.-O., Lee, K.-M., Lee, H.-B., Kim, K.-T., Ryu, H.S.,\n",
      "Kim, S., Lee, J.E., Park, Y.H., et al. (2017). Single-cell RNA-seq enablescomprehensive tumour and immune cell proﬁling in primary breast cancer.Nat. Commun. 8, 15081 .\n",
      "Colaprico, A., Silva, T.C., Olsen, C., Garofano, L., Cava, C., Garolini, D., Sabe-\n",
      "dot, T.S., Malta, T.M., Pagnotta, S.M., Castiglioni, I., et al. (2016). TCGAbio-links: an R/Bioconductor package for integrative analysis of TCGA data.Nucleic Acids Res. 44, e71 .\n",
      "Colaprico, A., Olsen, C., Cava, C., Terkelsen, T., Silva, T.C., Olsen, A., Cantini,\n",
      "L., Bertoli, G., Zinovyev, A., Barillot, E., et al. (2018). Moonlight: a tool forbiological interpretation and driver genes discovery. bioRxiv, doi 10.1101/265322.\n",
      "Daily, K., Ho Sui, S.J., Schriml, L.M., Dexheimer, P.J., Salomonis, N., Schroll,\n",
      "R., Bush, S., Keddache, M., Mayhew, C., Lotia, S., et al. (2017). Molecular,phenotypic, and sample-associated data to describe pluripotent stem cell\n",
      "lines and derivatives. Sci. Data 4, 170030 .\n",
      "Davis, S., Bilke, S., Triche Jr, T., and Bootwalla, M. (2015). methylumi: Handle\n",
      "Illumina methylation data. R Package Version 2.18.0.\n",
      "de Souza, C.F., Sabedot, T.S., Malta, T.M., Stetson, L., Morozova, O., Soko-\n",
      "lov, A., Laird, P.W., Wiznerowicz, M., Iavarone, A., Snyder, J., et al. (2018).Distinct DNA Methylation Shift in a Subset of Glioma CpG Island MethylatorPhenotype (G-CIMP) during Tumor Recurrence. Cell Rep. Published online\n",
      "April 10, 2018. https://doi.org/10.1016/j.celrep.2018.03.107 .\n",
      "Dolma, S., Selvadurai, H.J., Lan, X., Lee, L., Kushida, M., Voisin, V., Whet-\n",
      "stone, H., So, M., Aviv, T., Park, N., et al. (2016). Inhibition of dopamine recep-tor D4 impedes autophagic ﬂux, proliferation, and survival of glioblastomastem cells. Cancer Cell 29, 859–873 .\n",
      "Economopoulou, P., Perisanidis, C., Giotakis, E.I., and Psyrri, A. (2016). The\n",
      "emerging role of immunotherapy in head and neck squamous cell carcinoma(HNSCC): anti-tumor immunity and clinical applications. Ann. Transl. Med.4, 173 .\n",
      "Eppert, K., Takenaka, K., Lechman, E.R., Waldron, L., Nilsson, B., van Galen,\n",
      "P., Metzeler, K.H., Poeppl, A., Ling, V., Beyene, J., et al. (2011). Stem cell geneexpression programs inﬂuence clinical outcome in human leukemia. Nat. Med.17, 1086–1093 .\n",
      "Fabregat, I., Malfettone, A., and Soukupova, J. (2016). New Insights into the\n",
      "Crossroads between EMT and Stemness in the Context of Cancer. J. Clin.\n",
      "Med. 5.\n",
      "Friedmann-Morvinski, D., and Verma, I.M. (2014). Dedifferentiation and re-\n",
      "programming: origins of cancer stem cells. EMBO Rep. 15, 244–253 .\n",
      "Fuereder, T. (2016). Immunotherapy for head and neck squamous cell carci-\n",
      "noma. Memo 9, 66–69 .\n",
      "Ge, Y., Gomez, N.C., Adam, R.C., Nikolova, M., Yang, H., Verma, A., Lu,\n",
      "C.P.-J., Polak, L., Yuan, S., Elemento, O., and Fuchs, E. (2017). Stem cell line-age inﬁdelity drives wound repair and cancer. Cell 169, 636–650 .\n",
      "Gentles, A.J., Plevritis, S.K., Majeti, R., and Alizadeh, A.A. (2010). Association\n",
      "of a leukemic stem cell gene expression signature with clinical outcomes inacute myeloid leukemia. JAMA 304, 2706–2715 .\n",
      "Gentles, A.J., Newman, A.M., Liu, C.L., Bratman, S.V., Feng, W., Kim, D., Nair,\n",
      "V.S., Xu, Y., Khuong, A., Hoang, C.D., et al. (2015). The prognostic landscape\n",
      "of genes and inﬁltrating immune cells across human cancers. Nat. Med. 21,\n",
      "938–945 .\n",
      "Gevaert, O., Villalobos, V., Sikic, B.I., and Plevritis, S.K. (2013). Identiﬁcation of\n",
      "ovarian cancer driver genes by using module network integration of multi-omics data. Interface Focus 3, 20130013 .\n",
      "Gingold, J., Zhou, R., Lemischka, I.R., and Lee, D.-F. (2016). Modeling Cancer\n",
      "with Pluripotent Stem Cells. Trends Cancer 2, 485–494 .\n",
      "Gonzalez, D.M., and Medici, D. (2014). Signaling mechanisms of the epithelial-\n",
      "mesenchymal transition. Sci. Signal. 7, re8 .\n",
      "Gregory, P.A., Bert, A.G., Paterson, E.L., Barry, S.C., Tsykin, A., Farshid, G.,\n",
      "Vadas, M.A., Khew-Goodall, Y., and Goodall, G.J. (2008). The miR-200 familyand miR-205 regulate epithelial to mesenchymal transition by targeting ZEB1and SIP1. Nat. Cell Biol. 10, 593–601 .\n",
      "Hanahan, D., and Weinberg, R.A. (2011). Hallmarks of cancer: the next gener-\n",
      "ation. Cell 144, 646–674 .\n",
      "Hoadley, K.A., Yau, C., Wolf, D.M., Cherniack, A.D., Tamborero, D., Ng, S.,\n",
      "Leiserson, M.D.M., Niu, B., McLellan, M.D., Uzunangelov, V., et al.; CancerGenome Atlas Research Network (2014). Multiplatform analysis of 12 cancertypes reveals molecular classiﬁcation within and across tissues of origin.Cell158, 929–944 .\n",
      "Ivanova, N., Dobrin, R., Lu, R., Kotenko, I., Levorse, J., DeCoste, C., Schafer,\n",
      "X., Lun, Y., and Lemischka, I.R. (2006). Dissecting self-renewal in stem cellswith RNA interference. Nature 442, 533–538 .\n",
      "Kang, W.-Y., Chen, W.-T., Huang, Y.-C., Su, Y.-C., and Chai, C.-Y. (2012).\n",
      "Overexpression of annexin 1 in the development and differentiation of urothe-lial carcinoma. Kaohsiung J. Med. Sci. 28, 145–150 .\n",
      "352 Cell173, 338–354, April 5, 2018Kim, J., and Orkin, S.H. (2011). Embryonic stem cell-speciﬁc signatures in can-\n",
      "cer: insights into genomic regulatory networks and implications for medicine.Genome Med. 3,7 5.\n",
      "Kim, J., Woo, A.J., Chu, J., Snow, J.W., Fujiwara, Y., Kim, C.G., Cantor, A.B.,\n",
      "and Orkin, S.H. (2010). A Myc network accounts for similarities between em-bryonic stem and cancer cell transcription programs. Cell 143, 313–324 .\n",
      "Kooreman, N.G., Kim, Y., de Almeida, P.E., Termglinchan, V., Diecke, S.,\n",
      "Shao, N.-Y., Wei, T.-T., Yi, H., Dey, D., Nelakanti, R., et al. (2018). AutologousiPSC-Based Vaccines Elicit Anti-tumor Responses In Vivo. Cell Stem Cell.Published online February 8, 2018. https://doi.org/10.1016/j.stem.2018.\n",
      "01.016 .\n",
      "Kundaje, A., Meuleman, W., Ernst, J., Bilenky, M., Yen, A., Heravi-Moussavi,\n",
      "A., Kheradpour, P., Zhang, Z., Wang, J., Ziller, M.J., et al.; Roadmap Epige-\n",
      "nomics Consortium (2015). Integrative analysis of 111 reference human epige-nomes. Nature 518, 317–330 .\n",
      "Lamb, J., Crawford, E.D., Peck, D., Modell, J.W., Blat, I.C., Wrobel, M.J.,\n",
      "Lerner, J., Brunet, J.-P., Subramanian, A., Ross, K.N., et al. (2006). The Con-nectivity Map: using gene-expression signatures to connect small molecules,genes, and disease. Science 313, 1929–1935 .\n",
      "Lu, C., Ward, P.S., Kapoor, G.S., Rohle, D., Turcan, S., Abdel-Wahab, O., Ed-\n",
      "wards, C.R., Khanin, R., Figueroa, M.E., Melnick, A., et al. (2012). IDH mutationimpairs histone demethylation and results in a block to cell differentiation.Nature 483, 474–478 .\n",
      "Lu, M., Li, J., Luo, Z., Zhang, S., Xue, S., Wang, K., Shi, Y., Zhang, C., Chen, H.,\n",
      "and Li, Z. (2015). Roles of dopamine receptors and their antagonist thioridazinein hepatoma metastasis. Onco. Targets Ther 8, 1543–1552 .\n",
      "Lyssiotis, C.A., and Kimmelman, A.C. (2017). Metabolic interactions in the tu-\n",
      "mor microenvironment. Trends Cell Biol. 27, 863–875 .\n",
      "Mak, M.P., Tong, P., Diao, L., Cardnell, R.J., Gibbons, D.L., William, W.N.,\n",
      "Skoulidis, F., Parra, E.R., Rodriguez-Canales, J., Wistuba, I.I., et al. (2016). APatient-Derived, Pan-Cancer EMT Signature Identiﬁes Global Molecular Alter-ations and Immune Target Enrichment Following Epithelial-to-MesenchymalTransition. Clin. Cancer Res. 22, 609–620 .\n",
      "Mathur, D., Danford, T.W., Boyer, L.A., Young, R.A., Gifford, D.K., and Jae-\n",
      "nisch, R. (2008). Analysis of the mouse embryonic stem cell regulatory net-\n",
      "works obtained by ChIP-chip and ChIP-PET. Genome Biol. 9, R126 .\n",
      "Nazor, K.L., Altun, G., Lynch, C., Tran, H., Harness, J.V., Slavin, I., Garitaonan-\n",
      "dia, I., Mu ¨ller, F.-J., Wang, Y.-C., Boscolo, F.S., et al. (2012). Recurrent\n",
      "variations in DNA methylation in human pluripotent stem cells and their differ-entiated derivatives. Cell Stem Cell 10, 620–634 .\n",
      "Ng, S.W.K., Mitchell, A., Kennedy, J.A., Chen, W.C., McLeod, J., Ibrahimova,\n",
      "N., Arruda, A., Popescu, A., Gupta, V., Schimmer, A.D., et al. (2016). A 17-genestemness score for rapid determination of risk in acute leukaemia. Nature 540,\n",
      "433–437 .\n",
      "Noushmehr, H., Weisenberger, D.J., Diefes, K., Phillips, H.S., Pujara, K., Ber-\n",
      "man, B.P., Pan, F., Pelloski, C.E., Sulman, E.P., Bhat, K.P., et al.; CancerGenome Atlas Research Network (2010). Identiﬁcation of a CpG island meth-ylator phenotype that deﬁnes a distinct subgroup of glioma. Cancer Cell 17,\n",
      "510–522 .\n",
      "Onuchic, V., Hartmaier, R.J., Boone, D.N., Samuels, M.L., Patel, R.Y., White,\n",
      "W.M., Garovic, V.D., Oesterreich, S., Roth, M.E., Lee, A.V., and Milosavljevic,A. (2016). Epigenomic Deconvolution of Breast Tumors Reveals MetabolicCoupling between Constituent Cell Types. Cell Rep. 17, 2075–2086 .\n",
      "Palmer, N.P., Schmid, P.R., Berger, B., and Kohane, I.S. (2012). A gene\n",
      "expression proﬁle of stem cell pluripotentiality and differentiation is conservedacross diverse solid and hematopoietic cancers. Genome Biol. 13, R71 .\n",
      "Papillon-Cavanagh, S., Lu, C., Gayden, T., Mikael, L.G., Bechet, D., Karam-\n",
      "boulas, C., Ailles, L., Karamchandani, J., Marchione, D.M., Garcia, B.A.,et al. (2017). Impaired H3K36 methylation deﬁnes a subset of head and necksquamous cell carcinomas. Nat. Genet. 49, 180–185 .\n",
      "Pinto, J.P., Kalathur, R.K., Oliveira, D.V., Barata, T., Machado, R.S.R.,\n",
      "Machado, S., Pacheco-Leyva, I., Duarte, I., and Futschik, M.E. (2015).StemChecker: a web-based tool to discover and explore stemness signatures\n",
      "in gene sets. Nucleic Acids Res. 43(W1), W72–W77 .\n",
      "Polo´nia, A., Pinto, R., Cameselle-Teijeiro, J.F., Schmitt, F.C., and Paredes, J.\n",
      "(2017). Prognostic value of stromal tumour inﬁltrating lymphocytes and pro-grammed cell death-ligand 1 expression in breast cancer. J. Clin. Pathol. 70,\n",
      "860–867\n",
      ".\n",
      "R Development Core Team (2017). R: A Language and Environment for Statis-\n",
      "tical Computing (R Foundation for Statistical Computing) .\n",
      "Reyngold, M., Turcan, S., Giri, D., Kannan, K., Walsh, L.A., Viale, A., Drobnjak,\n",
      "M., Vahdat, L.T., Lee, W., and Chan, T.A. (2014). Remodeling of the methyl-ation landscape in breast cancer metastasis. PLoS ONE 9, e103896 .\n",
      "Robinson, D.R., Wu, Y.-M., Lonigro, R.J., Vats, P., Cobain, E., Everett, J., Cao,\n",
      "X., Rabban, E., Kumar-Sinha, C., Raymond, V., et al. (2017). Integrative clinicalgenomics of metastatic cancer. Nature 548, 297–303 .\n",
      "Royston, P., and Altman, D.G. (1994). Regression Using Fractional Polyno-\n",
      "mials of Continuous Covariates: Parsimonious Parametric Modelling. J. R.Stat. Soc. Ser. C. Appl. Stat 43, 429 .\n",
      "Salomonis, N., Dexheimer, P.J., Omberg, L., Schroll, R., Bush, S., Huo, J.,\n",
      "Schriml, L., Ho Sui, S., Keddache, M., Mayhew, C., et al. (2016). IntegratedGenomic Analysis of Diverse Induced Pluripotent Stem Cells from the Progen-itor Cell Biology Consortium. Stem Cell Reports 7, 110–125 .\n",
      "Sato, N., Sanjuan, I.M., Heke, M., Uchida, M., Naef, F., and Brivanlou, A.H.\n",
      "(2003). Molecular signature of human embryonic stem cells and its comparisonwith the mouse. Dev. Biol. 260, 404–413 .\n",
      "Sergushichev, A. (2016). An algorithm for fast preranked gene set enrichment\n",
      "analysis using cumulative statistic calculation. bioRxiv, doi 10.1101/060012.\n",
      "Shibue, T., and Weinberg, R.A. (2017). EMT, CSCs, and drug resistance: the\n",
      "mechanistic link and clinical implications. Nat. Rev. Clin. Oncol. 14, 611–629 .\n",
      "Silva, T.C., Colaprico, A., Olsen, C., Bontempi, G., Ceccarelli, M., Berman,\n",
      "B.P., and Noushmehr, H. (2017). TCGAbiolinksGUI: A graphical user interfaceto analyze cancer molecular and clinical data. bioRxiv, doi 10.1101/147496.\n",
      "Sokolov, A., Paull, E.O., and Stuart, J.M. (2016). One-class detection of cell\n",
      "states in tumor subtypes. Pac. Symp. Biocomput. 21, 405–416 .\n",
      "Spitzer, M.H., Carmi, Y., Reticker-Flynn, N.E., Kwek, S.S., Madhireddy, D.,\n",
      "Martins, M.M., Gherardini, P.F., Prestwood, T.R., Chabon, J., Bendall, S.C.,\n",
      "et al. (2017). Systemic immunity is required for effective cancer immuno-therapy. Cell 168, 487–502 .\n",
      "StataCorp (2013). Stata Statistical Software: Release 13 (College Station, TX:\n",
      "StataCorp LP) .\n",
      "Sturm, D., Witt, H., Hovestadt, V., Khuong-Quang, D.-A., Jones, D.T.W., Ko-\n",
      "nermann, C., Pfaff, E., To ¨njes, M., Sill, M., Bender, S., et al. (2012). Hotspot\n",
      "mutations in H3F3A and IDH1 deﬁne distinct epigenetic and biological sub-groups of glioblastoma. Cancer Cell 22, 425–437 .\n",
      "Subramanian, A., Tamayo, P., Mootha, V.K., Mukherjee, S., Ebert, B.L., Gil-\n",
      "lette, M.A., Paulovich, A., Pomeroy, S.L., Golub, T.R., Lander, E.S., and Me-sirov, J.P. (2005). Gene set enrichment analysis: a knowledge-based approachfor interpreting genome-wide expression proﬁles. Proc. Natl. Acad. Sci. USA102, 15545–15550 .\n",
      "Subramanian, A., Narayan, R., Corsello, S.M., Peck, D.D., Natoli, T.E., Lu, X.,\n",
      "Gould, J., Davis, J.F., Tubelli, A.A., Asiedu, J.K., et al. (2017). A next generationconnectivity map: L1000 platform and the ﬁrst 1,000,000 proﬁles. Cell 171,\n",
      "1437–1452 .\n",
      "Tellez, C.S., Juri, D.E., Do, K., Bernauer, A.M., Thomas, C.L., Damiani, L.A.,\n",
      "Tessema, M., Leng, S., and Belinsky, S.A. (2011). EMT and stem cell-like prop-erties associated with miR-205 and miR-200 epigenetic silencing are earlymanifestations during carcinogen-induced transformation of human lungepithelial cells. Cancer Res. 71, 3087–3097 .\n",
      "Therneau, T.M., and Grambsch, P.M. (2000). Estimating the survival and haz-\n",
      "ard functions. In Modeling Survival Data: Extending the Cox Model (New York,NY: Springer New York), pp. 7–37 .\n",
      "Tirosh, I., Venteicher, A.S., Hebert, C., Escalante, L.E., Patel, A.P., Yizhak, K.,\n",
      "Fisher, J.M., Rodman, C., Mount, C., Filbin, M.G., et al. (2016). Single-cell\n",
      "Cell173, 338–354, April 5, 2018 353RNA-seq supports a developmental hierarchy in human oligodendroglioma.\n",
      "Nature 539, 309–313 .\n",
      "Tomczak, K., Czerwi /C19nska, P., and Wiznerowicz, M. (2015). The Cancer\n",
      "Genome Atlas (TCGA): an immeasurable source of knowledge. Contemp. On-col. (Pozn.) 19(1A), A68–A77 .\n",
      "Tsai, S.-C., Lin, C.-C., Shih, T.-C., Tseng, R.-J., Yu, M.-C., Lin, Y.-J., and\n",
      "Hsieh, S.-Y. (2017). The miR-200b-ZEB1 circuit regulates diverse stemnessof human hepatocellular carcinoma. Mol. Carcinog. 56, 2035–2047 .\n",
      "Turcan, S., Rohle, D., Goenka, A., Walsh, L.A., Fang, F., Yilmaz, E., Campos,\n",
      "C., Fabius, A.W.M., Lu, C., Ward, P.S., et al. (2012). IDH1 mutation is sufﬁcientto establish the glioma hypermethylator phenotype. Nature 483, 479–483 .\n",
      "Venezia, T.A., Merchant, A.A., Ramos, C.A., Whitehouse, N.L., Young, A.S.,\n",
      "Shaw, C.A., and Goodell, M.A. (2004). Molecular signatures of proliferationand quiescence in hematopoietic stem cells. PLoS Biol. 2, e301 .\n",
      "Visvader, J.E., and Lindeman, G.J. (2012). Cancer stem cells: current status\n",
      "and evolving complexities. Cell Stem Cell 10, 717–728 .\n",
      "Volate, S.R., Kawsaki, B.T., Hurt, E.M., Milner, J.A., Kim, Y.S., White, J., and\n",
      "Farrar, W.L. (2010). Gossypol induces apoptosis by activating p53 in prostatecancer cells and prostate tumor-initiating cells. Mol. Cancer Ther 9, 461–470 .\n",
      "Wang, M.-D., Wu, H., Fu, G.-B., Zhang, H.-L., Zhou, X., Tang, L., Dong, L.-W.,\n",
      "Qin, C.-J., Huang, S., Zhao, L.-H., et al. (2016). Acetyl-coenzyme A carbox-ylase alpha promotion of glucose-mediated fatty acid synthesis enhancessurvival of hepatocellular carcinoma in mice and patients. Hepatology 63,\n",
      "1272–1286 .\n",
      "Wen, N., Wang, Y., Wen, L., Zhao, S.-H., Ai, Z.-H., Wang, Y., Wu, B., Lu, H.-X.,\n",
      "Yang, H., Liu, W.-C., and Li, Y. (2014). Overexpression of FOXM1 predicts poorprognosis and promotes cancer cell proliferation, migration and invasion inepithelial ovarian cancer. J. Transl. Med. 12, 134 .\n",
      "Wickham, H. (2009). ggplot2 (Springer Nature) .\n",
      "Wong, C.-M., Wei, L., Au, S.L.-K., Fan, D.N.-Y., Zhou, Y., Tsang, F.H.-C., Law,\n",
      "C.-T., Lee, J.M.-F., He, X., Shi, J., et al. (2015). MiR-200b/200c/429 subfamilynegatively regulates Rho/ROCK signaling pathway to suppress hepatocellular\n",
      "carcinoma metastasis. Oncotarget 6, 13658–13670 .\n",
      "Xu, L., Zhang, L., Hu, C., Liang, S., Fei, X., Yan, N., Zhang, Y., and Zhang, F.\n",
      "(2016). WNT pathway inhibitor pyrvinium pamoate inhibits the self-renewal\n",
      "and metastasis of breast cancer stem cells. Int. J. Oncol. 48, 1175–1186 .\n",
      "Xue, Y.J., Xiao, R.H., Long, D.Z., Wang, X.N., Zhang, G.N., Yuan, Y.H., Wu,\n",
      "G.Q., Yang, J., Wu, Y.T., Xu, H., et al. (2012). Overexpression of FoxM1 is asso-ciated with tumor progression in patients with clear cell renal cell carcinoma.J. Transl. Med 10, 200 .\n",
      "Yan, X., Ma, L., Yi, D., Yoon, J.G., Diercks, A., Foltz, G., Price, N.D., Hood, L.E.,\n",
      "and Tian, Q. (2011). A CD133-related gene expression signature identiﬁes an\n",
      "aggressive glioblastoma subtype with excessive mutations. Proc. Natl.Acad. Sci. USA 108, 1591–1596 .\n",
      "Yao, L., Shen, H., Laird, P.W., Farnham, P.J., and Berman, B.P. (2015). Infer-\n",
      "ring regulatory element landscapes and transcription factor networks fromcancer methylomes. Genome Biol. 16, 105 .\n",
      "Young, R.A. (2011). Control of the embryonic stem cell state. Cell 144,\n",
      "940–954 .\n",
      "Yue, H., Huang, D., Qin, L., Zheng, Z., Hua, L., Wang, G., Huang, J., and\n",
      "Huang, H. (2016). Targeting Lung Cancer Stem Cells with AntipsychologicalDrug Thioridazine. BioMed Res. Int. 2016 , 6709828 .\n",
      "Zaretsky, J.M., Garcia-Diaz, A., Shin, D.S., Escuin-Ordinas, H., Hugo, W.,\n",
      "Hu-Lieskovan, S., Torrejon, D.Y., Abril-Rodriguez, G., Sandoval, S., Barthly,\n",
      "L., et al. (2016). Mutations Associated with Acquired Resistance to PD-1Blockade in Melanoma. N. Engl. J. Med. 375, 819–829 .\n",
      "Zheng, S., Cherniack, A.D., Dewal, N., Mofﬁtt, R.A., Danilova, L., Murray, B.A.,\n",
      "Lerario, A.M., Else, T., Knijnenburg, T.A., Ciriello, G., et al.; Cancer GenomeAtlas Research Network (2016). Comprehensive Pan-Genomic Characteriza-tion of Adrenocortical Carcinoma. Cancer Cell 29, 723–736 .\n",
      "354 Cell173, 338–354, April 5, 2018STAR+METHODS\n",
      "KEY RESOURCES TABLE\n",
      "CONTACT FOR REAGENT AND RESOURCE SHARING\n",
      "Further information and requests for resources should be directed to and will be fulﬁlled by the Lead Contact, Maciej Wiznerowicz\n",
      "(maciej.wiznerowicz@iimo.pl ).\n",
      "EXPERIMENTAL MODEL AND SUBJECT DETAILS\n",
      "Clinical and molecular data were collected from the NIH Genomic Data Commons (GDC) of 11,392 participants from The Cancer\n",
      "Genome Atlas PanCancer Atlas cohort ( https://gdc.cancer.gov/about-data/publications/PanCanStemness-2018 ).REAGENT or RESOURCE SOURCE IDENTIFIER\n",
      "Software and Algorithms\n",
      "Workﬂow to reproduce the stemness index This paper https://bioinformaticsfmrp.github.io/PanCanStem_Web/\n",
      "CIBERSORT Gentles et al., 2015 https://precog.stanford.edu/\n",
      "R 3.3.1 R Development Core\n",
      "Team, 2017https://www.R-project.org\n",
      "ggplot2 (v2.2.1) Wickham, 2009 https://cran.r-project.org/web/packages/ggplot2/index.html\n",
      "gelnet (v1.2.1) Sokolov et al., 2016 https://cran.r-project.org/web/packages/gelnet/index.html\n",
      "GSEA Subramanian et al., 2005 https://software.broadinstitute.org/gsea/index.jsp\n",
      "TCGAbiolinks (v2.4.3) Colaprico et al., 2016 http://bioconductor.org/packages/release/bioc/html/\n",
      "TCGAbiolinks.html\n",
      "ELMER (v1.4.1) Yao et al., 2015 http://bioconductor.org/packages/ELMER/\n",
      "fgsea (v1.2.1) Sergushichev, 2016 http://bioconductor.org/packages/release/bioc/html/fgsea.html\n",
      "Methylumi (v2.20.0) Davis et al., 2015 http://bioconductor.org/packages/release/bioc/html/\n",
      "methylumi.html\n",
      "MoonlightR (v1.2.0) Colaprico et al., 2018 http://bioconductor.org/packages/release/bioc/html/\n",
      "MoonlightR.html\n",
      "Amaretto Gevaert et al., 2013 http://med.stanford.edu/gevaertlab/software.html\n",
      "STATA (v13) StataCorp, 2013 https://www.stata.com/\n",
      "Deposited Data\n",
      "TCGA data NIH Genomic Data\n",
      "Commons (GDC)https://gdc.cancer.gov/about-data/publications/\n",
      "PanCanStemness-2018\n",
      "Progenitor Cell Biology Consortium (PCBC) Daily et al., 2017;\n",
      "Salomonis et al., 2016https://www.synapse.org/#!Synapse:syn1773109/wiki/54962\n",
      "Chromatin State (ChromHMM) Kundaje et al., 2015 http://www.roadmapepigenomics.org\n",
      "Stem Cells Validation set Nazor et al., 2012 mRNA expression (GEO: GSE30652) and DNA methylation\n",
      "(GEO: GSE30654)\n",
      "Glioma validation set Sturm et al., 2012 mRNA expression (GEO: GSE36245) and DNA methylation\n",
      "(GEO: GSE36278)\n",
      "Glioma validation set Turcan et al., 2012 GEO: GSE30339\n",
      "BRCA validation set Reyngold et al., 2014 GEO: GSE59000\n",
      "Deconvolution of breast cancer (BRCA) Onuchic et al., 2016 http://genboree.org/theCommons/projects/edec\n",
      "MET500 - Metastatic solid tumors Robinson et al., 2017 Database of Genotypes and Phenotypes (dbGaP) accession\n",
      "number phs000673.v2.p1\n",
      "Gliomas Single Cell RNA expression Tirosh et al., 2016 GEO: GSE70630\n",
      "BRCA Single Cell RNA expression Chung et al., 2017 GEO: GSE75688\n",
      "Cell173, 338–354.e1–e5, April 5, 2018 e1METHOD DETAILS\n",
      "DNA Methylation Data\n",
      "A total of 9,627 PanCancer TCGA samples across 33 different tumor types were available for DNA methylation using the robust Illu-mina HumanMethylation 450 (HM450) platform. TCGA samples included primary (8,471), recurrent (41), and metastatic tumor (394)\n",
      "tissues and a set of 721 non-tumor tissues.\n",
      "Level 3 data were downloaded from TCGA Data Portal using TCGAbiolinks functions GDCquery, GDCdownload and GDCprepare\n",
      "importing into R ( http://www.r-project.org ) for further analysis ( Colaprico et al., 2016 ).\n",
      "DNA methylation level 3 data are b-values that were calculated from pre-processed raw data using the methylumi Bioconductor\n",
      "package ( Davis et al., 2015 ). Pre-processing steps included background correction, dye-bias normalization, and calculation of\n",
      "b-values and detection p values. b-values range from zero to one, with zero indicating no DNA methylation and one indicating com-\n",
      "plete DNA methylation. A detection p value compares the signal intensity difference between the analytical probes and a set of nega-\n",
      "tive control probes on the array. Any data point with a corresponding p value greater than 0.01 is deemed not statistically signiﬁcantly\n",
      "different from background and is thus masked as ‘‘NA’’ in TCGA level 3 data. The data levels and the ﬁles contained in each data levelpackage are on the NIH Genomic Data Commons (GDC).\n",
      "In addition to TCGA data, we used a dataset of 99 human stem/progenitor cells from the Progenitor Cell Biology Consortium\n",
      "(PCBC) ( https://www.synapse.org/#!Synapse:syn1773109 ) to deﬁne stem cell signatures ( Daily et al., 2017; Salomonis et al.,\n",
      "2016 ). PCBC samples were proﬁled using the Illumina HumanMethylation 450 (HM450) platform and consisted of 4 embryonic\n",
      "stem cells (ESC), 40 induced pluripotent stem cells (iPSC), 22 stem cell (SC)-derived embryoid bodies (EB), 11 SC-derived mesoderm\n",
      "day 5 (MESO, mesothelioma), 11 SC-derived ectoderm (ECTO), and 11 SC-derived deﬁnitive endoderm (DE). We downloaded raw\n",
      "IDAT ﬁles from PCBC Genomic Data Commons and processed the data according to the TCGA standard level 3 protocoldescribed above.\n",
      "RNA Expression Data\n",
      "PanCancer TCGA RNA sequence level 3 normalized data were downloaded from the GDC Data Portal using TCGAbiolinks functions\n",
      "GDCquery, GDCdownload and GDCprepare importing into R ( http://www.r-project.org ) for further analysis ( Colaprico et al., 2016 ).\n",
      "A total of 10,852 samples across 33 tumor types were available, including primary (9,702), recurrent (45) and metastatic tumor (395)tissues and a set of 710 non-tumor tissues.\n",
      "We also downloaded PCBC RNA sequence data from the PCBC Synapse Portal ( https://www.synapse.org/#!\n",
      "Synapse:syn1773109 ), consisting 16 ESC, 77 iPSC, 66 SC-derived EB, 29 SC-derived MESO, 29 SC-derived ECTO, and 36\n",
      "SC-derived DE ( Daily et al., 2017; Salomonis et al., 2016 ).\n",
      "Stemness Index Derived Using OCLR\n",
      "To calculate a stemness index (si) based on mRNA expression or DNA methylation, we built a predictive model using one-classlogistic regression (OCLR) ( Sokolov et al., 2016 ) on the pluripotent stem cell samples (ESC and iPSC) from the PCBC dataset\n",
      "(Daily et al., 2017; Salomonis et al., 2016 ).\n",
      "For mRNA expression-based signatures, to ensure compatibility with the TCGA PanCancer cohort, we ﬁrst mapped the gene\n",
      "names from Ensembl IDs to Human Genome Organization (HUGO), dropping any genes that had no such mapping. The resulting\n",
      "training matrix contained 12,945 mRNA expression values measured across all available PCBC samples. For DNA methylation-\n",
      "based signatures, we used each of the signatures (probe set) described below.\n",
      "We mean-centered the data, then applied OCLR to just the samples labeled SC (which included both ESC and iPSC). We chose to\n",
      "use the one-class framework because of its robustness in the absence of the a ‘‘negative’’ class. The PCBC data does not have data\n",
      "for fully differentiated cells, and progenitor cell types might exhibit some of the stemness signals.\n",
      "Once the signature is obtained, it can be applied to score new samples. For RNA expression data, we computed Spearman cor-\n",
      "relations between the model’s weight vector and the new sample’s expression proﬁle. We advocate for the use of Spearman corre-\n",
      "lation over the more traditional dot product operation because it is more robust with respect to potential cross-dataset batch effects\n",
      "that may arise. For DNA methylation data, which follow the beta distribution, the samples were scored using the standard applicationof a linear model: f(x) = w\n",
      "^Tx+b .\n",
      "We validated our approach using leave-one-out cross-validation by withholding each SC sample in turn. A separate signature was\n",
      "then trained on all other SC samples and used to score the withheld sample as well as all the non-SC samples. The performance wasmeasured using the area under the curve (AUC) metric, which can be interpreted as the probability that the model correctly ranks a\n",
      "positive sample above a negative ( Agarwal et al., 2005 ). In our cross-validation experiment, every withheld SC sample was scored\n",
      "higher than all the non-SC samples, yielding an overall AUC of 1.0.\n",
      "We performed additional validation of the stemness signature by applying it to an external dataset composed of pluripotent stem\n",
      "cells (ESC and iPSC), somatic cells (17 distinct tissue types and several primary cell lines of diverse origin), and hydatidiform mole\n",
      "samples ( Nazor et al., 2012 ). The mRNA expression data for the study were downloaded from GEO (GEO: GSE30652) as were DNA\n",
      "methylation data (GEO: GSE30654). We observed that all of the SC samples were correctly scored above all of the somatic samples\n",
      "e2Cell173, 338–354.e1–e5, April 5, 2018by both platforms ( Figure 1 B). This is particularly striking for mRNA expression, because mRNA expression in study by the Nazor et al.\n",
      "was measured using microarrays, whereas the signature was trained using RNA-seq data.\n",
      "Having validated the signature by using cross-validation and external SC data, we then applied it to score the TCGA PanCancer\n",
      "cohort using the same Spearman correlation (RNA expression) or linear model (DNA methylation) operators. The indices were sub-sequently mapped to the [0,1] range by using a linear transformation that subtracted the minimum and divided by the maximum. The\n",
      "mapping was done to assist with interpretation as well as integration with the stemness indices derived from other data platforms\n",
      "(i.e., DNA methylation and mRNA expression).\n",
      "Additionally, we downloaded independent, non-TCGA datasets of gliomas [( Sturm et al., 2012 ) (GEO: GSE36245, GEO:\n",
      "GSE36278) and ( Turcan et al., 2012 ) (GEO: GSE30339)] and BRCA samples ( Reyngold et al., 2014 ) (GSE59000) and applied our met-\n",
      "rics to measure the stemness in the validation data. For mRNA expression, the preprocessing consisted of mapping the Illumina\n",
      "probe IDs (Illumina HumanHT-12 V3.0 platform) to HUGO symbols, and then reducing the signature and the external dataset to acommon set of genes. We then computed the Spearman correlation between the signature and the external samples. For DNA\n",
      "methylation, we applied the linear model.\n",
      "DNA Methylation Stemness Signatures\n",
      "Due to the magnitude of the available DNA methylation platform Inﬁnium HumanMethylation450 (HM450), we deﬁned DNA methyl-\n",
      "ation-based stemness signatures as a reduced input to the OCLR machine learning algorithm. For the DNA methylation-based stem-ness indices, three signatures were utilized, each deﬁning a distinct, biologically relevant, molecular phenotype of stemness. First, weperformed a supervised analysis between human pluripotent stem cells (ESC and iPSC) and stem cell-derived progenitors (embryoid\n",
      "bodies [EB], mesoderm [MESO], ectoderm [ECTO], and deﬁnitive endoderm [DE]) ( bvalue mean difference < /C00.4 and false discov-\n",
      "ery rate [FDR] < 10e-22; bvalue mean difference > 0.3 and false discovery rate [FDR] < 10e-17).All ‘rs’ and ‘ch’ probes were removed\n",
      "prior to analyses. To eliminate somatic tissue-speciﬁc probes, we removed probes that were consistently methylated (standard de-\n",
      "viation bvalue > 0.05) in non-tumor adult tissues available through TCGA. This resulted in a set of 62 pluripotent cell-speciﬁc and\n",
      "differentially methylated regions, which was then used as input for the OCLR to determine the stemness index for each TCGA tumorsample, named ‘‘differentially methylated probes-based stemness index’’ (DMPsi). Interestingly, most of these probes (85%) were\n",
      "positioned within intergenic regions known as open seas ( Figure S2 A).\n",
      "Second, we deﬁned a stem cell signature associated with genomic enhancer elements. Enhancers have been shown to be a crit-\n",
      "ically relevant functional element for deﬁning gene target expression and chromatin organization. For this, we downloaded Chromatin\n",
      "State data (ChromHMM) from the NIH Roadmap Epigenomics Consortium ( http://www.roadmapepigenomics.org ), which deﬁned 18\n",
      "chromatin states (based on 6 different histone marks: H3K4me3, H3K27ac, H3K4me1, H3K36me3, H3K9me3, and H3K27me3)across 98 different cell types ( Kundaje et al., 2015 ). Brieﬂy, by using ChromHMM data we mapped the HM450 probes to the chro-\n",
      "matin states in each individual cell type; then we identiﬁed genomic regions corresponding to active enhancers that are speciﬁc to\n",
      "pluripotent stem cell states (ESC and iPSC), meaning that each region was deﬁned as active enhancers (according to their states:\n",
      "9-EnhA1 and 10-EnhA2 ( Kundaje et al., 2015 )) in all pluripotent stem cells (n = 9) whereas not enhancer (enhancer in less than\n",
      "25% of non-pluripotent stem cells (n = 89)) in non-pluripotent stem cells. We identiﬁed 82 DNA methylation probes of the HM450\n",
      "platform that mapped to enhancer elements and considered them to be a DNA methylation-based pluripotent stem cell enhancer\n",
      "signature, which was then used as input for the OCLR to evaluate stemness signatures for TCGA samples, named ‘‘enhancer-basedstemness index’’ (ENHsi) ( Figure S2 A).\n",
      "Third, we applied ELMER (Enhancer Linking by Methylation/Expression Relationships), an R/Bioconductor package ( Yao et al.,\n",
      "2015 ) that uses DNA methylation to identify enhancer elements and correlates enhancer state with expression of nearby genes to\n",
      "identify putative transcriptional targets. Using ELMER, we compared pluripotent stem cells (ESC and iPSC) to stem cell-derived pro-genitors (EB, MESO, ECTO, DE) from PCBC and identiﬁed 87 CpGs that were hypomethylated in the pluripotent state (ESC and iPSC)\n",
      "compared to stem cell-derived progenitors and that potentially regulate 103 genes. We conﬁrmed the importance of these probe-\n",
      "gene pair targets by identifying that the SOX2-OCT4 transcription factor binding motif was among the most highly enriched signa-tures within these elements (+/ /C0250 bp from the center). The SOX2-OCT4 complex is an important master regulator of pluripotency\n",
      "and stemness. We then derived a new set of signatures using the OCLR and deﬁned TCGA samples’ stemness as ‘‘epigenetically\n",
      "regulated stemness indices’’ for each molecular feature (RNA expression-based Epigenetically regulated-mRNAsi [EREG-mRNAsi]and DNA methylation-based [EREG-mDNAsi]).\n",
      "Because there was high concordance among the three DNA methylation-based indices (DMPsi, ENHsi, and EREG-mDNAsi)\n",
      "(not shown) and each contributes important and complementary biological relevance to stemness, we combined the three stemnesssignatures (total of 219 probes) and derived a comprehensive DNA methylation index, named mDNAsi ( Figure S2 A). The lists of\n",
      "probes and genes used to derive the stemenss indices are provided on the publication portal accompanying this publication\n",
      "(https://gdc.cancer.gov/about-data/publications/PanCanStemness-2018 ).\n",
      "Stemness versus Molecular and Clinical Features\n",
      "To evaluate the performance of our stemness indices across the entire TCGA cohort, we performed an enrichment analysis by sorting\n",
      "TCGA samples by stemness index for each tumor type and looked for associations with all available genomic features (by usingcomprehensive mutation data [MC3]), molecular features (previously published TCGA molecular subtypes available at TCGAbiolinks\n",
      "Cell173, 338–354.e1–e5, April 5, 2018 e3package ( http://bioconductor.org/packages/release/bioc/html/TCGAbiolinks.html )(Colaprico et al., 2016; Silva et al., 2017 ), through\n",
      "the function ‘‘PanCancerAtlas_subtypes(),’’ which provides full access to the curated matrix used for this study), and clinical features\n",
      "(more than 10,000 features). We used the fgsea R/Bioconductor package to compute the enrichment scores ( Sergushichev, 2016 ).\n",
      "Brieﬂy, for each tumor type we ranked the TCGA samples according to their stemness index (from -low to -high stemness index) andtested if any particular genomic/molecular/clinical feature was associated with either -low or -high stemness index in a non-random\n",
      "behavior. We performed 10,000 permutations for each parameter analyzed to calculated our enrichment score. We then normalized\n",
      "the enrichment scores to mean enrichment of random samples of the same size (NES - normalized enrichment score). Tables con-taining all the results can be accessed at https://gdc.cancer.gov/about-data/publications/PanCanStemness-2018 . In addition, an\n",
      "interactive portal with the results across all tumor samples/types versus mDNAsi and mRNAsi can be accessed at https://\n",
      "bioinformaticsfmrp.github.io/PanCanStem_Web/ where the user can search for any gene or molecular/clinical feature of interest.\n",
      "Stemness versus Clinical Predictors\n",
      "The associations between the three stemness indices and overall survival (OS) and progression free survival (PFS) in different tumors\n",
      "were evaluated in two stages. First, the proportional hazard (PH) model with the index as a single continuous covariate was used totest whether there was a statistically signiﬁcant effect on OS or PFS. Given that, for each outcome, the effects of the three indices\n",
      "were tested for 33 cancer types. The signiﬁcance level of the tests was adjusted for multiple testing to control the overall type I error\n",
      "probability at 5%. In the next stage, the cancer types for which at least one index showed a statistically signiﬁcant association witheither OS or PFS were analyzed in more detail by using a multivariable PH model that included relevant clinical factors. Moreover, themodel included a functional form of the index obtained by using degree-2 fractional polynomials ( Royston and Altman, 1994 ). The\n",
      "plausibility of the PH assumption was checked by using the test based on the scaled Schoenfeld residuals ( Therneau and Grambsch,\n",
      "2000 ). The analyses were conducted using STATA v13 software.\n",
      "To select the clinical factors for inclusion in the PH model used in the second stage of the OS/PFS analysis for selected cancer\n",
      "types, a detailed analysis of the association between the stemness indices and demographic and clinical features (such as sex,\n",
      "age, race, stage, grade, etc.) was carried out by using linear models. mRNAsi and EREG-mRNAsi were analyzed on the original scale,while mDNAsi was transformed logarithmically to make its distribution more symmetric. The ﬁt of the constructed models was as-\n",
      "sessed by using residual plots. The analyses were conducted using STATA v13 software.\n",
      "The screening of the association between the stemness indices and OS ( Figure 4 E) by using univariable proportional hazard (PH)\n",
      "models indicated a statistically signiﬁcant (using p values adjusted for multiple testing) effect of mRNAsi on OS for LGG (p < 0.0001)\n",
      "and STAD (p = 0.005) and on PFS for GBM (p = 0.04), LGG (p < 0.0001), LIHC (p = 0.05), STAD (p = 0.04), and UCEC (p = 0.03). For\n",
      "mDNAsi, an effect on OS was found for LGG (p < 0.0001) and on PFS for kidney renal papillary cell carcinoma (KIRP) (p = 0.04) andLGG (p < 0.0001). Finally, for EREG-mRNAsi, a statistically signiﬁcant effect on OS was found for ACC (p = 0.005), KIRC (p = 0.008),and LGG (p = 0.03), and on PFS for ACC (p = 0.03), LGG (p = 0.03), and UCEC (p = 0.04). In these selected cases, multivariable an-\n",
      "alyses were conducted (using STATA v13 software), which took into account the effect of clinical factors. The analyses conﬁrmed (by\n",
      "using unadjusted p values) the effect of mRNAsi on OS for STAD (p = 0.0001) and for GBM/LGG (p = 0.002) and the effect on PFS forGBM/LGG (p = 0.008) and LIHC (p = 0.002). For mDNAsi, the effect on PFS in KIRP was conﬁrmed (p = 0.0001), while for EREG-\n",
      "mRNAsi, the effect on PFS in UCEC was conﬁrmed (p = 0.05). These conﬁrmed results indicate that the indices have a potential\n",
      "role as novel, independent prognostic factors for the indicated tumor types.\n",
      "Compounds Targeting with Cancer Stemness\n",
      "To determine which target drugs might be useful against cancer stem cells, we used the Broad Institute’s Connectivity Map build 02(CM) ( Lamb et al., 2006 ), a public online tool ( https://portals.broadinstitute.org/cmap/ ) (with registration) that allows users to predict\n",
      "compounds that can activate or inhibit based on a gene expression signature.\n",
      "To further investigate about mechanism of actions (MoA) and drug-target we performed speciﬁc analysis within Connectivity Map\n",
      "tools ( https://clue.io/ )(Subramanian et al., 2017 ).\n",
      "Using Connectivity Map (Query) in May 2017 having data available from a collection of cell lines (MCF, PC3, HL60 and SKMEL5)\n",
      "and 164 compounds as small molecules perturbagens. We obtained 33 mRNA expression signatures (one for each cancer type) by\n",
      "applying a differential expression analysis to samples with high mRNAsi and low mRNAsi, using the function TCGAanalyze_DEA fromthe the R/Bioconductor package TCGAbiolinks version 2.5.9 ( Colaprico et al., 2016 ), carrying edgeR pipeline. The table with differ-\n",
      "entially expressed genes is reported as Table S3 . Due to a limitation of the Connectivity Map tool that matches gene symbol and\n",
      "HG-U133A probe set (eg 200800_s_at) GPL96 platform ID, we had to remove duplicate IDs after sorting by decreasing jlogFC j.\n",
      "We selected the top 1000 genes (500 upregulated and 500 downregulated) where the number of differentially expressed genes\n",
      "was enough or considering the aggregation of upregulated or downregulated genes.\n",
      "Connectivity MAP is a method similar to GSEA analysis and follows a 4 step approach: (i) looking for similarity between a query\n",
      "signature (diff.expr. genes) and expression proﬁles present in the dataset using pattern-matching strategy based on Kolmogorov-Smirnov test (ii) rank-ordering the list of genes according their diff.expr. relative to the control from the above expression proﬁles\n",
      "with signiﬁcantly similarity (iii) comparison of each rank-ordered list with a query signature to specify when upregulated query genes\n",
      "appear in the proximity of the top of the list or near the bottom (‘‘positive connectivity’’) or vice versa (‘‘negative connectivity’’) pro-ducing an Enrichment Score (ES) from /C01 to 1. (iv) All instances in the database are then ranked according to their connectivity\n",
      "e4Cell173, 338–354.e1–e5, April 5, 2018scores; those at the top are most strongly correlated to the query signature, and those at the bottom are most strongly anticorrelated.\n",
      "For each cancer type we obtained two tables that applied the Connectivity Map’s ﬁndings to stemness mRNA expression signa-\n",
      "tures, namely, ‘‘detailed results’’ and ‘‘permuted results.’’ We used the permuted results and ﬁlter (with p < 0.05), to identify an\n",
      "average of 74 compounds per tumor type that are predicted to repress or activate the stemness signature ( Table S4 A).\n",
      "Connectivity Map (CMap) was recently updated (September 2017) ( Subramanian et al., 2017 ), providing the end-users new func-\n",
      "tionalities and new graphical interface as web-server, previous registration ( https://clue.io/ ) allowing easily the extraction of drug-\n",
      "interaction knowledge using as input a signature of genes or compounds.\n",
      "The new interface ( https://clue.io/ ), provided 7 different analysis (query, touchstone, proteomics query, command, data library,\n",
      "repurposing, morpheus).\n",
      "In particular CMap Query it is a tool for perturbagens that give rise to similar (or opposing) expression signatures, for a technical\n",
      "limit, the CMap Query 2017 allows only to upload 150 genes max for upregulated genes and 150 genes for downregulated genes. Forthis reason we considered the results analyzed in May 2017 using 500 genes for up-downregulated genes.\n",
      "QUANTIFICATION AND STATISTICAL ANALYSISR version 3.3.1 was used for all statistical analyses, unless speciﬁed otherwise. The statistical details of all experiments are reported\n",
      "in the ﬁgure legends and ﬁgures, including statistical analysis performed, statistical signiﬁcance and exact n values.\n",
      "To identify differentially methylated DNA methylation probes, we used the Wilcoxon test followed by multiple testing using the Ben-\n",
      "jamini-Hochberg (BH) method to estimate false discovery rate ( Benjamini and Hochberg, 1995 ).\n",
      "To identify proteins and microRNAs differentially expressed between tumors with low versus high stemness index, we used a t test\n",
      "followed by multiple testing using BH.\n",
      "P values for the association between stemness index and continuous clinical data were also computed using a t test followed by\n",
      "multiple testing using BH.\n",
      "DATA AND SOFTWARE AVAILABILITY\n",
      "All data are available on the NIH Genomic Data Commons (GDC), https://gdc.cancer.gov/about-data/publications/\n",
      "PanCanStemness-2018 .\n",
      "The workﬂow to reproduce the stemness index, including downloading PCBC and TCGA PanCan33 datasets, training a stemness\n",
      "signature, and applying it to score TCGA samples can be accessed at https://bioinformaticsfmrp.github.io/PanCanStem_Web/ .\n",
      "An interactive portal with the results for enrichment of molecular and clinical features and Stemness Indices across all tumor sam-\n",
      "ples/types can be accessed at https://bioinformaticsfmrp.github.io/PanCanStem_Web/ where the user can search for any gene or\n",
      "molecular/clinical feature of interest.\n",
      "Cell173, 338–354.e1–e5, April 5, 2018 e5Supplemental Figures\n",
      "Figure S1. Validation of mRNAsi and mDNAsi, Related to Figures 1 and3\n",
      "(A) Comparison of mRNAsi and mDNAsi values, as scored by signatures learned from PCBC data by OCLR (x axis) and Random Forest (y axis).\n",
      "(B) Validation of mRNAsi in non-TCGA breast cancer (BRCA) samples ( Reyngold et al., 2014 ) to deﬁne stemness status. Stratiﬁcation of mRNAsi according to\n",
      "histology of TCGA samples (left) and of the validation cohort (right). IDC - invasive ductal carcinoma; ILC - invasive lobular carcinoma.(C) Validation of stemness indices (mDNAsi and mRNAsi) in non-TCGA glioma samples ( Sturm et al., 2012; Turcan et al., 2012 ) using our metric to deﬁne stemness\n",
      "status. The validation sets were previously classiﬁed into molecular subtypes of TCGA gliomas. Stratiﬁcation of stemness indices according to mole cular\n",
      "subtypes in the TCGA (left) and in the validation cohort (right).(D) Correlation of stemness indices and estimated cell composition derived from deconvolution of tumor epigenomic features in TCGA breast cancer sa mples\n",
      "(Onuchic et al., 2016 ).\n",
      "(legend continued on next page)(E) Stemness indices in tumor versus non-tumor samples across tissue type. Top plots show mRNAsi and bottom plots show mDNAsi. See list of Abbreviatio ns of\n",
      "the TCGA Tumor Types.(F) Correlation of stemness indices across TCGA tumor types. Tumor types on the x axis are sorted by low-to-high correlation between mDNAsi and mRNAsi . See\n",
      "list of Abbreviations of the TCGA Tumor Types. mRNAsi, RNA-based stemness index. mDNAsi, DNA methylation-based stemness index. EREG-mRNAsi,epigenetically regulated-mRNAsi.Figure S2. Validation of mRNAsi and mDNAsi and Correlation of mRNAsi with EMT, Related to Figures 1 and2\n",
      "(A) Heatmaps of DNA methylation levels of probe signature used to derive the mDNAsi in: Progenitor Cell Biology Consortium (PCBC) dataset (left), TCG A normal\n",
      "adjacent tissue (middle), and TCGA tumors (right). Columns represent samples. TCGA tumor samples are sorted from lowest to highest mDNAsi. Rows repr esent\n",
      "the DNA methylation probe signature (219 probes), which is a combination of three DNA methylation-based signatures: differentially methylated pro bes stemness\n",
      "index (DMPsi), enhancer stemness index (ENHsi), and epigenetically regulated-mDNAsi (EREG-mDNAsi) (see STAR Methods for detail). Annotation of probes are\n",
      "provided in the left of the heatmaps. First column (mDNAsi) indicates to which signature the probe belong. Second column (genomic location) indicate s the\n",
      "genomic location annotation of each probe from the UCSC genome browser. Chromatin State columns indicates the chromatin state of pluripotent stem ce lls\n",
      "(iPSC and ESC) from Roadmap (n = 9) ( Kundaje et al., 2015 ).\n",
      "(legend continued on next page)(B) Scatterplots showing correlation of mDNAsi (x axis) and mRNAsi (y axis) by tumor type. Samples are colored according to TCGA classiﬁcation in: pri mary,\n",
      "recurrent, metastatic, and non-tumors.(C) Correlation between mRNAsi and mRNA expression of epithelial-to-mesenchymal transition (EMT) markers. The markers are grouped according to wh ether\n",
      "they are associated with epithelial (pink) or mesenchymal (teal blue) phenotype.Figure S3. Mutation Enrichment Analysis in Correlation with the mRNAsi and mDNAsi, Related to Figures 3 and4\n",
      "(A) (Left Panel) Comparison of normalized enrichment scores (NES) in mDNAsi (x axis) and mRNAsi (y axis) computed by tumor type in the mutation enrichm ent\n",
      "analysis. Positive and negative NES entail enrichment and depletion of a given mutation being associated with a high stemness index, respectively. T he quadrants\n",
      "and colors represent genes with agreement (I) or disagreement (II) between the two distinct indices and being associated only with mDNAsi (III) or ass ociated only\n",
      "with mRNAsi (IV) between the two distinct indices (V, not signiﬁcant). (Right Panel) Stemness indices stratiﬁed by representative mutation status a nd tumor types.\n",
      "(B) Relationship of mutations in epigenetic modiﬁer genes and stemness indices mRNAsi (top) and mDNAsi (bottom). NES, normalized enrichment scores .Figure S4. Correlation of miRNA and Protein Expression against mRNAsi and mDNAsi, Related to Figures 3 and4.\n",
      "(A) (Left Panel) Comparison of miRNA expression associated with mDNAsi and mRNAsi. Log10-normalized FDR-adjusted P value is plotted for mDNAsi (x ax is)\n",
      "and mRNAsi (y axis) for each miRNA probe for each tumor type. If a particular miRNA is upregulated in samples with a high stemness index, the values aremultiplied by /C01. The quadrants and colors represent proteins with agreement or disagreement between the two distinct indices. (Right Panel) Correlation of\n",
      "stemness indices and representative miRNAs for selected tumor types.(B) Comparison of protein expression associated with mDNAsi and mRNAsi (left panel). Log10 (FDR-adjusted P value) is plotted for mDNAsi (x axis) and m RNAsi\n",
      "(y axis) for each protein for each tumor type. If the protein is upregulated in high stemness index, /C01 is multiplied to log10 providing positive values. The quadrants\n",
      "and colors represent proteins with agreement or disagreement between the two distinct indices. Right, correlation scores of stemness indices and re presentative\n",
      "proteins and tumor types.\n",
      "(C) Correlation of ANNEXIN1 and FOXM1 with mDNAsi (top) and mRNAsi (bottom) across tumor types.Figure S5. Tumor Pathology and Clinical Outcome in Correlation with the mRNAsi and mDNAsi, Related to Figures 3 and4.\n",
      "(A and B) Tumor pathology grade associations with mRNAsi (A) and mDNAsi (B) by TCGA tumors types.\n",
      "(C) OS and PFS curves for four (quartile-based) categories of the relevant indices. The ﬁgure also includes the estimated (solid line) functional for ms of the\n",
      "dependence of the logarithm of hazard ratio (log-HR) of OS or PFS on the indices. The forms in general correspond to the ordering of the survival curves. For\n",
      "selected indices and outcomes the estimated functional form suggests a non-monotonic relationship between log-HR and the index.Figure S6. Evaluation of CIBERSORT Estimates of Immune Cell Subpopulation Fractions in the Context of mRNAsi and mDNAsi, Related to\n",
      "Figure 6\n",
      "(A) Plots show correlations between the mRNAsi (top) and mDNAsi (bottom) and the estimated fraction of cells for a selected panel of immune cell types.\n",
      "Additional correlations of stemness with estimated immune cell activity, computed as the difference between activated and resting population frac tions, are\n",
      "included for NK cells, CD4+ T cells and macrophages. Correlation with PD-L1 protein expression is included for reference and the tumor types are sorte d ac-\n",
      "cording to it.(B) Moonlight dynamic recognition analysis to identify associations with the mRNAsi across tumor types.See also Table S3 .(legend on next page)Figure S7. Connectivity Map, Related to Figure 7\n",
      "Heatmap showing each compound (perturbagen) from the Connectivity Map that share gene targets in (rows). Sorted by descending number of targets.\n",
      "See also Table S3 .\n",
      "```\n",
      "\n"
     ]
    }
   ],
   "source": [
    "console.log(reorgPrompt(rawPaper))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 13,
   "metadata": {},
   "outputs": [],
   "source": [
    "import OpenAI from 'https://deno.land/x/openai@v4.68.1/mod.ts';"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 17,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Current directory: /Users/danielgeorge/Documents/work/random/hackathon/paper-to-lecture\n",
      "Environment files found:\n",
      "\n",
      "Loading file: .env\n",
      "The .env file contains content.\n",
      "Environment variables loaded successfully.\n"
     ]
    }
   ],
   "source": [
    "async function getEnvFiles() {\n",
    "  try {\n",
    "    const currentDir = Deno.cwd();\n",
    "    console.log('Current directory:', currentDir);\n",
    "    const envPath = '/Users/danielgeorge/Documents/work/random/hackathon/paper-to-lecture/experiments/.env';\n",
    "    const content = await Deno.readTextFile(envPath);\n",
    "    return [{ name: '.env', content }];\n",
    "  } catch (error) {\n",
    "    console.error('Error reading .env file:', error);\n",
    "    return [];\n",
    "  }\n",
    "}\n",
    "\n",
    "const envFiles = await getEnvFiles();\n",
    "console.log('Environment files found:');\n",
    "if (envFiles.length === 0) {\n",
    "  console.log('No .env file found.');\n",
    "} else {\n",
    "  const envFile = envFiles[0];\n",
    "  console.log(`\\nLoading file: ${envFile.name}`);\n",
    "  if (envFile.content.trim()) {\n",
    "    console.log('The .env file contains content.');\n",
    "  } else {\n",
    "    console.log('The .env file is empty.');\n",
    "  }\n",
    "  \n",
    "  // Load the environment variables\n",
    "  const envVars = envFile.content.split('\\n').reduce((acc, line) => {\n",
    "    const [key, value] = line.split('=');\n",
    "    if (key && value) {\n",
    "      acc[key.trim()] = value.trim();\n",
    "    }\n",
    "    return acc;\n",
    "  }, {});\n",
    "\n",
    "  // Set the environment variables\n",
    "  Object.entries(envVars).forEach(([key, value]) => {\n",
    "    Deno.env.set(key, value);\n",
    "  });\n",
    "\n",
    "  console.log('Environment variables loaded successfully.');\n",
    "}\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 19,
   "metadata": {},
   "outputs": [
    {
     "ename": "",
     "evalue": "",
     "output_type": "error",
     "traceback": [
      "\u001b[1;31mThe Kernel crashed while executing code in the current cell or a previous cell. \n",
      "\u001b[1;31mPlease review the code in the cell(s) to identify a possible cause of the failure. \n",
      "\u001b[1;31mClick <a href='https://aka.ms/vscodeJupyterKernelCrash'>here</a> for more info. \n",
      "\u001b[1;31mView Jupyter <a href='command:jupyter.viewOutput'>log</a> for further details."
     ]
    },
    {
     "ename": "",
     "evalue": "",
     "output_type": "error",
     "traceback": [
      "\u001b[1;31mCanceled future for execute_request message before replies were done"
     ]
    },
    {
     "ename": "",
     "evalue": "",
     "output_type": "error",
     "traceback": [
      "\u001b[1;31mCanceled future for execute_request message before replies were done. \n",
      "\u001b[1;31mView Jupyter <a href='command:jupyter.viewOutput'>log</a> for further details."
     ]
    }
   ],
   "source": [
    "import { encode } from \"https://deno.land/x/tiktoken@1.0.3/mod.ts\";\n",
    "\n",
    "function truncatePaper(paper, maxTokens) {\n",
    "    const encoder = encode();\n",
    "    const tokens = encoder.encode(reorgPrompt(paper));\n",
    "    \n",
    "    if (tokens.length <= maxTokens) {\n",
    "        return paper;\n",
    "    }\n",
    "\n",
    "    const lines = paper.split('\\n');\n",
    "    let truncatedPaper = '';\n",
    "    let currentTokens = 0;\n",
    "\n",
    "    for (const line of lines) {\n",
    "        const lineTokens = encoder.encode(reorgPrompt(line)).length;\n",
    "        if (currentTokens + lineTokens > maxTokens) {\n",
    "            break;\n",
    "        }\n",
    "        truncatedPaper += line + '\\n';\n",
    "        currentTokens += lineTokens;\n",
    "    }\n",
    "\n",
    "    return truncatedPaper.trim();\n",
    "}\n",
    "\n",
    "const openai = new OpenAI({\n",
    "    apiKey: Deno.env.get('OPENAI_API_KEY')\n",
    "});\n",
    "\n",
    "const truncatedPaper = truncatePaper(rawPaper, 25000);\n",
    "\n",
    "const response = await openai.chat.completions.create({\n",
    "    model: 'o1-preview',\n",
    "    messages: [{role: 'user', content: reorgPrompt(truncatedPaper)}]\n",
    "});"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 52,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "```\n",
      "SUMMARY\n",
      "\n",
      "Cancer progression involves the gradual loss of a differentiated phenotype and acquisition of progenitor and stem-cell-like features. Here, we provide novel stemness indices for assessing the degree of oncogenic dedifferentiation. We used an innovative one-class logistic regression (OCLR) machine-learning algorithm to extract transcriptomic and epigenetic feature sets derived from non-transformed pluripotent stem cells and their differentiated progeny. Using OCLR, we were able to identify previously undiscovered biological mechanisms associated with the dedifferentiated oncogenic state. Analyses of the tumor microenvironment revealed unanticipated correlation of cancer stemness with immune checkpoint expression and infiltrating immune cells. We found that the dedifferentiated oncogenic phenotype was generally most prominent in metastatic tumors. Application of our stemness indices to single-cell data revealed patterns of intra-tumor molecular heterogeneity. Finally, the indices allowed for the identification of novel targets and possible targeted therapies aimed at tumor differentiation.\n",
      "\n",
      "INTRODUCTION\n",
      "\n",
      "Stemness, defined as the potential for self-renewal and differentiation from the cell of origin, was originally attributed to normal stem cells that possess the ability to give rise to all cell types in the adult organism. Cancer progression involves gradual loss of a differentiated phenotype and acquisition of progenitor-like, stem-cell-like features. Undifferentiated primary tumors are more likely to result in cancer cell spread to distant organs, causing disease progression and poor prognosis, particularly because metastases are usually resistant to available therapies (Friedmann-Morvinski and Verma, 2014; Ge et al., 2017; Shibue and Weinberg, 2017; Visvader and Lindeman, 2012).\n",
      "\n",
      "An increasing number of genomic, epigenomic, transcriptomic, and proteomic signatures have been associated with cancer stemness. Those molecular features are causally connected to particular oncogenic signaling pathways that regulate transcriptional networks that sustain the growth and proliferation of cancer cells (Ben-Porath et al., 2008; Eppert et al., 2011; Kim et al., 2010). Transcriptional and epigenetic dysregulation of cancer cells frequently leads to oncogenic dedifferentiation and acquisition of stemness features by altering core signaling pathways that regulate the phenotypes of normal stem cells (Bradner et al., 2017; Young, 2011). Cell-extrinsic mechanisms can also affect maintenance of the undifferentiated state, largely through epigenetic mechanisms. Tumors comprise a complex, diverse, integrated ecosystem of relatively differentiated cancer cells, cancer stem cells, endothelial cells, tumor-associated fibroblasts, and infiltrating immune cells, among other cell types. The microenvironment of a tumor, considered as a pathologically formed “organ,” is frequently characterized by hypoxia, as well as by abnormal levels of various cytokines, growth factors, and metabolites (Lyssiotis and Kimmelman, 2017). It provides numerous opportunities for cell-cell signals to modulate the epigenome and expression of stem-cell-like programs in cancer cells, frequently independent of their genetic backgrounds (Gingold et al., 2016).\n",
      "\n",
      "Over the last decade, The Cancer Genome Atlas (TCGA) has illuminated the landscapes of primary tumors by generating comprehensive molecular profiles composed of genomic, epigenomic, transcriptomic, and (post-translational) proteomic characteristics (Hoadley et al., 2014; Tomczak et al., 2015), along with histopathological and clinical annotations. The resulting resource enabled us to analyze cancer stemness quite extensively in almost 12,000 samples of 33 tumor types.\n",
      "\n",
      "First, we defined signatures to quantify stemness using publicly available molecular profiles from normal cell types that exhibit various degrees of stemness. By multi-platform analyses of their transcriptome, methylome, and transcription-factor binding sites using an innovative one-class logistic regression (OCLR) machine-learning algorithm (Sokolov et al., 2016), we obtained two independent stemness indices. One (mDNAsi) was reflective of epigenetic features; the other (mRNAsi) was reflective of gene expression. We then identified associations between the two stemness indices and novel oncogenic pathways, somatic alterations, and microRNA (miRNA) and transcriptional regulatory networks. Those features correlated with, and perhaps govern, cancer stemness in particular molecular subtypes of TCGA tumors. Importantly, higher values for stemness indices were associated with biological processes active in cancer stem cells and with greater tumor dedifferentiation, as reflected in histopathological grade. Metastatic tumor cells appeared more dedifferentiated phenotypically, probably contributing to their aggressiveness. We also found tumor heterogeneity at the single-cell level by measuring stemness in transcriptome profiles obtained from individual cancer cells. Using CIBERSORT to profile immune cell types in TCGA tumors, we obtained insight into the interface of the immune system with stemness. Finally, we identified compounds specific to selected molecular targets and mechanisms that may eventually lead to novel treatments that trigger differentiation and exhaust the stemness potential of highly aggressive neoplasms.\n",
      "\n",
      "RESULTS\n",
      "\n",
      "**DNA-Methylation- and mRNA-Expression-Based Stemness Classifiers**\n",
      "\n",
      "We analyzed publicly available non-tumor and tumor datasets for which transcriptomic and epigenomic molecular profiles were available. We derived stemness indices using an OCLR algorithm trained on stem cell (ESC, embryonic stem cell; iPSC, induced pluripotent stem cell) classes and their differentiated ecto-, meso-, and endoderm progenitors. We chose OCLR because it does not penalize misclassification of stem-cell-derived progenitors at different stages of differentiation that still carry some of the undifferentiated features in their molecular profiles (its output was also validated against random forests). OCLR-based transcriptomic and epigenetic signatures were applied to TCGA datasets to calculate the mRNAsi and mDNAsi. Each stemness index (si) ranges from low (zero) to high (one) stemness. The tumor samples stratified by the indices were used for the integrative analyses.\n",
      "\n",
      "**mRNA Expression-Based Stemness Index**\n",
      "\n",
      "We validated the mRNAsi by applying it to an external dataset composed of both stem cells and somatic differentiated cells and by scoring molecular subtypes of breast cancers and gliomas that are characterized by different degrees of oncogenic dedifferentiation associated with pathology and clinical outcome. All stem cell samples attained higher stemness index values than samples from differentiated cells. TCGA tumors display various degrees of cancer stemness as revealed by mRNAsi and mDNAsi. Germ-cell tumors, basal breast cancer, and lympho-hematopoietic (Ly-Hem) cancers displayed highly dedifferentiated phenotypes in comparison to other tumor types.\n",
      "\n",
      "Using gene set enrichment analysis (GSEA), we compared our signature to 16 gene sets that were associated with stemness in cancer and healthy cells in previous studies. These sets spanned 2,564 unique genes, with no two sets overlapping by more than 134 genes. In all cases, the published stemness gene sets were significantly enriched in mRNAsi. We found that “cancer hallmark” gene sets were significantly enriched, as were MYC targets, which significantly contributed to the positive side of the signature. This is consistent with MYC being one of the transcription factors that drive pluripotency in ESCs. Wingless-related integration site (Wnt)/β-catenin and TGF-β signaling pathways were significantly enriched on the negative side of the stemness signature. This negative enrichment does not imply absence of specific signals in cancer stem cells, but rather that this signaling is lower relative to stem-cell-derived progenitors, as captured by the signature weights. This is again consistent with other GSEA results, as both signaling pathways are known mediators of the epithelial-mesenchymal transition (EMT) mechanism. We also computed the correlation of mRNAsi against mRNA expression of published pan-cancer EMT markers, which revealed significant correlations for most tumors. This is consistent with the biology of ESCs, which grow as epithelioid, polygonal cells in vitro and epithelial cancer precursors having stem-like properties. Importantly, most TCGA samples are primary tumors of an epithelial phenotype. Most skin melanoma cases come from lymph nodes, and this tumor type shows higher expression of vimentin, a key marker of a mesenchymal phenotype. mRNAsi is positively correlated with other core stem cell factors: EZH2, OCT4, and SOX2. Finally, Moonlight analysis of the oncogenic signatures from the Molecular Signatures Database (MSigDB) further validated our gene-expression-based index and confirmed engagement of MYC and EZH2, along with E2F3, MTOR, and SHH in driving oncogenic dedifferentiation.\n",
      "\n",
      "**DNA Methylation-Based Stemness Index**\n",
      "\n",
      "We defined the mDNAsi using OCLR by combining (1) supervised classification between ESCs/iPSCs and their progenies, (2) stem cell signatures associated with pluripotency-specific genomic enhancer elements based on ChromHMM from Roadmap, and (3) ELMER, which uses DNA methylation to identify enhancer elements and correlates their state with the expression of nearby genes. 219 CpG probes were selected in training OCLR using the Progenitor Cell Biology Consortium (PCBC) datasets. By selecting probes previously defined to be active stemness-specific enhancers, we confirmed the ability of our approach to derive an mDNAsi. Since we focused exclusively on hypomethylated, functionally important CpG probes associated with stem cells, we further explored cis-activated genes.\n",
      "\n",
      "We scored each TCGA sample using the mDNAsi and used an external dataset to confirm that stem cells had higher mDNAsi than differentiated samples. TCGA tumor types show different degrees of an inferred dedifferentiated phenotype. Within these, individual tumor samples show variation for cancer stemness. As anticipated, TCGA samples derived from the primary tumors show higher cancer stemness indices compared to non-tumor samples obtained from adjacent normal tissue of origin. Most of our selected probes fell within non-promoter elements, yet the SOX2-OCT4 transcription factor binding motif is one of the most highly enriched signatures within these regions. The SOX2-OCT4 complex is a critical master regulator of pluripotency and stemness and is highly enriched in tumor samples with high mDNAsi.\n",
      "\n",
      "**Correlations of mRNAsi and mDNAsi**\n",
      "\n",
      "Since the inputs for mDNAsi and mRNAsi are not necessarily complementary, we explored stratification of glioma samples by the epigenetically regulated mRNAsi (EREG-mRNAsi), a stemness index generated using a set of stemness-related epigenetically regulated genes. The EREG-mRNAsi, based on both RNA expression and epigenetics, elucidates the discrepancy between mDNAsi and mRNAsi and shows a positive correlation with both indices. Both mRNAsi and mDNAsi show good correspondence for a majority of tumors. We observed major discrepancies in the case of brain lower grade glioma (LGG), thyroid carcinoma (THCA), and thymoma (THYM). For gliomas, mDNAsi is correlated positively with tumor pathology and clinical features, while mRNAsi shows a negative correlation. This result could arise from a high frequency of isocitrate dehydrogenase mutations (IDH1/2) and resulting DNA hypermethylation.\n",
      "\n",
      "**Stemness Index Can Stratify Recognized Undifferentiated Cancers**\n",
      "\n",
      "We examined breast invasive carcinoma (BRCA), acute myeloid leukemia (AML), and gliomas to study if the mRNAsi/mDNAsi predict stemness in poorly differentiated tumors.\n",
      "\n",
      "In BRCA, we found a strong association between the stemness index and known clinical and molecular features. The mRNAsi was highest in the basal subtype, known to exhibit an aggressive phenotype associated with an undifferentiated state. BRCA samples with high mRNAsi were more likely to be estrogen receptor (ER)-negative and enriched for FAT3 and TP53 mutations. We noted that high mRNAsi was associated with higher protein expression of FOXM1, CYCLINB1, and MSH6, as well as higher miRNA-200 family expression. Invasive lobular type of BRCA (ILC), characterized by better prognosis in comparison to invasive ductal carcinoma (IDC), has a lower mRNAsi. We also applied our indices to non-TCGA BRCA samples and found a similar correlation between mRNAsi and mDNAsi in those samples. Moreover, mRNAsi also stratified BRCA samples with distinct histology in this dataset. Using datasets with estimated tumor cell type composition provided by the epigenetic deconvolution method, we found that both mRNAsi and mDNAsi were more highly correlated with malignant epithelial cells than with normal epithelial cells, suggesting that our indices identify distinct cancerous epithelial cell populations characterized by different features or degrees of stemness.\n",
      "\n",
      "We found an association between the mRNAsi, RNA expression subtypes previously defined by TCGA, and the French-American-British (FAB) classification of AML. The mRNAsi showed the strongest correlation with the stage of myeloid differentiation of the AML samples. FAB subtypes M0 (undifferentiated), M1 (with minimal maturation), and M2 (with maturation) were characterized by high mRNAsi. In contrast, the M3 well-matured promyelocytic subtype, which is associated with benign chromosomal abnormalities and favorable clinical outcome, had low mRNAsi. High mRNAsi was associated with higher expression of miR-181c-3p, miR-22-3p, and miR-30b-3p.\n",
      "\n",
      "We found a strong association between high mDNAsi, high pathologic grade, and recently published molecular subtypes of glioma. mDNAsi was low in less-aggressive gliomas that are characterized by codel and glioma CpG methylator phenotype (G-CIMP)-high features and was highest in highly aggressive glioblastoma multiformes (GBMs) characterized by IDH mutations (G-CIMP low) and poor clinical outcome. Also, high mDNAsi is strongly associated with more aggressive classical and mesenchymal subtypes of GBM, suggesting that it can stratify tumors with distinct clinical outcomes. We also found that high mDNAsi was associated with mutations in NF1 and EGFR and infrequent mutations in IDH1, TP53, CIC, and ATRX, with higher expression of ANNEXIN-A1 protein and lower expression of ANNEXIN-A7 and with expression of the miR-200 family.\n",
      "\n",
      "We obtained similar results on non-TCGA glioma samples for which both mRNA expression and DNA methylation data were available. The negative correlation between mDNAsi and mRNAsi was restricted to LGG samples—specifically, the IDH mutant subtypes (G-CIMP high and codel). IDH1 mutations are known to reduce cell differentiation, and high values of the mRNAsi in a subset of IDH mutant gliomas might capture this phenomenon.\n",
      "\n",
      "**Pan-cancer Stemness Landscape**\n",
      "\n",
      "Next, we tested the ability of our indices to identify previously unexplored features of cancer stemness across all TCGA tumors. First, we performed an enrichment analysis by sorting all TCGA samples by stemness index for each tumor type and looking for associations with mutations and molecular and clinical features. The most salient associations of mRNAsi and mDNAsi are presented, while the following results of the comprehensive analyses are shown in the supplementary material: associations with mutations, associations with miRNA expression and protein abundance, associations with the tumor grading, and clinical outcome.\n",
      "\n",
      "**Correlations of mRNAsi and mDNAsi with Mutations in Genes, miRNA, and Expression of Proteins**\n",
      "\n",
      "We found a strong association of mDNAsi with known molecular subtypes, with somatic mutations in SETD2 and TP53 genes, and with tobacco smoking status in lung adenocarcinoma (LUAD). Current smokers and recently reformed smokers have higher mDNAsi than non-smokers or long-term reformed smokers. This suggests that the stemness of LUAD tumors might be activated in response to environmental stimuli such as smoking and might influence the aggressiveness of the tumor. We also found an association between mDNAsi and higher protein expression of CYCLINB1 and FOXM1, which is a pro-stemness transcription factor upstream of CYCLINB1. FOXM1 has been associated with dedifferentiation in pancreatic cancer cells, as well as tumor proliferation in the kidney and ovarian cancers. Our result suggests that it could be a driver of dedifferentiation and proliferation in breast and lung cancers. Stemness of LUAD tumors is also associated with lower expression of ANNEXIN-A1. ANNEXIN-A1 has been indicated as a differentiation marker in pancreatic and urothelial cancers; therefore, we suspect that the relationship between ANNEXIN and FOXM1 expression and tumor differentiation may extend to other tumor types.\n",
      "\n",
      "Analyses of head and neck squamous cell carcinoma (HNSC) samples revealed that high indices are correlated with NSD1 mutation, E-cadherin protein expression, miR-200-3p, and previously identified classical molecular subtypes. NSD1 mutation was recently linked in HNSC tumors to blockade of cellular differentiation and promotion of oncogenesis. Interestingly, miR-200 family members have been implicated in cancer initiation and metastasis, as well as self-renewal of healthy stem cells. HNSC tumors with high mDNAsi have reduced programmed death ligand 1 (PD-L1) protein level.\n",
      "\n",
      "In liver hepatocellular carcinoma (LIHC) samples, we found an association between mRNAsi and high pathological grade. Negative associations between mRNAsi and the probability of overall survival (OS) or progression-free survival (PFS) were detected. In contrast to the majority of tumor types, LIHC samples with high mRNAsi have low expression of miR-200 family members. The miR-200 family is known to be associated with progression of hepatocellular carcinoma, and the miR-200b-ZEB1 circuit has been suggested as a master regulator of stemness in these cancers. We found associations of mRNAsi with higher CYCLINB1 and ACC1 and with lower PD-L1 and ANNEXIN-A1 protein expression in LIHC. ACC1 was associated with pathomorphological markers of LIHC aggressiveness (vascular invasion and poor differentiation), and its upregulation was correlated with poor OS and disease recurrence in hepatocellular carcinoma patients. LIHC samples with high mRNAsi were associated with specific genomic alterations (e.g., TP53, CTNNB1, AXIN1).\n",
      "\n",
      "Detailed analyses of adrenocortical carcinoma (ACC) samples revealed an association between high mRNAsi and defined molecular subtypes, clinical stage, and mutations in PRKAR1A and TP53 genes. We found a positive correlation between mRNAsi and adrenal differentiation score that is based on expression of 25 genes that are important for adrenal function.\n",
      "\n",
      "**Stemness Indices Are Correlated with Tumor Pathology and Predictive of Clinical Outcome**\n",
      "\n",
      "We observed a positive correlation between tumor histology and pathology grading and both stemness indices for the majority of the TCGA cases. For mRNAsi, the most significant correlations were found for BRCA (IDC and ILC), cervical squamous cell carcinoma and endocervical adenocarcinoma (CESC), LIHC, pancreatic adenocarcinoma (PAAD), and uterine corpus endometrial carcinoma (UCEC). Interestingly, mRNAsi shows low values in GBM and stomach adenocarcinoma (STAD). On the other hand, mDNAsi strongly stratifies glioma by the pathology grade, culminating with the highest value for GBM. The reversed values of mDNAsi and mRNAsi in the case of gliomas were also evident in the clinical data analyses. We found a positive correlation between previously published glioma subtypes and mDNAsi, suggesting that mDNAsi might recapitulate prognostic molecular subtypes.\n",
      "\n",
      "The discordance between the mRNAsi and the mDNAsi for gliomas may be explained in part by the dominant genomic alteration associated with the LGG tumor type. Roughly 80% of LGG tumors carry an IDH1/2 mutation and, as demonstrated by our group and others, confer a genome-wide hypermethylator phenotype (G-CIMP). Given that the mDNAsi is driven primarily by low methylation levels associated with the stemness phenotype, the LGG tumors might resemble non-stem-like phenotypes, which are predominantly hypermethylated. The subgroup of G-CIMP with the lowest overall DNA methylation levels (G-CIMP low) is associated with the worst outcomes. Compared to G-CIMP-high tumors, G-CIMP-low tumors are known to be more proliferative, express cell-cycle-related genes, and have various stem-cell-like genomic features.\n",
      "\n",
      "**Cancer Stemness Indices Are Higher in Tumor Metastases and Reveal Intratumor Heterogeneity**\n",
      "\n",
      "The TCGA samples are derived mostly from primary tumors, except for skin melanoma, for which tissues are mostly metastatic lymph nodes. We used the mRNAsi to interrogate the MET500 dataset comprising expression profiles from 500 metastatic samples obtained from 22 different organs. In most cases, mRNAsi was significantly higher in metastatic samples compared to primary TCGA tumors. Prostate and pancreatic adenocarcinoma metastases had the most dedifferentiated phenotypes and are also more aggressive and resistant to therapies in contrast to primary tumors. Weaker association with the mRNAsi was due to a small number of available samples. Interestingly, testicular germ cell tumors (TGCTs) present the less differentiated phenotypes in primary tumors when compared to distant metastases. Primary TGCT tumor cells have high mRNAsi and may differentiate when metastasizing to distant organs. A similar trend was observed for STAD.\n",
      "\n",
      "Using another dataset, we found that mDNAsi was significantly higher in glioma samples obtained at first recurrence in contrast to primary gliomas. Our results reveal significant dedifferentiation of glioma cancer cells that contribute to glioma recurrence, which is frequently associated with poor prognosis and resistance to treatment. By taking advantage of single-cell transcriptome datasets, we used mRNAsi to probe tumor heterogeneity for oncogenic dedifferentiation of individual cancer cells. We revealed high variation of stemness in the glioma and breast primary tumors. Individual glioma cells showed higher variation of oncogenic dedifferentiation in comparison to breast cancer cells. Single cells from metastases had higher stemness index in breast cancer. Interestingly, the negative correlation of EMT signature and stemness that we observed in TCGA primary tumors was also found in metastatic samples.\n",
      "\n",
      "**Stemness Index Evaluated in the Context of Immune Response**\n",
      "\n",
      "We found that for many tumors, higher stemness indices are associated with a reduced leukocyte fraction and lower PD-L1 expression. For mDNAsi, the most distinctive negative correlations were found in the PanCan-12 squamous cluster (LUSC, lung squamous cell carcinoma; HNSC; BLCA, bladder urothelial carcinoma) and in GBM. For the mRNAsi, the highest negative correlation values were seen in GBM/LGG, prostate adenocarcinoma (PRAD), LIHC, and uterine carcinosarcoma (UCS) tumors. We expect that such tumors will be less susceptible to immune checkpoint blockade treatments due to insufficient immune cell infiltration or preexisting downregulation of the PD-L1 pathway, which makes further inhibition ineffective. Our findings are consistent with previous reports showing a strong correlation between PD-L1 protein expression and infiltration of CD8+ cytotoxic lymphocytes.\n",
      "\n",
      "We further explored correlations between stemness and immune microenvironment variables in the context of molecular subtypes of tumors. Several tumor types exhibit strong (positive or negative) correlations. Except for kidney renal clear cell carcinoma (KIRC), the association between stemness and PD-L1 expression and leukocyte fraction is readily apparent from the increasing and decreasing trends of individual variables across the molecular subtypes. For example, we found mesenchymal tumors to have the highest PD-L1 expression levels, the most significant leukocyte fractions, and the lowest mDNAsi compared to other HNSC subtypes, suggesting potential susceptibility to checkpoint blockade inhibitors. The use of immunotherapy for HNSC tumors is under active investigation with the recent FDA approval of pembrolizumab; however, whether the effectiveness of therapy is limited to specific HNSC molecular subtypes is not clear from those reports.\n",
      "\n",
      "To assess other relationships between stemness and tumor microenvironment, we computed correlations between stemness indices and individual types of immune cells. By applying CIBERSORT, we scored 22 immune cell types for their relative abundance in TCGA tumor samples. These cell types included natural killer (NK) cells, monocytes, macrophages, dendritic and mast cells, eosinophils, and neutrophils. We also obtained absolute estimates by scaling their relative abundance by overall leukocyte infiltration in each tumor as determined by ESTIMATE applied to DNA methylation data. For any given TCGA sample, we calculated the correlation between mDNAsi/mRNAsi and the estimated fraction of individual immune cell types. In addition to individual immune subpopulation fractions, we considered the functional activation of distinct cells by measuring the difference between activated and resting fractions of NK cells, CD4+ T cells, and macrophages. This approach was motivated by recent observations that activation of peripheral CD4+ T cells triggered by immunotherapy is responsible for the specific killing of tumor cells.\n",
      "\n",
      "Although the squamous cluster tumors had a negative correlation between stemness and the fraction of CD4+ T cell populations, the activation state of the CD4+ T cells was higher in dedifferentiated tumors. This finding is consistent with our observation that PD-L1 protein expression is lower in these tumors, suggesting again that immune checkpoint blockade might be ineffective, and an additional mechanism of immune evasion may be operative. The opposite trend is present in thymomas, where PD-L1 protein expression and the fraction of the CD4+ T cell population are positively correlated with tumor dedifferentiation. Likewise, the activation state of CD4+ T cells is lower in dedifferentiated tumors, suggesting that they might be more susceptible to immunotherapy treatments.\n",
      "\n",
      "**Connectivity Map Analysis Identifies Potential Compounds/Inhibitors Capable of Targeting the Stemness Signature**\n",
      "\n",
      "We employed the Connectivity Map (CMap), a data-driven, systematic approach for discovering associations among genes, chemicals, and biological conditions, to search for candidate compounds that might target pathways associated with stemness. We found enrichment for compounds associated with stemness in at least three cancer types. Five compounds are significantly enriched in more than 10 cancer types and have been reported to inhibit stemness-related tumorigenicity: the dopamine receptor antagonists thioridazine and prochlorperazine, the Wnt signaling inhibitor pyrvinium, the HSP90 inhibitor tanespimycin, and the protein synthesis inhibitor puromycin. Further, the telomerase inhibitor gossypol induced apoptosis and growth inhibition of cancer stem cells (CSCs), and histone deacetylase inhibitors such as trichostatin A (SAHA) reduced glioblastoma stem cell growth. According to our analysis, pyrvinium and puromycin could inhibit stemness in LUAD. We found several candidates with recognized anti-CSC activity for HNSC, including the aforementioned compounds. For LIHC, thioridazine, a prospective inhibitor of lung cancer stem cells, pyrvinium, puromycin, prochlorperazine, and others are potential compounds targeting undifferentiated tumors.\n",
      "\n",
      "CMap mode-of-action (MoA) analysis of the 74 compounds revealed 56 mechanisms of action shared by the above compounds. Five compounds (fluspirilene, pimozide, prochlorperazine, thioridazine, and trifluoperazine) shared the MoA of dopamine receptor antagonist. We observed that entinostat, trichostatin-a, and vorinostat shared MoA as HDAC inhibitors, and LY-294002, zaprinast, zardaverine share MoA as phosphodiesterase inhibitors.\n",
      "\n",
      "CMap target analysis revealed 212 distinct drug-target genes shared by the mentioned compounds. Eight genes are targets of five different compounds—namely, DRD2 (8 drugs), HTR2A (7 drugs), HRH1 (6 drugs), ADRA1A (5 drugs), CALM1 (5 drugs), CHRM3 (5 drugs), HTR1A (5 drugs), and HTR2C (5 drugs).\n",
      "\n",
      "Recent polypharmacology studies suggest the need to design compounds that act on multiple genes or molecular pathways. In this study, we observed similar mechanisms of action among different compounds, suggesting that selective therapies can target the undifferentiated phenotypes for selected cancer types.\n",
      "\n",
      "DISCUSSION\n",
      "\n",
      "This study is based on integrated analysis of cancer stemness in almost 12,000 primary human tumors of 33 different cancer types. We interrogated TCGA data for mutations, DNA methylation, expression of mRNA and miRNA, expression and post-translational modification of proteins, histopathological grade, and clinical outcome. Applying CIBERSORT, we gained insight into the tumor microenvironment and composition of immune cell infiltrates. By applying a machine-learning algorithm to molecular datasets from normal stem cells and their progeny, we developed two different molecular metrics of stemness and then used them to assess epigenomic and transcriptomic features of TCGA cancers according to their grade of oncogenic dedifferentiation. Ultimately, the analyses led us to potentially actionable targets (and their MoAs) as candidates for possible differentiation therapy of solid tumors and metastases. Our approach could be applied to longitudinal study of samples from primary, recurrent, and metastatic cancers, and gene expression signatures identified in the tumor samples can be used to interrogate CMap to suggest actionable targets and inhibitors for further analysis.\n",
      "\n",
      "To the best of our knowledge, this is the first study in which molecular PCBC datasets comprised of stem cells and defined populations of their differentiated progeny have been leveraged to develop a classification tool and machine-learning algorithm for analysis of a spectrum of human malignancies. A number of cancer stemness scores, based on genes that are differentially expressed between CSCs and non-CSCs, have been published and are relevant to clinical outcomes in AML (Eppert et al., 2011; Gentles et al., 2010; Ng et al., 2016). In those studies, gene sets enriched in ESCs (e.g., targets of NANOG, OCT4, SOX2, and c-MYC) were frequently overexpressed in poorly differentiated tumors compared with well-differentiated ones. In breast cancers, those gene sets were associated with high-grade estrogen receptor-negative, basal-like tumors and poor clinical outcome (Ben-Porath et al., 2008). Another web-based tool, StemChecker, uses a curated set of 49 published stemness signatures defined by gene expression, RNAi screens, transcription factor binding sites, text mining of the literature, and other computational approaches. But it has been tested only for pancreatic ductal adenocarcinoma. In that case, high expression of stemness genes correlated with poor prognosis (Pintó et al., 2015). All previous studies were transcriptome-based and limited to a narrow set of genes and a small number of tumor types.\n",
      "\n",
      "In the present study, we found oncogenic dedifferentiation to be associated with several characteristics: mutations in genes that encode oncogenes and epigenetic modifiers, perturbations in specific mRNA/miRNA transcriptional networks, and deregulation of signaling pathways. Cancer stemness also appeared to involve core expression of MYC, OCT4, SOX2, and other genes involved in the regulatory circuitry that underlies normal and malignant self-renewal potential. Our indices derived from mRNA expression and DNA methylation signatures reliably stratified tumors of known stemness phenotype. High mRNAsi was associated with basal breast carcinomas but also Her2 and lumB subtypes that are more aggressive than the hormone-dependent lumA group. In contrast, high mDNAsi was strongly associated with high-grade glioblastomas, poor OS, and PFS. The association between stemness signatures and adverse outcome for some tumor types, including gliomas, may reflect malignant cell origins or the impact of their microenvironment.\n",
      "\n",
      "Dedifferentiated cells can arise from different sources: from long-lived stem or progenitor cells that accumulate mutations in oncogenic pathways or via dedifferentiation from non-stem cancer cells that convert to CSCs through deregulation of developmental and/or non-developmental pathways. It is important to distinguish between the inherent stemness of CSCs and dedifferentiation induced by the tumor microenvironment. However, addressing that issue would require further validation beyond the scope of this study using other genomic datasets and/or laboratory experiments.\n",
      "\n",
      "Both stemness indices were lowest in normal cells, increased in primary tumors, and highest in metastases, consistent with the idea that tumor progression generally involves oncogenic dedifferentiation. Interestingly, we observed negative associations between stemness and EMT gene signatures. The relationship between EMT and stemness remains a hotly debated topic, with several studies showing that EMT is necessarily associated with stemness (Fabregat et al., 2016). However, most TCGA data are obtained from primary tumors, which exist in a pre-EMT state, since EMT is strongly associated with tumor progression and with metastasis for many tumor types. Cancer cells in many primary solid tumors are basically epithelial regardless of their degrees of dedifferentiation, but some cells in such contexts could acquire mesenchymal characteristics either by accumulating additional mutations or by undergoing epigenetic changes shaped by the tumor microenvironment. Those mesenchymal cells can traverse the underlying tissue, enter the bloodstream, and seed distant organs, where they reacquire an epithelial phenotype to form metastatic tumors.\n",
      "\n",
      "We observed epithelial phenotypes and increased stemness index in molecular profiles of tumor-type-matched metastatic samples in the MET500 cohort. This portends an association between dedifferentiation and spread of tumor cells to distant organs. The observation is further supported by high mDNAsi in samples from recurrent gliomas. It appears that tumor growth de novo, or at recurrence/metastasis, is associated with an increased stemness phenotype. Decreased mRNAsi levels seen in TGCT suggest its possible differentiation as a germ cell tumor type induced by the microenvironment of liver or lung parenchyma, the organs it most often colonizes. Clinically, in general, tumor progression is associated with greater aggressiveness and resistance to therapy of almost all types.\n",
      "\n",
      "The mRNAsi was high for individual primary glioma and breast cancer cells. Interestingly, when applied to transcriptomic profiles obtained from analysis of single cancer cells in bulk tumors, stemness indices revealed a high degree of intratumor heterogeneity with respect to dedifferentiation phenotype. The heterogeneity was greater in gliomas than in breast cancer, suggesting that intratumor environment, including stromal cells, hypoxia, and infiltration of immune cells, may play a role in shaping CSC niches and affect cancer cell developmental plasticity. Further molecular analyses of cancer cells stratified by the stemness phenotype would provide novel insights into the biology of primary tumors.\n",
      "\n",
      "We found that for a number of tumor types (GBM, LUSC, HNSC, and BLCA), higher mDNAsi was associated with reduced leukocyte fraction and/or lower PD-L1 expression. Such tumors are expected to be less susceptible to immune checkpoint blockade, due either to insufficient immune cell infiltration of tumors or to inherent downregulation of the PD-L1 pathway. Both factors can render immune checkpoint immunotherapy ineffective. The interaction between PD-L1 on cancer cells and PD1 receptor on T cells helps cancer cells elude the immune system by preventing activation of cytotoxic T cells in lymph nodes and subsequent recruitment of other immune cell types to the tumor site (Chen and Mellman, 2013). The presence of tumor-infiltrating lymphocytes and/or PD-L1 expression correlates with aggressiveness in gastrointestinal stromal tumors (Bertucci et al., 2015) and breast carcinomas (Poloñia et al., 2017). Common features shared between cancer cells and stem cells in the context of the immune response are being highlighted by a growing number of studies showing that vaccination with ESCs or iPSCs can raise specific immune response against cancer cells (Kooreman et al., 2018). That finding may indicate that both cell populations use protein networks that, in tumors, result in uncontrolled self-renewal and dedifferentiated phenotypes histopathologically defined by loss of architecture specific to the tissue of origin. We speculate that the indices described here may help predict the efficacy of stem-cell-based immunotherapies and contribute to the identification of patients who will respond to such therapies.\n",
      "\n",
      "We interrogated CMap using the gene expression signatures from tumor samples with the highest and lowest mRNAsi levels. Surprisingly, the CMap analysis, which is based on only a limited number of treated cell lines, very precisely selected drugs that have been shown to affect cancer stem cells with specificity. These translational analyses may ultimately pave the way for implementation of differentiation therapies for solid tumors. Here, we have also shown that cancer hallmarks can be extracted from datasets on cells with defined phenotypes and used to train machine-learning methods applicable to index molecular profiles of cancer. Our mRNAsi and mDNAsi can be translated into stemness scores (e.g., STEM50) that stratify tumors based on their dedifferentiation features, thus providing biomarkers for prediction of patient outcomes and response to differentiation therapies.\n",
      "\n",
      "By defining new metrics of cancer stemness and using them to interrogate TCGA datasets, our results provide a comprehensive characterization of dedifferentiation as new and significant hallmarks of cancer. The strengths of the approach are that it leverages features of dedifferentiated cells across a spectrum of tumor types that reflect tumor pathology and, in some cases, clinical outcome. This study also provides strategies for integrated analysis of cancer genomics based on machine-learning methods trained on molecular profiles obtained from cells with defined phenotypes. The findings based on those methods may advance the development of objective diagnostics tools for quantitating cancer stemness in clinical tumors, perhaps leading eventually to new biomarkers that predict tumor recurrence, guide treatment selection, or improve responses to therapy.\n",
      "```\n"
     ]
    }
   ],
   "source": [
    "console.log(response.choices[0].message.content)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 56,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "SUMMARY\n",
      "\n",
      "Cancer progression involves the gradual loss of a differentiated phenotype and acquisition of progenitor and stem-cell-like features. Here, we provide novel stemness indices for assessing the degree of oncogenic dedifferentiation. We used an innovative one-class logistic regression (OCLR) machine-learning algorithm to extract transcriptomic and epigenetic feature sets derived from non-transformed pluripotent stem cells and their differentiated progeny. Using OCLR, we were able to identify previously undiscovered biological mechanisms associated with the dedifferentiated oncogenic state. Analyses of the tumor microenvironment revealed unanticipated correlation of cancer stemness with immune checkpoint expression and infiltrating immune cells. We found that the dedifferentiated oncogenic phenotype was generally most prominent in metastatic tumors. Application of our stemness indices to single-cell data revealed patterns of intra-tumor molecular heterogeneity. Finally, the indices allowed for the identification of novel targets and possible targeted therapies aimed at tumor differentiation.\n",
      "\n",
      "INTRODUCTION\n",
      "\n",
      "Stemness, defined as the potential for self-renewal and differentiation from the cell of origin, was originally attributed to normal stem cells that possess the ability to give rise to all cell types in the adult organism. Cancer progression involves gradual loss of a differentiated phenotype and acquisition of progenitor-like, stem-cell-like features. Undifferentiated primary tumors are more likely to result in cancer cell spread to distant organs, causing disease progression and poor prognosis, particularly because metastases are usually resistant to available therapies (Friedmann-Morvinski and Verma, 2014; Ge et al., 2017; Shibue and Weinberg, 2017; Visvader and Lindeman, 2012).\n",
      "\n",
      "An increasing number of genomic, epigenomic, transcriptomic, and proteomic signatures have been associated with cancer stemness. Those molecular features are causally connected to particular oncogenic signaling pathways that regulate transcriptional networks that sustain the growth and proliferation of cancer cells (Ben-Porath et al., 2008; Eppert et al., 2011; Kim et al., 2010). Transcriptional and epigenetic dysregulation of cancer cells frequently leads to oncogenic dedifferentiation and acquisition of stemness features by altering core signaling pathways that regulate the phenotypes of normal stem cells (Bradner et al., 2017; Young, 2011). Cell-extrinsic mechanisms can also affect maintenance of the undifferentiated state, largely through epigenetic mechanisms. Tumors comprise a complex, diverse, integrated ecosystem of relatively differentiated cancer cells, cancer stem cells, endothelial cells, tumor-associated fibroblasts, and infiltrating immune cells, among other cell types. The microenvironment of a tumor, considered as a pathologically formed “organ,” is frequently characterized by hypoxia, as well as by abnormal levels of various cytokines, growth factors, and metabolites (Lyssiotis and Kimmelman, 2017). It provides numerous opportunities for cell-cell signals to modulate the epigenome and expression of stem-cell-like programs in cancer cells, frequently independent of their genetic backgrounds (Gingold et al., 2016).\n",
      "\n",
      "Over the last decade, The Cancer Genome Atlas (TCGA) has illuminated the landscapes of primary tumors by generating comprehensive molecular profiles composed of genomic, epigenomic, transcriptomic, and (post-translational) proteomic characteristics (Hoadley et al., 2014; Tomczak et al., 2015), along with histopathological and clinical annotations. The resulting resource enabled us to analyze cancer stemness quite extensively in almost 12,000 samples of 33 tumor types.\n",
      "\n",
      "First, we defined signatures to quantify stemness using publicly available molecular profiles from normal cell types that exhibit various degrees of stemness. By multi-platform analyses of their transcriptome, methylome, and transcription-factor binding sites using an innovative one-class logistic regression (OCLR) machine-learning algorithm (Sokolov et al., 2016), we obtained two independent stemness indices. One (mDNAsi) was reflective of epigenetic features; the other (mRNAsi) was reflective of gene expression. We then identified associations between the two stemness indices and novel oncogenic pathways, somatic alterations, and microRNA (miRNA) and transcriptional regulatory networks. Those features correlated with, and perhaps govern, cancer stemness in particular molecular subtypes of TCGA tumors. Importantly, higher values for stemness indices were associated with biological processes active in cancer stem cells and with greater tumor dedifferentiation, as reflected in histopathological grade. Metastatic tumor cells appeared more dedifferentiated phenotypically, probably contributing to their aggressiveness. We also found tumor heterogeneity at the single-cell level by measuring stemness in transcriptome profiles obtained from individual cancer cells. Using CIBERSORT to profile immune cell types in TCGA tumors, we obtained insight into the interface of the immune system with stemness. Finally, we identified compounds specific to selected molecular targets and mechanisms that may eventually lead to novel treatments that trigger differentiation and exhaust the stemness potential of highly aggressive neoplasms.\n",
      "\n",
      "RESULTS\n",
      "\n",
      "**DNA-Methylation- and mRNA-Expression-Based Stemness Classifiers**\n",
      "\n",
      "We analyzed publicly available non-tumor and tumor datasets for which transcriptomic and epigenomic molecular profiles were available. We derived stemness indices using an OCLR algorithm trained on stem cell (ESC, embryonic stem cell; iPSC, induced pluripotent stem cell) classes and their differentiated ecto-, meso-, and endoderm progenitors. We chose OCLR because it does not penalize misclassification of stem-cell-derived progenitors at different stages of differentiation that still carry some of the undifferentiated features in their molecular profiles (its output was also validated against random forests). OCLR-based transcriptomic and epigenetic signatures were applied to TCGA datasets to calculate the mRNAsi and mDNAsi. Each stemness index (si) ranges from low (zero) to high (one) stemness. The tumor samples stratified by the indices were used for the integrative analyses.\n",
      "\n",
      "**mRNA Expression-Based Stemness Index**\n",
      "\n",
      "We validated the mRNAsi by applying it to an external dataset composed of both stem cells and somatic differentiated cells and by scoring molecular subtypes of breast cancers and gliomas that are characterized by different degrees of oncogenic dedifferentiation associated with pathology and clinical outcome. All stem cell samples attained higher stemness index values than samples from differentiated cells. TCGA tumors display various degrees of cancer stemness as revealed by mRNAsi and mDNAsi. Germ-cell tumors, basal breast cancer, and lympho-hematopoietic (Ly-Hem) cancers displayed highly dedifferentiated phenotypes in comparison to other tumor types.\n",
      "\n",
      "Using gene set enrichment analysis (GSEA), we compared our signature to 16 gene sets that were associated with stemness in cancer and healthy cells in previous studies. These sets spanned 2,564 unique genes, with no two sets overlapping by more than 134 genes. In all cases, the published stemness gene sets were significantly enriched in mRNAsi. We found that “cancer hallmark” gene sets were significantly enriched, as were MYC targets, which significantly contributed to the positive side of the signature. This is consistent with MYC being one of the transcription factors that drive pluripotency in ESCs. Wingless-related integration site (Wnt)/β-catenin and TGF-β signaling pathways were significantly enriched on the negative side of the stemness signature. This negative enrichment does not imply absence of specific signals in cancer stem cells, but rather that this signaling is lower relative to stem-cell-derived progenitors, as captured by the signature weights. This is again consistent with other GSEA results, as both signaling pathways are known mediators of the epithelial-mesenchymal transition (EMT) mechanism. We also computed the correlation of mRNAsi against mRNA expression of published pan-cancer EMT markers, which revealed significant correlations for most tumors. This is consistent with the biology of ESCs, which grow as epithelioid, polygonal cells in vitro and epithelial cancer precursors having stem-like properties. Importantly, most TCGA samples are primary tumors of an epithelial phenotype. Most skin melanoma cases come from lymph nodes, and this tumor type shows higher expression of vimentin, a key marker of a mesenchymal phenotype. mRNAsi is positively correlated with other core stem cell factors: EZH2, OCT4, and SOX2. Finally, Moonlight analysis of the oncogenic signatures from the Molecular Signatures Database (MSigDB) further validated our gene-expression-based index and confirmed engagement of MYC and EZH2, along with E2F3, MTOR, and SHH in driving oncogenic dedifferentiation.\n",
      "\n",
      "**DNA Methylation-Based Stemness Index**\n",
      "\n",
      "We defined the mDNAsi using OCLR by combining (1) supervised classification between ESCs/iPSCs and their progenies, (2) stem cell signatures associated with pluripotency-specific genomic enhancer elements based on ChromHMM from Roadmap, and (3) ELMER, which uses DNA methylation to identify enhancer elements and correlates their state with the expression of nearby genes. 219 CpG probes were selected in training OCLR using the Progenitor Cell Biology Consortium (PCBC) datasets. By selecting probes previously defined to be active stemness-specific enhancers, we confirmed the ability of our approach to derive an mDNAsi. Since we focused exclusively on hypomethylated, functionally important CpG probes associated with stem cells, we further explored cis-activated genes.\n",
      "\n",
      "We scored each TCGA sample using the mDNAsi and used an external dataset to confirm that stem cells had higher mDNAsi than differentiated samples. TCGA tumor types show different degrees of an inferred dedifferentiated phenotype. Within these, individual tumor samples show variation for cancer stemness. As anticipated, TCGA samples derived from the primary tumors show higher cancer stemness indices compared to non-tumor samples obtained from adjacent normal tissue of origin. Most of our selected probes fell within non-promoter elements, yet the SOX2-OCT4 transcription factor binding motif is one of the most highly enriched signatures within these regions. The SOX2-OCT4 complex is a critical master regulator of pluripotency and stemness and is highly enriched in tumor samples with high mDNAsi.\n",
      "\n",
      "**Correlations of mRNAsi and mDNAsi**\n",
      "\n",
      "Since the inputs for mDNAsi and mRNAsi are not necessarily complementary, we explored stratification of glioma samples by the epigenetically regulated mRNAsi (EREG-mRNAsi), a stemness index generated using a set of stemness-related epigenetically regulated genes. The EREG-mRNAsi, based on both RNA expression and epigenetics, elucidates the discrepancy between mDNAsi and mRNAsi and shows a positive correlation with both indices. Both mRNAsi and mDNAsi show good correspondence for a majority of tumors. We observed major discrepancies in the case of brain lower grade glioma (LGG), thyroid carcinoma (THCA), and thymoma (THYM). For gliomas, mDNAsi is correlated positively with tumor pathology and clinical features, while mRNAsi shows a negative correlation. This result could arise from a high frequency of isocitrate dehydrogenase mutations (IDH1/2) and resulting DNA hypermethylation.\n",
      "\n",
      "**Stemness Index Can Stratify Recognized Undifferentiated Cancers**\n",
      "\n",
      "We examined breast invasive carcinoma (BRCA), acute myeloid leukemia (AML), and gliomas to study if the mRNAsi/mDNAsi predict stemness in poorly differentiated tumors.\n",
      "\n",
      "In BRCA, we found a strong association between the stemness index and known clinical and molecular features. The mRNAsi was highest in the basal subtype, known to exhibit an aggressive phenotype associated with an undifferentiated state. BRCA samples with high mRNAsi were more likely to be estrogen receptor (ER)-negative and enriched for FAT3 and TP53 mutations. We noted that high mRNAsi was associated with higher protein expression of FOXM1, CYCLINB1, and MSH6, as well as higher miRNA-200 family expression. Invasive lobular type of BRCA (ILC), characterized by better prognosis in comparison to invasive ductal carcinoma (IDC), has a lower mRNAsi. We also applied our indices to non-TCGA BRCA samples and found a similar correlation between mRNAsi and mDNAsi in those samples. Moreover, mRNAsi also stratified BRCA samples with distinct histology in this dataset. Using datasets with estimated tumor cell type composition provided by the epigenetic deconvolution method, we found that both mRNAsi and mDNAsi were more highly correlated with malignant epithelial cells than with normal epithelial cells, suggesting that our indices identify distinct cancerous epithelial cell populations characterized by different features or degrees of stemness.\n",
      "\n",
      "We found an association between the mRNAsi, RNA expression subtypes previously defined by TCGA, and the French-American-British (FAB) classification of AML. The mRNAsi showed the strongest correlation with the stage of myeloid differentiation of the AML samples. FAB subtypes M0 (undifferentiated), M1 (with minimal maturation), and M2 (with maturation) were characterized by high mRNAsi. In contrast, the M3 well-matured promyelocytic subtype, which is associated with benign chromosomal abnormalities and favorable clinical outcome, had low mRNAsi. High mRNAsi was associated with higher expression of miR-181c-3p, miR-22-3p, and miR-30b-3p.\n",
      "\n",
      "We found a strong association between high mDNAsi, high pathologic grade, and recently published molecular subtypes of glioma. mDNAsi was low in less-aggressive gliomas that are characterized by codel and glioma CpG methylator phenotype (G-CIMP)-high features and was highest in highly aggressive glioblastoma multiformes (GBMs) characterized by IDH mutations (G-CIMP low) and poor clinical outcome. Also, high mDNAsi is strongly associated with more aggressive classical and mesenchymal subtypes of GBM, suggesting that it can stratify tumors with distinct clinical outcomes. We also found that high mDNAsi was associated with mutations in NF1 and EGFR and infrequent mutations in IDH1, TP53, CIC, and ATRX, with higher expression of ANNEXIN-A1 protein and lower expression of ANNEXIN-A7 and with expression of the miR-200 family.\n",
      "\n",
      "We obtained similar results on non-TCGA glioma samples for which both mRNA expression and DNA methylation data were available. The negative correlation between mDNAsi and mRNAsi was restricted to LGG samples—specifically, the IDH mutant subtypes (G-CIMP high and codel). IDH1 mutations are known to reduce cell differentiation, and high values of the mRNAsi in a subset of IDH mutant gliomas might capture this phenomenon.\n",
      "\n",
      "**Pan-cancer Stemness Landscape**\n",
      "\n",
      "Next, we tested the ability of our indices to identify previously unexplored features of cancer stemness across all TCGA tumors. First, we performed an enrichment analysis by sorting all TCGA samples by stemness index for each tumor type and looking for associations with mutations and molecular and clinical features. The most salient associations of mRNAsi and mDNAsi are presented, while the following results of the comprehensive analyses are shown in the supplementary material: associations with mutations, associations with miRNA expression and protein abundance, associations with the tumor grading, and clinical outcome.\n",
      "\n",
      "**Correlations of mRNAsi and mDNAsi with Mutations in Genes, miRNA, and Expression of Proteins**\n",
      "\n",
      "We found a strong association of mDNAsi with known molecular subtypes, with somatic mutations in SETD2 and TP53 genes, and with tobacco smoking status in lung adenocarcinoma (LUAD). Current smokers and recently reformed smokers have higher mDNAsi than non-smokers or long-term reformed smokers. This suggests that the stemness of LUAD tumors might be activated in response to environmental stimuli such as smoking and might influence the aggressiveness of the tumor. We also found an association between mDNAsi and higher protein expression of CYCLINB1 and FOXM1, which is a pro-stemness transcription factor upstream of CYCLINB1. FOXM1 has been associated with dedifferentiation in pancreatic cancer cells, as well as tumor proliferation in the kidney and ovarian cancers. Our result suggests that it could be a driver of dedifferentiation and proliferation in breast and lung cancers. Stemness of LUAD tumors is also associated with lower expression of ANNEXIN-A1. ANNEXIN-A1 has been indicated as a differentiation marker in pancreatic and urothelial cancers; therefore, we suspect that the relationship between ANNEXIN and FOXM1 expression and tumor differentiation may extend to other tumor types.\n",
      "\n",
      "Analyses of head and neck squamous cell carcinoma (HNSC) samples revealed that high indices are correlated with NSD1 mutation, E-cadherin protein expression, miR-200-3p, and previously identified classical molecular subtypes. NSD1 mutation was recently linked in HNSC tumors to blockade of cellular differentiation and promotion of oncogenesis. Interestingly, miR-200 family members have been implicated in cancer initiation and metastasis, as well as self-renewal of healthy stem cells. HNSC tumors with high mDNAsi have reduced programmed death ligand 1 (PD-L1) protein level.\n",
      "\n",
      "In liver hepatocellular carcinoma (LIHC) samples, we found an association between mRNAsi and high pathological grade. Negative associations between mRNAsi and the probability of overall survival (OS) or progression-free survival (PFS) were detected. In contrast to the majority of tumor types, LIHC samples with high mRNAsi have low expression of miR-200 family members. The miR-200 family is known to be associated with progression of hepatocellular carcinoma, and the miR-200b-ZEB1 circuit has been suggested as a master regulator of stemness in these cancers. We found associations of mRNAsi with higher CYCLINB1 and ACC1 and with lower PD-L1 and ANNEXIN-A1 protein expression in LIHC. ACC1 was associated with pathomorphological markers of LIHC aggressiveness (vascular invasion and poor differentiation), and its upregulation was correlated with poor OS and disease recurrence in hepatocellular carcinoma patients. LIHC samples with high mRNAsi were associated with specific genomic alterations (e.g., TP53, CTNNB1, AXIN1).\n",
      "\n",
      "Detailed analyses of adrenocortical carcinoma (ACC) samples revealed an association between high mRNAsi and defined molecular subtypes, clinical stage, and mutations in PRKAR1A and TP53 genes. We found a positive correlation between mRNAsi and adrenal differentiation score that is based on expression of 25 genes that are important for adrenal function.\n",
      "\n",
      "**Stemness Indices Are Correlated with Tumor Pathology and Predictive of Clinical Outcome**\n",
      "\n",
      "We observed a positive correlation between tumor histology and pathology grading and both stemness indices for the majority of the TCGA cases. For mRNAsi, the most significant correlations were found for BRCA (IDC and ILC), cervical squamous cell carcinoma and endocervical adenocarcinoma (CESC), LIHC, pancreatic adenocarcinoma (PAAD), and uterine corpus endometrial carcinoma (UCEC). Interestingly, mRNAsi shows low values in GBM and stomach adenocarcinoma (STAD). On the other hand, mDNAsi strongly stratifies glioma by the pathology grade, culminating with the highest value for GBM. The reversed values of mDNAsi and mRNAsi in the case of gliomas were also evident in the clinical data analyses. We found a positive correlation between previously published glioma subtypes and mDNAsi, suggesting that mDNAsi might recapitulate prognostic molecular subtypes.\n",
      "\n",
      "The discordance between the mRNAsi and the mDNAsi for gliomas may be explained in part by the dominant genomic alteration associated with the LGG tumor type. Roughly 80% of LGG tumors carry an IDH1/2 mutation and, as demonstrated by our group and others, confer a genome-wide hypermethylator phenotype (G-CIMP). Given that the mDNAsi is driven primarily by low methylation levels associated with the stemness phenotype, the LGG tumors might resemble non-stem-like phenotypes, which are predominantly hypermethylated. The subgroup of G-CIMP with the lowest overall DNA methylation levels (G-CIMP low) is associated with the worst outcomes. Compared to G-CIMP-high tumors, G-CIMP-low tumors are known to be more proliferative, express cell-cycle-related genes, and have various stem-cell-like genomic features.\n",
      "\n",
      "**Cancer Stemness Indices Are Higher in Tumor Metastases and Reveal Intratumor Heterogeneity**\n",
      "\n",
      "The TCGA samples are derived mostly from primary tumors, except for skin melanoma, for which tissues are mostly metastatic lymph nodes. We used the mRNAsi to interrogate the MET500 dataset comprising expression profiles from 500 metastatic samples obtained from 22 different organs. In most cases, mRNAsi was significantly higher in metastatic samples compared to primary TCGA tumors. Prostate and pancreatic adenocarcinoma metastases had the most dedifferentiated phenotypes and are also more aggressive and resistant to therapies in contrast to primary tumors. Weaker association with the mRNAsi was due to a small number of available samples. Interestingly, testicular germ cell tumors (TGCTs) present the less differentiated phenotypes in primary tumors when compared to distant metastases. Primary TGCT tumor cells have high mRNAsi and may differentiate when metastasizing to distant organs. A similar trend was observed for STAD.\n",
      "\n",
      "Using another dataset, we found that mDNAsi was significantly higher in glioma samples obtained at first recurrence in contrast to primary gliomas. Our results reveal significant dedifferentiation of glioma cancer cells that contribute to glioma recurrence, which is frequently associated with poor prognosis and resistance to treatment. By taking advantage of single-cell transcriptome datasets, we used mRNAsi to probe tumor heterogeneity for oncogenic dedifferentiation of individual cancer cells. We revealed high variation of stemness in the glioma and breast primary tumors. Individual glioma cells showed higher variation of oncogenic dedifferentiation in comparison to breast cancer cells. Single cells from metastases had higher stemness index in breast cancer. Interestingly, the negative correlation of EMT signature and stemness that we observed in TCGA primary tumors was also found in metastatic samples.\n",
      "\n",
      "**Stemness Index Evaluated in the Context of Immune Response**\n",
      "\n",
      "We found that for many tumors, higher stemness indices are associated with a reduced leukocyte fraction and lower PD-L1 expression. For mDNAsi, the most distinctive negative correlations were found in the PanCan-12 squamous cluster (LUSC, lung squamous cell carcinoma; HNSC; BLCA, bladder urothelial carcinoma) and in GBM. For the mRNAsi, the highest negative correlation values were seen in GBM/LGG, prostate adenocarcinoma (PRAD), LIHC, and uterine carcinosarcoma (UCS) tumors. We expect that such tumors will be less susceptible to immune checkpoint blockade treatments due to insufficient immune cell infiltration or preexisting downregulation of the PD-L1 pathway, which makes further inhibition ineffective. Our findings are consistent with previous reports showing a strong correlation between PD-L1 protein expression and infiltration of CD8+ cytotoxic lymphocytes.\n",
      "\n",
      "We further explored correlations between stemness and immune microenvironment variables in the context of molecular subtypes of tumors. Several tumor types exhibit strong (positive or negative) correlations. Except for kidney renal clear cell carcinoma (KIRC), the association between stemness and PD-L1 expression and leukocyte fraction is readily apparent from the increasing and decreasing trends of individual variables across the molecular subtypes. For example, we found mesenchymal tumors to have the highest PD-L1 expression levels, the most significant leukocyte fractions, and the lowest mDNAsi compared to other HNSC subtypes, suggesting potential susceptibility to checkpoint blockade inhibitors. The use of immunotherapy for HNSC tumors is under active investigation with the recent FDA approval of pembrolizumab; however, whether the effectiveness of therapy is limited to specific HNSC molecular subtypes is not clear from those reports.\n",
      "\n",
      "To assess other relationships between stemness and tumor microenvironment, we computed correlations between stemness indices and individual types of immune cells. By applying CIBERSORT, we scored 22 immune cell types for their relative abundance in TCGA tumor samples. These cell types included natural killer (NK) cells, monocytes, macrophages, dendritic and mast cells, eosinophils, and neutrophils. We also obtained absolute estimates by scaling their relative abundance by overall leukocyte infiltration in each tumor as determined by ESTIMATE applied to DNA methylation data. For any given TCGA sample, we calculated the correlation between mDNAsi/mRNAsi and the estimated fraction of individual immune cell types. In addition to individual immune subpopulation fractions, we considered the functional activation of distinct cells by measuring the difference between activated and resting fractions of NK cells, CD4+ T cells, and macrophages. This approach was motivated by recent observations that activation of peripheral CD4+ T cells triggered by immunotherapy is responsible for the specific killing of tumor cells.\n",
      "\n",
      "Although the squamous cluster tumors had a negative correlation between stemness and the fraction of CD4+ T cell populations, the activation state of the CD4+ T cells was higher in dedifferentiated tumors. This finding is consistent with our observation that PD-L1 protein expression is lower in these tumors, suggesting again that immune checkpoint blockade might be ineffective, and an additional mechanism of immune evasion may be operative. The opposite trend is present in thymomas, where PD-L1 protein expression and the fraction of the CD4+ T cell population are positively correlated with tumor dedifferentiation. Likewise, the activation state of CD4+ T cells is lower in dedifferentiated tumors, suggesting that they might be more susceptible to immunotherapy treatments.\n",
      "\n",
      "**Connectivity Map Analysis Identifies Potential Compounds/Inhibitors Capable of Targeting the Stemness Signature**\n",
      "\n",
      "We employed the Connectivity Map (CMap), a data-driven, systematic approach for discovering associations among genes, chemicals, and biological conditions, to search for candidate compounds that might target pathways associated with stemness. We found enrichment for compounds associated with stemness in at least three cancer types. Five compounds are significantly enriched in more than 10 cancer types and have been reported to inhibit stemness-related tumorigenicity: the dopamine receptor antagonists thioridazine and prochlorperazine, the Wnt signaling inhibitor pyrvinium, the HSP90 inhibitor tanespimycin, and the protein synthesis inhibitor puromycin. Further, the telomerase inhibitor gossypol induced apoptosis and growth inhibition of cancer stem cells (CSCs), and histone deacetylase inhibitors such as trichostatin A (SAHA) reduced glioblastoma stem cell growth. According to our analysis, pyrvinium and puromycin could inhibit stemness in LUAD. We found several candidates with recognized anti-CSC activity for HNSC, including the aforementioned compounds. For LIHC, thioridazine, a prospective inhibitor of lung cancer stem cells, pyrvinium, puromycin, prochlorperazine, and others are potential compounds targeting undifferentiated tumors.\n",
      "\n",
      "CMap mode-of-action (MoA) analysis of the 74 compounds revealed 56 mechanisms of action shared by the above compounds. Five compounds (fluspirilene, pimozide, prochlorperazine, thioridazine, and trifluoperazine) shared the MoA of dopamine receptor antagonist. We observed that entinostat, trichostatin-a, and vorinostat shared MoA as HDAC inhibitors, and LY-294002, zaprinast, zardaverine share MoA as phosphodiesterase inhibitors.\n",
      "\n",
      "CMap target analysis revealed 212 distinct drug-target genes shared by the mentioned compounds. Eight genes are targets of five different compounds—namely, DRD2 (8 drugs), HTR2A (7 drugs), HRH1 (6 drugs), ADRA1A (5 drugs), CALM1 (5 drugs), CHRM3 (5 drugs), HTR1A (5 drugs), and HTR2C (5 drugs).\n",
      "\n",
      "Recent polypharmacology studies suggest the need to design compounds that act on multiple genes or molecular pathways. In this study, we observed similar mechanisms of action among different compounds, suggesting that selective therapies can target the undifferentiated phenotypes for selected cancer types.\n",
      "\n",
      "DISCUSSION\n",
      "\n",
      "This study is based on integrated analysis of cancer stemness in almost 12,000 primary human tumors of 33 different cancer types. We interrogated TCGA data for mutations, DNA methylation, expression of mRNA and miRNA, expression and post-translational modification of proteins, histopathological grade, and clinical outcome. Applying CIBERSORT, we gained insight into the tumor microenvironment and composition of immune cell infiltrates. By applying a machine-learning algorithm to molecular datasets from normal stem cells and their progeny, we developed two different molecular metrics of stemness and then used them to assess epigenomic and transcriptomic features of TCGA cancers according to their grade of oncogenic dedifferentiation. Ultimately, the analyses led us to potentially actionable targets (and their MoAs) as candidates for possible differentiation therapy of solid tumors and metastases. Our approach could be applied to longitudinal study of samples from primary, recurrent, and metastatic cancers, and gene expression signatures identified in the tumor samples can be used to interrogate CMap to suggest actionable targets and inhibitors for further analysis.\n",
      "\n",
      "To the best of our knowledge, this is the first study in which molecular PCBC datasets comprised of stem cells and defined populations of their differentiated progeny have been leveraged to develop a classification tool and machine-learning algorithm for analysis of a spectrum of human malignancies. A number of cancer stemness scores, based on genes that are differentially expressed between CSCs and non-CSCs, have been published and are relevant to clinical outcomes in AML (Eppert et al., 2011; Gentles et al., 2010; Ng et al., 2016). In those studies, gene sets enriched in ESCs (e.g., targets of NANOG, OCT4, SOX2, and c-MYC) were frequently overexpressed in poorly differentiated tumors compared with well-differentiated ones. In breast cancers, those gene sets were associated with high-grade estrogen receptor-negative, basal-like tumors and poor clinical outcome (Ben-Porath et al., 2008). Another web-based tool, StemChecker, uses a curated set of 49 published stemness signatures defined by gene expression, RNAi screens, transcription factor binding sites, text mining of the literature, and other computational approaches. But it has been tested only for pancreatic ductal adenocarcinoma. In that case, high expression of stemness genes correlated with poor prognosis (Pintó et al., 2015). All previous studies were transcriptome-based and limited to a narrow set of genes and a small number of tumor types.\n",
      "\n",
      "In the present study, we found oncogenic dedifferentiation to be associated with several characteristics: mutations in genes that encode oncogenes and epigenetic modifiers, perturbations in specific mRNA/miRNA transcriptional networks, and deregulation of signaling pathways. Cancer stemness also appeared to involve core expression of MYC, OCT4, SOX2, and other genes involved in the regulatory circuitry that underlies normal and malignant self-renewal potential. Our indices derived from mRNA expression and DNA methylation signatures reliably stratified tumors of known stemness phenotype. High mRNAsi was associated with basal breast carcinomas but also Her2 and lumB subtypes that are more aggressive than the hormone-dependent lumA group. In contrast, high mDNAsi was strongly associated with high-grade glioblastomas, poor OS, and PFS. The association between stemness signatures and adverse outcome for some tumor types, including gliomas, may reflect malignant cell origins or the impact of their microenvironment.\n",
      "\n",
      "Dedifferentiated cells can arise from different sources: from long-lived stem or progenitor cells that accumulate mutations in oncogenic pathways or via dedifferentiation from non-stem cancer cells that convert to CSCs through deregulation of developmental and/or non-developmental pathways. It is important to distinguish between the inherent stemness of CSCs and dedifferentiation induced by the tumor microenvironment. However, addressing that issue would require further validation beyond the scope of this study using other genomic datasets and/or laboratory experiments.\n",
      "\n",
      "Both stemness indices were lowest in normal cells, increased in primary tumors, and highest in metastases, consistent with the idea that tumor progression generally involves oncogenic dedifferentiation. Interestingly, we observed negative associations between stemness and EMT gene signatures. The relationship between EMT and stemness remains a hotly debated topic, with several studies showing that EMT is necessarily associated with stemness (Fabregat et al., 2016). However, most TCGA data are obtained from primary tumors, which exist in a pre-EMT state, since EMT is strongly associated with tumor progression and with metastasis for many tumor types. Cancer cells in many primary solid tumors are basically epithelial regardless of their degrees of dedifferentiation, but some cells in such contexts could acquire mesenchymal characteristics either by accumulating additional mutations or by undergoing epigenetic changes shaped by the tumor microenvironment. Those mesenchymal cells can traverse the underlying tissue, enter the bloodstream, and seed distant organs, where they reacquire an epithelial phenotype to form metastatic tumors.\n",
      "\n",
      "We observed epithelial phenotypes and increased stemness index in molecular profiles of tumor-type-matched metastatic samples in the MET500 cohort. This portends an association between dedifferentiation and spread of tumor cells to distant organs. The observation is further supported by high mDNAsi in samples from recurrent gliomas. It appears that tumor growth de novo, or at recurrence/metastasis, is associated with an increased stemness phenotype. Decreased mRNAsi levels seen in TGCT suggest its possible differentiation as a germ cell tumor type induced by the microenvironment of liver or lung parenchyma, the organs it most often colonizes. Clinically, in general, tumor progression is associated with greater aggressiveness and resistance to therapy of almost all types.\n",
      "\n",
      "The mRNAsi was high for individual primary glioma and breast cancer cells. Interestingly, when applied to transcriptomic profiles obtained from analysis of single cancer cells in bulk tumors, stemness indices revealed a high degree of intratumor heterogeneity with respect to dedifferentiation phenotype. The heterogeneity was greater in gliomas than in breast cancer, suggesting that intratumor environment, including stromal cells, hypoxia, and infiltration of immune cells, may play a role in shaping CSC niches and affect cancer cell developmental plasticity. Further molecular analyses of cancer cells stratified by the stemness phenotype would provide novel insights into the biology of primary tumors.\n",
      "\n",
      "We found that for a number of tumor types (GBM, LUSC, HNSC, and BLCA), higher mDNAsi was associated with reduced leukocyte fraction and/or lower PD-L1 expression. Such tumors are expected to be less susceptible to immune checkpoint blockade, due either to insufficient immune cell infiltration of tumors or to inherent downregulation of the PD-L1 pathway. Both factors can render immune checkpoint immunotherapy ineffective. The interaction between PD-L1 on cancer cells and PD1 receptor on T cells helps cancer cells elude the immune system by preventing activation of cytotoxic T cells in lymph nodes and subsequent recruitment of other immune cell types to the tumor site (Chen and Mellman, 2013). The presence of tumor-infiltrating lymphocytes and/or PD-L1 expression correlates with aggressiveness in gastrointestinal stromal tumors (Bertucci et al., 2015) and breast carcinomas (Poloñia et al., 2017). Common features shared between cancer cells and stem cells in the context of the immune response are being highlighted by a growing number of studies showing that vaccination with ESCs or iPSCs can raise specific immune response against cancer cells (Kooreman et al., 2018). That finding may indicate that both cell populations use protein networks that, in tumors, result in uncontrolled self-renewal and dedifferentiated phenotypes histopathologically defined by loss of architecture specific to the tissue of origin. We speculate that the indices described here may help predict the efficacy of stem-cell-based immunotherapies and contribute to the identification of patients who will respond to such therapies.\n",
      "\n",
      "We interrogated CMap using the gene expression signatures from tumor samples with the highest and lowest mRNAsi levels. Surprisingly, the CMap analysis, which is based on only a limited number of treated cell lines, very precisely selected drugs that have been shown to affect cancer stem cells with specificity. These translational analyses may ultimately pave the way for implementation of differentiation therapies for solid tumors. Here, we have also shown that cancer hallmarks can be extracted from datasets on cells with defined phenotypes and used to train machine-learning methods applicable to index molecular profiles of cancer. Our mRNAsi and mDNAsi can be translated into stemness scores (e.g., STEM50) that stratify tumors based on their dedifferentiation features, thus providing biomarkers for prediction of patient outcomes and response to differentiation therapies.\n",
      "\n",
      "By defining new metrics of cancer stemness and using them to interrogate TCGA datasets, our results provide a comprehensive characterization of dedifferentiation as new and significant hallmarks of cancer. The strengths of the approach are that it leverages features of dedifferentiated cells across a spectrum of tumor types that reflect tumor pathology and, in some cases, clinical outcome. This study also provides strategies for integrated analysis of cancer genomics based on machine-learning methods trained on molecular profiles obtained from cells with defined phenotypes. The findings based on those methods may advance the development of objective diagnostics tools for quantitating cancer stemness in clinical tumors, perhaps leading eventually to new biomarkers that predict tumor recurrence, guide treatment selection, or improve responses to therapy.\n"
     ]
    }
   ],
   "source": [
    "console.log(response.choices[0].message.content.replace(/^```|```$/g, '').trim())"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Deno",
   "language": "typescript",
   "name": "deno"
  },
  "language_info": {
   "codemirror_mode": "typescript",
   "file_extension": ".ts",
   "mimetype": "text/x.typescript",
   "name": "typescript",
   "nbconvert_exporter": "script",
   "pygments_lexer": "typescript",
   "version": "5.6.2"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
